sql/tests: TestCreateRandomSchema failed

sql/tests.TestCreateRandomSchema [failed](https://mesolite.cluster.engflow.com/invocations/default/26c20bfc-64e3-49fc-88af-23563ca6ec08?testReportRun=1&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvc3FsL3Rlc3RzOnRlc3RzX3Rlc3Q=) on master @ [108ce0f99414e4a76f5fa3e460706e6236e02f08](https://github.com/cockroachdb/cockroach/commits/108ce0f99414e4a76f5fa3e460706e6236e02f08):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCreateRandomSchema477618548
    test_log_scope.go:76: use -show-logs to present logs inline
    random_schema_test.go:107: for input statement
        CREATE TABLE table74 (col74_0 BPCHAR NOT NULL, col74_1 STRING, col74_2 INT8 NOT NULL, col74_3 INT8 NOT NULL, col74_4 INTERVAL NOT NULL, col74_5 INT4 NOT NULL, col74_6 BOX2D NOT NULL, col74_7 INT4 NOT NULL AS (abs(col74_5)) STORED, col74_8 INT8 NOT NULL AS (abs(col74_3)) STORED, col74_9 INT8 NOT NULL AS ((col74_3 + col74_2) + col74_5) VIRTUAL, col74_10 INT8 NOT NULL AS (col74_2 + col74_5) VIRTUAL, col74_11 STRING NOT NULL AS (lower(CAST(col74_6 AS STRING))) VIRTUAL, col74_12 INT8 NOT NULL AS (abs(col74_3)) VIRTUAL, PRIMARY KEY (col74_10 ASC, col74_6 ASC, col74_4 DESC), UNIQUE (col74_9 ASC, col74_0 DESC, col74_12 ASC, col74_1 DESC, col74_4 ASC, col74_6, col74_7 DESC, col74_11), INDEX (col74_8, col74_2 ASC, col74_0, col74_1 DESC, col74_7 DESC, col74_4, col74_3, col74_5, col74_12 DESC, col74_10 DESC), UNIQUE (col74_3), UNIQUE (col74_11 DESC, col74_0 ASC, col74_2 ASC) PARTITION BY LIST (col74_11, col74_0, col74_2) (PARTITION table74_part_0 VALUES IN ((e'(\x0e\n]':::STRING, ' ':::STRING, 719093526511692457:::INT8), (e'\x11\x19Q#7]':::STRING, '(':::STRING, 5274592778260151416:::INT8)), PARTITION table74_part_1 VALUES IN ((e'4U_f>\x7f\x03+':::STRING, e'\r':::STRING, 2208581895336924722:::INT8), ('jV@_':::STRING, e'\x02':::STRING, (-2812879838062368897):::INT8)), PARTITION table74_part_2 VALUES IN ((e'\x11\x06N\x14\x11':::STRING, 'u':::STRING, 2370269809838248347:::INT8), (e'\x1e"':::STRING, '3':::STRING, 127:::INT8)), PARTITION table74_part_3 VALUES IN ((e'\x0eW\x0eG88\'\x05\x1f':::STRING, 'Y':::STRING, (-5675552390126004252):::INT8), (e'\x00':::STRING, e'\x1d':::STRING, (-5353909169020148120):::INT8)), PARTITION table74_part_4 VALUES IN ((e'8t\x14\x13':::STRING, e'\x00':::STRING, (-2076855166914933686):::INT8), (e'r\x07nc q':::STRING, e'\x11':::STRING, 1899570840361400656:::INT8)), PARTITION table74_part_5 VALUES IN ((e'\x1b':::STRING, 'J':::STRING, (-3228451624343019173):::INT8), (e'\x17Z\ty\x16Uyj{':::STRING, e'\x19':::STRING, (-4116376490675854141):::INT8)), PARTITION table74_part_6 VALUES IN (('X':::STRING, '2':::STRING, (-5865707888760715052):::INT8), (e'\x17\x05\x1fcTU':::STRING, 'p':::STRING, 417701144915150808:::INT8)), PARTITION table74_part_7 VALUES IN ((e'8\x04&Y\\.\x11':::STRING, e'\r':::STRING, 127:::INT8), (e'+y\x14\x18':::STRING, e'\x00':::STRING, 3717268026797901550:::INT8)), PARTITION table74_part_8 VALUES IN (('':::STRING, '@':::STRING, (-3952916644766049872):::INT8), ('':::STRING, 'I':::STRING, 2783054515474739358:::INT8))), UNIQUE (col74_8) STORING (col74_0, col74_1, col74_2, col74_7), UNIQUE (col74_3 ASC, col74_1), UNIQUE (col74_6) STORING (col74_2, col74_8), INDEX (col74_9 ASC), FAMILY (col74_7, col74_3, col74_0, col74_1, col74_5), FAMILY (col74_4, col74_8, col74_2, col74_6))
        found first output
        "CREATE TABLE public.table74 (col74_0 BPCHAR NOT NULL, col74_1 STRING NULL, col74_2 INT8 NOT NULL, col74_3 INT8 NOT NULL, col74_4 INTERVAL NOT NULL, col74_5 INT4 NOT NULL, col74_6 BOX2D NOT NULL, col74_7 INT4 NOT NULL AS (abs(col74_5)) STORED, col74_8 INT8 NOT NULL AS (abs(col74_3)) STORED, col74_9 INT8 NOT NULL AS ((col74_3 + col74_2) + col74_5) VIRTUAL, col74_10 INT8 NOT NULL AS (col74_2 + col74_5) VIRTUAL, col74_11 STRING NOT NULL AS (lower(CAST(col74_6 AS STRING))) VIRTUAL, col74_12 INT8 NOT NULL AS (abs(col74_3)) VIRTUAL, CONSTRAINT table74_pkey PRIMARY KEY (col74_10 ASC, col74_6 ASC, col74_4 DESC), CONSTRAINT table74_col74_9_col74_0_col74_12_col74_1_col74_4_col74_6_col74_7_col74_11_key UNIQUE (col74_9 ASC, col74_0 DESC, col74_12 ASC, col74_1 DESC, col74_4 ASC, col74_6 ASC, col74_7 DESC, col74_11 ASC), INDEX table74_col74_8_col74_2_col74_0_col74_1_col74_7_col74_4_col74_3_col74_5_col74_12_col74_10_idx (col74_8 ASC, col74_2 ASC, col74_0 ASC, col74_1 DESC, col74_7 DESC, col74_4 ASC, col74_3 ASC, col74_5 ASC, col74_12 DESC, col74_10 DESC), CONSTRAINT table74_col74_3_key UNIQUE (col74_3 ASC), CONSTRAINT table74_col74_11_col74_0_col74_2_key UNIQUE (col74_11 DESC, col74_0 ASC, col74_2 ASC) PARTITION BY LIST (col74_11, col74_0, col74_2) (PARTITION table74_part_0 VALUES IN ((e'(\\x0e\\n]', ' ', 719093526511692457), (e'\\x11\\x19Q#7]', '(', 5274592778260151416)), PARTITION table74_part_1 VALUES IN ((e'4U_f>\\x7f\\x03+', e'\\r', 2208581895336924722), ('jV@_', e'\\x02', -2812879838062368897)), PARTITION table74_part_2 VALUES IN ((e'\\x11\\x06N\\x14\\x11', 'u', 2370269809838248347), (e'\\x1e\"', '3', 127)), PARTITION table74_part_3 VALUES IN ((e'\\x0eW\\x0eG88\\'\\x05\\x1f', 'Y', -5675552390126004252), (e'\\x00', e'\\x1d', -5353909169020148120)), PARTITION table74_part_4 VALUES IN ((e'8t\\x14\\x13', e'\\x00', -2076855166914933686), (e'r\\x07nc q', e'\\x11', 1899570840361400656)), PARTITION table74_part_5 VALUES IN ((e'\\x1b', 'J', -3228451624343019173), (e'\\x17Z\\ty\\x16Uyj{', e'\\x19', -4116376490675854141)), PARTITION table74_part_6 VALUES IN (('X', '2', -5865707888760715052), (e'\\x17\\x05\\x1fcTU', 'p', 417701144915150808)), PARTITION table74_part_7 VALUES IN ((e'8\\x04&Y\\\\.\\x11', e'\\r', 127), (e'+y\\x14\\x18', e'\\x00', 3717268026797901550)), PARTITION table74_part_8 VALUES IN (('', '@', -3952916644766049872), ('', 'I', 2783054515474739358))), CONSTRAINT table74_col74_8_key UNIQUE (col74_8 ASC) STORING (col74_0, col74_1, col74_2, col74_7), CONSTRAINT table74_col74_3_col74_1_key UNIQUE (col74_3 ASC, col74_1 ASC), CONSTRAINT table74_col74_6_key UNIQUE (col74_6 ASC) STORING (col74_2, col74_8), INDEX table74_col74_9_idx (col74_9 ASC), FAMILY fam_0_col74_7_col74_3_col74_0_col74_1_col74_5 (col74_7, col74_3, col74_0, col74_1, col74_5), FAMILY fam_1_col74_4_col74_8_col74_2_col74_6 (col74_4, col74_8, col74_2, col74_6))"
        but second output
        "CREATE TABLE public.table74 (col74_0 BPCHAR NOT NULL, col74_1 STRING NULL, col74_2 INT8 NOT NULL, col74_3 INT8 NOT NULL, col74_4 INTERVAL NOT NULL, col74_5 INT4 NOT NULL, col74_6 BOX2D NOT NULL, col74_7 INT4 NOT NULL AS (abs(col74_5)) STORED, col74_8 INT8 NOT NULL AS (abs(col74_3)) STORED, col74_9 INT8 NOT NULL AS ((col74_3 + col74_2) + col74_5) VIRTUAL, col74_10 INT8 NOT NULL AS (col74_2 + col74_5) VIRTUAL, col74_11 STRING NOT NULL AS (lower(CAST(col74_6 AS STRING))) VIRTUAL, col74_12 INT8 NOT NULL AS (abs(col74_3)) VIRTUAL, CONSTRAINT table74_pkey PRIMARY KEY (col74_10 ASC, col74_6 ASC, col74_4 DESC), CONSTRAINT table74_col74_9_col74_0_col74_12_col74_1_col74_4_col74_6_col74_7_col74_11_key UNIQUE (col74_9 ASC, col74_0 DESC, col74_12 ASC, col74_1 DESC, col74_4 ASC, col74_6 ASC, col74_7 DESC, col74_11 ASC), INDEX table74_col74_8_col74_2_col74_0_col74_1_col74_7_col74_4_col74_3_col74_5_col74_12_col74_10_idx (col74_8 ASC, col74_2 ASC, col74_0 ASC, col74_1 DESC, col74_7 DESC, col74_4 ASC, col74_3 ASC, col74_5 ASC, col74_12 DESC, col74_10 DESC), CONSTRAINT table74_col74_3_key UNIQUE (col74_3 ASC), CONSTRAINT table74_col74_11_col74_0_col74_2_key UNIQUE (col74_11 DESC, col74_0 ASC, col74_2 ASC) PARTITION BY LIST (col74_11, col74_0, col74_2) (PARTITION table74_part_0 VALUES IN ((e'(\\x0e\\n]', '', 719093526511692457), (e'\\x11\\x19Q#7]', '(', 5274592778260151416)), PARTITION table74_part_1 VALUES IN ((e'4U_f>\\x7f\\x03+', e'\\r', 2208581895336924722), ('jV@_', e'\\x02', -2812879838062368897)), PARTITION table74_part_2 VALUES IN ((e'\\x11\\x06N\\x14\\x11', 'u', 2370269809838248347), (e'\\x1e\"', '3', 127)), PARTITION table74_part_3 VALUES IN ((e'\\x0eW\\x0eG88\\'\\x05\\x1f', 'Y', -5675552390126004252), (e'\\x00', e'\\x1d', -5353909169020148120)), PARTITION table74_part_4 VALUES IN ((e'8t\\x14\\x13', e'\\x00', -2076855166914933686), (e'r\\x07nc q', e'\\x11', 1899570840361400656)), PARTITION table74_part_5 VALUES IN ((e'\\x1b', 'J', -3228451624343019173), (e'\\x17Z\\ty\\x16Uyj{', e'\\x19', -4116376490675854141)), PARTITION table74_part_6 VALUES IN (('X', '2', -5865707888760715052), (e'\\x17\\x05\\x1fcTU', 'p', 417701144915150808)), PARTITION table74_part_7 VALUES IN ((e'8\\x04&Y\\\\.\\x11', e'\\r', 127), (e'+y\\x14\\x18', e'\\x00', 3717268026797901550)), PARTITION table74_part_8 VALUES IN (('', '@', -3952916644766049872), ('', 'I', 2783054515474739358))), CONSTRAINT table74_col74_8_key UNIQUE (col74_8 ASC) STORING (col74_0, col74_1, col74_2, col74_7), CONSTRAINT table74_col74_3_col74_1_key UNIQUE (col74_3 ASC, col74_1 ASC), CONSTRAINT table74_col74_6_key UNIQUE (col74_6 ASC) STORING (col74_2, col74_8), INDEX table74_col74_9_idx (col74_9 ASC), FAMILY fam_0_col74_7_col74_3_col74_0_col74_1_col74_5 (col74_7, col74_3, col74_0, col74_1, col74_5), FAMILY fam_1_col74_4_col74_8_col74_2_col74_6 (col74_4, col74_8, col74_2, col74_6))"
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema477618548
--- FAIL: TestCreateRandomSchema (4.98s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-44209

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17670838?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17670838?buildTab=artifacts#/) on master @ [688e82e8d015350fe3aa263484416d28b232a25d](https://github.com/cockroachdb/cockroach/commits/688e82e8d015350fe3aa263484416d28b232a25d):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema1901652946
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:107: for input statement
        CREATE TABLE table67 (col67_0 INT4 NOT NULL, col67_1 TSVECTOR NOT NULL, col67_2 REGTYPE NULL, col67_3 VECTOR NOT NULL, col67_4 NAME NOT NULL, col67_5 VARCHAR NOT NULL, col67_6 PG_LSN, col67_7 TIME NULL, col67_8 TSQUERY NOT NULL, col67_9 BPCHAR NOT NULL, col67_10 OID, col67_11 STRING AS (lower(CAST(col67_10 AS STRING))) VIRTUAL, UNIQUE (lower(col67_5) DESC) STORING (col67_0, col67_1, col67_7, col67_10), INVERTED INDEX (col67_2 DESC, col67_4 gin_trgm_ops), UNIQUE (col67_2 ASC) STORING (col67_1, col67_5, col67_8), UNIQUE (col67_2 DESC) STORING (col67_1, col67_3, col67_5, col67_6, col67_9, col67_10), UNIQUE (col67_9) PARTITION BY LIST (col67_9) (PARTITION table67_part_0 VALUES IN (('_':::STRING,), ('4':::STRING,), ('(':::STRING,), ('e':::STRING,), (' ':::STRING,), ('=':::STRING,))), UNIQUE (col67_11), FAMILY (col67_7), FAMILY (col67_5), FAMILY (col67_9), FAMILY (col67_2, col67_6, col67_10, col67_4, col67_0, col67_8, col67_3, col67_1))
        found first output
        "CREATE TABLE public.table67 (col67_0 INT4 NOT NULL, col67_1 TSVECTOR NOT NULL, col67_2 REGTYPE NULL, col67_3 VECTOR NOT NULL, col67_4 NAME NOT NULL, col67_5 VARCHAR NOT NULL, col67_6 PG_LSN NULL, col67_7 TIME NULL, col67_8 TSQUERY NOT NULL, col67_9 BPCHAR NOT NULL, col67_10 OID NULL, col67_11 STRING NULL AS (lower(CAST(col67_10 AS STRING))) VIRTUAL, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table67_pkey PRIMARY KEY (rowid ASC), CONSTRAINT table67_expr_key UNIQUE (lower(col67_5) DESC) STORING (col67_0, col67_1, col67_7, col67_10), INVERTED INDEX table67_col67_2_col67_4_idx (col67_2 DESC, col67_4 gin_trgm_ops), CONSTRAINT table67_col67_2_key UNIQUE (col67_2 ASC) STORING (col67_1, col67_5, col67_8), CONSTRAINT table67_col67_2_key1 UNIQUE (col67_2 DESC) STORING (col67_1, col67_3, col67_5, col67_6, col67_9, col67_10), CONSTRAINT table67_col67_9_key UNIQUE (col67_9 ASC) PARTITION BY LIST (col67_9) (PARTITION table67_part_0 VALUES IN (('_'), ('4'), ('('), ('e'), (' '), ('='))), CONSTRAINT table67_col67_11_key UNIQUE (col67_11 ASC), FAMILY fam_0_col67_7_rowid (col67_7, rowid), FAMILY fam_1_col67_5 (col67_5), FAMILY fam_2_col67_9 (col67_9), FAMILY fam_3_col67_2_col67_6_col67_10_col67_4_col67_0_col67_8_col67_3_col67_1 (col67_2, col67_6, col67_10, col67_4, col67_0, col67_8, col67_3, col67_1))"
        but second output
        "CREATE TABLE public.table67 (col67_0 INT4 NOT NULL, col67_1 TSVECTOR NOT NULL, col67_2 REGTYPE NULL, col67_3 VECTOR NOT NULL, col67_4 NAME NOT NULL, col67_5 VARCHAR NOT NULL, col67_6 PG_LSN NULL, col67_7 TIME NULL, col67_8 TSQUERY NOT NULL, col67_9 BPCHAR NOT NULL, col67_10 OID NULL, col67_11 STRING NULL AS (lower(CAST(col67_10 AS STRING))) VIRTUAL, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table67_pkey PRIMARY KEY (rowid ASC), CONSTRAINT table67_expr_key UNIQUE (lower(col67_5) DESC) STORING (col67_0, col67_1, col67_7, col67_10), INVERTED INDEX table67_col67_2_col67_4_idx (col67_2 DESC, col67_4 gin_trgm_ops), CONSTRAINT table67_col67_2_key UNIQUE (col67_2 ASC) STORING (col67_1, col67_5, col67_8), CONSTRAINT table67_col67_2_key1 UNIQUE (col67_2 DESC) STORING (col67_1, col67_3, col67_5, col67_6, col67_9, col67_10), CONSTRAINT table67_col67_9_key UNIQUE (col67_9 ASC) PARTITION BY LIST (col67_9) (PARTITION table67_part_0 VALUES IN (('_'), ('4'), ('('), ('e'), (''), ('='))), CONSTRAINT table67_col67_11_key UNIQUE (col67_11 ASC), FAMILY fam_0_col67_7_rowid (col67_7, rowid), FAMILY fam_1_col67_5 (col67_5), FAMILY fam_2_col67_9 (col67_9), FAMILY fam_3_col67_2_col67_6_col67_10_col67_4_col67_0_col67_8_col67_3_col67_1 (col67_2, col67_6, col67_10, col67_4, col67_0, col67_8, col67_3, col67_1))"
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema1901652946
--- FAIL: TestCreateRandomSchema (3.75s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://tanzanite.cluster.engflow.com/invocations/default/7a043f45-0327-499c-88c8-5a3f68932ec2?testReportRun=2&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvc3FsL3Rlc3RzOnRlc3RzX3Rlc3Q=) on master @ [688e82e8d015350fe3aa263484416d28b232a25d](https://github.com/cockroachdb/cockroach/commits/688e82e8d015350fe3aa263484416d28b232a25d):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCreateRandomSchema844529484
    test_log_scope.go:76: use -show-logs to present logs inline
    random_schema_test.go:107: for input statement
        CREATE TABLE table34 (col34_0 BPCHAR NOT NULL, col34_1 BOX2D NOT NULL, col34_2 JSONB NULL, col34_3 INET NOT NULL, col34_4 TIME NOT NULL, col34_5 "char", PRIMARY KEY (col34_1 DESC), INDEX (col34_1, col34_4, col34_0, col34_3) PARTITION BY LIST (col34_1, col34_4, col34_0) (PARTITION table34_part_0 VALUES IN (('BOX(1.1766037947813084 -0.6880163510903475,2.4115761098397557 1.1194017062675239)':::BOX2D, '05:52:34.154728':::TIME, e'\x00':::STRING), ('BOX(-0.5517836950724522 -3.0537082932569284,0.1697360120602066 0.3987271926513374)':::BOX2D, '21:22:30.022393':::TIME, 'j':::STRING), ('BOX(-1.5779220778713676 -0.8110999366566771,0.5174036305989527 0.14853823426039237)':::BOX2D, '24:00:00':::TIME, e'\x1f':::STRING), ('BOX(-1.3782714067977908 0.14562970701477906,0.005436316216338499 0.72704321348867)':::BOX2D, '22:37:39.641203':::TIME, '?':::STRING), ('BOX(-0.18649144416214813 -0.7199426360801102,0.3148769069426456 0.9375929467813413)':::BOX2D, '09:28:30.257182':::TIME, 'o':::STRING), ('BOX(-1.5767412247104424 -1.9508083651131145,0.5371989495315994 -0.011515345431297841)':::BOX2D, '04:12:06.996734':::TIME, '}':::STRING), ('BOX(-10 -10,10 10)':::BOX2D, '08:42:48.15491':::TIME, '%':::STRING), ('BOX(-1.1649719827144267 -1.2183336963912152,0.8298000496348784 1.27925138215931)':::BOX2D, '21:00:52.342986':::TIME, ' ':::STRING), ('BOX(-1.0865088100556264 -0.5007346514269788,1.5371723891624152 0.07257667284388719)':::BOX2D, '12:18:50.520269':::TIME, '@':::STRING)), PARTITION table34_part_1 VALUES IN (('BOX(0.15177524258491343 0.6657637170538228,0.9146268168906159 1.0455770315838793)':::BOX2D, '07:06:30.272017':::TIME, '2':::STRING), ('BOX(0.19555842439502302 -0.027657817106677474,1.7514012654037163 0.6839724416158071)':::BOX2D, '07:46:28.494044':::TIME, '%':::STRING), ('BOX(-0.28219724479428465 1.3192853987023634,0.3530113143100737 1.5506255223481207)':::BOX2D, '10:46:57.350359':::TIME, e'\x17':::STRING), ('BOX(-0.639776053763641 -0.3080320377049511,0.8062925440760199 -0.3060923146761414)':::BOX2D, '15:29:34.917556':::TIME, e'\x04':::STRING), ('BOX(-0.3643046968953012 0.3586766500072135,1.2335026937609985 1.8766872067299638)':::BOX2D, '14:25:59.273726':::TIME, '!':::STRING), ('BOX(-0.3792370602042613 -0.9597635465846246,-0.291768925367919 2.5751698577386812)':::BOX2D, '20:40:49.956369':::TIME, 'c':::STRING), ('BOX(-0.03229583622262311 0.531129995309495,0.5488788714470543 0.6917724236001495)':::BOX2D, '18:15:28.257922':::TIME, '%':::STRING), ('BOX(-1.2930788875460415 0.2091353742497617,0.9868308555917071 0.4864302397153122)':::BOX2D, '21:52:22.664248':::TIME, 'u':::STRING), ('BOX(-1.1387758321974832 0.3140456237585604,1.4291770171003186 0.7281822770145192)':::BOX2D, '04:29:00.992256':::TIME, 'J':::STRING)), PARTITION table34_part_2 VALUES IN (('BOX(-1.7178144605892909 -1.0324388595564864,0.7112763208042603 -0.7701128815204799)':::BOX2D, '10:30:06.823366':::TIME, e'\'':::STRING), ('BOX(-10 -10,10 10)':::BOX2D, '18:44:26.027361':::TIME, 'Y':::STRING), ('BOX(-0.9085007101708407 0.3448183957717996,1.734725462750076 1.774078221808989)':::BOX2D, '02:21:53.973142':::TIME, e'\x00':::STRING), ('BOX(-1.1780820217350434 -0.7846966199695697,-0.43012811675697216 1.2473947194142285)':::BOX2D, '11:40:11.258159':::TIME, '[':::STRING), ('BOX(-1.3956632384624577 -0.19895419972009198,1.1935031180200117 0.2078648669865304)':::BOX2D, '24:00:00':::TIME, 'Y':::STRING), ('BOX(0.45092646326750196 -1.2761025912491393,1.422195410973808 0.06010218273037235)':::BOX2D, '02:30:01.12322':::TIME, 'M':::STRING), ('BOX(-0.9593790758117677 -0.6982214117921737,0.007450723444372809 0.799929335401822)':::BOX2D, '22:07:28.871323':::TIME, e'\x06':::STRING), ('BOX(-0.18379591639608295 0.4583466840463313,0.4572971663751879 2.9049610535165584)':::BOX2D, '04:09:02.415273':::TIME, '-':::STRING), ('BOX(-0.042942979949114946 0.13967635654073707,0.16708021121429728 0.15956227986390487)':::BOX2D, '00:48:32.451029':::TIME, '*':::STRING))), INVERTED INDEX (col34_5 gin_trgm_ops ASC), UNIQUE (col34_0 DESC, col34_5 DESC), INDEX (col34_1 ASC) STORING (col34_4) NOT VISIBLE, UNIQUE (col34_1 DESC) STORING (col34_0, col34_2), INVERTED INDEX (col34_0 ASC, col34_5, col34_2), FAMILY (col34_2), FAMILY (col34_5, col34_3, col34_0, col34_1), FAMILY (col34_4))
        found first output
        "CREATE TABLE public.table34 (col34_0 BPCHAR NOT NULL, col34_1 BOX2D NOT NULL, col34_2 JSONB NULL, col34_3 INET NOT NULL, col34_4 TIME NOT NULL, col34_5 \"char\" NULL, CONSTRAINT table34_pkey PRIMARY KEY (col34_1 DESC), INDEX table34_col34_1_col34_4_col34_0_col34_3_idx (col34_1 ASC, col34_4 ASC, col34_0 ASC, col34_3 ASC) PARTITION BY LIST (col34_1, col34_4, col34_0) (PARTITION table34_part_0 VALUES IN (('BOX(1.1766037947813084 -0.6880163510903475,2.4115761098397557 1.1194017062675239)', '05:52:34.154728', e'\\x00'), ('BOX(-0.5517836950724522 -3.0537082932569284,0.1697360120602066 0.3987271926513374)', '21:22:30.022393', 'j'), ('BOX(-1.5779220778713676 -0.8110999366566771,0.5174036305989527 0.14853823426039237)', '24:00:00', e'\\x1f'), ('BOX(-1.3782714067977908 0.14562970701477906,0.005436316216338499 0.72704321348867)', '22:37:39.641203', '?'), ('BOX(-0.18649144416214813 -0.7199426360801102,0.3148769069426456 0.9375929467813413)', '09:28:30.257182', 'o'), ('BOX(-1.5767412247104424 -1.9508083651131145,0.5371989495315994 -0.011515345431297841)', '04:12:06.996734', '}'), ('BOX(-10 -10,10 10)', '08:42:48.15491', '%'), ('BOX(-1.1649719827144267 -1.2183336963912152,0.8298000496348784 1.27925138215931)', '21:00:52.342986', ' '), ('BOX(-1.0865088100556264 -0.5007346514269788,1.5371723891624152 0.07257667284388719)', '12:18:50.520269', '@')), PARTITION table34_part_1 VALUES IN (('BOX(0.15177524258491343 0.6657637170538228,0.9146268168906159 1.0455770315838793)', '07:06:30.272017', '2'), ('BOX(0.19555842439502302 -0.027657817106677474,1.7514012654037163 0.6839724416158071)', '07:46:28.494044', '%'), ('BOX(-0.28219724479428465 1.3192853987023634,0.3530113143100737 1.5506255223481207)', '10:46:57.350359', e'\\x17'), ('BOX(-0.639776053763641 -0.3080320377049511,0.8062925440760199 -0.3060923146761414)', '15:29:34.917556', e'\\x04'), ('BOX(-0.3643046968953012 0.3586766500072135,1.2335026937609985 1.8766872067299638)', '14:25:59.273726', '!'), ('BOX(-0.3792370602042613 -0.9597635465846246,-0.291768925367919 2.5751698577386812)', '20:40:49.956369', 'c'), ('BOX(-0.03229583622262311 0.531129995309495,0.5488788714470543 0.6917724236001495)', '18:15:28.257922', '%'), ('BOX(-1.2930788875460415 0.2091353742497617,0.9868308555917071 0.4864302397153122)', '21:52:22.664248', 'u'), ('BOX(-1.1387758321974832 0.3140456237585604,1.4291770171003186 0.7281822770145192)', '04:29:00.992256', 'J')), PARTITION table34_part_2 VALUES IN (('BOX(-1.7178144605892909 -1.0324388595564864,0.7112763208042603 -0.7701128815204799)', '10:30:06.823366', e'\\''), ('BOX(-10 -10,10 10)', '18:44:26.027361', 'Y'), ('BOX(-0.9085007101708407 0.3448183957717996,1.734725462750076 1.774078221808989)', '02:21:53.973142', e'\\x00'), ('BOX(-1.1780820217350434 -0.7846966199695697,-0.43012811675697216 1.2473947194142285)', '11:40:11.258159', '['), ('BOX(-1.3956632384624577 -0.19895419972009198,1.1935031180200117 0.2078648669865304)', '24:00:00', 'Y'), ('BOX(0.45092646326750196 -1.2761025912491393,1.422195410973808 0.06010218273037235)', '02:30:01.12322', 'M'), ('BOX(-0.9593790758117677 -0.6982214117921737,0.007450723444372809 0.799929335401822)', '22:07:28.871323', e'\\x06'), ('BOX(-0.18379591639608295 0.4583466840463313,0.4572971663751879 2.9049610535165584)', '04:09:02.415273', '-'), ('BOX(-0.042942979949114946 0.13967635654073707,0.16708021121429728 0.15956227986390487)', '00:48:32.451029', '*'))), INVERTED INDEX table34_col34_5_idx (col34_5 gin_trgm_ops), CONSTRAINT table34_col34_0_col34_5_key UNIQUE (col34_0 DESC, col34_5 DESC), INDEX table34_col34_1_idx (col34_1 ASC) STORING (col34_4) NOT VISIBLE, CONSTRAINT table34_col34_1_key UNIQUE (col34_1 DESC) STORING (col34_0, col34_2), INVERTED INDEX table34_col34_0_col34_5_col34_2_idx (col34_0 ASC, col34_5 ASC, col34_2), FAMILY fam_0_col34_2 (col34_2), FAMILY fam_1_col34_5_col34_3_col34_0_col34_1 (col34_5, col34_3, col34_0, col34_1), FAMILY fam_2_col34_4 (col34_4))"
        but second output
        "CREATE TABLE public.table34 (col34_0 BPCHAR NOT NULL, col34_1 BOX2D NOT NULL, col34_2 JSONB NULL, col34_3 INET NOT NULL, col34_4 TIME NOT NULL, col34_5 \"char\" NULL, CONSTRAINT table34_pkey PRIMARY KEY (col34_1 DESC), INDEX table34_col34_1_col34_4_col34_0_col34_3_idx (col34_1 ASC, col34_4 ASC, col34_0 ASC, col34_3 ASC) PARTITION BY LIST (col34_1, col34_4, col34_0) (PARTITION table34_part_0 VALUES IN (('BOX(1.1766037947813084 -0.6880163510903475,2.4115761098397557 1.1194017062675239)', '05:52:34.154728', e'\\x00'), ('BOX(-0.5517836950724522 -3.0537082932569284,0.1697360120602066 0.3987271926513374)', '21:22:30.022393', 'j'), ('BOX(-1.5779220778713676 -0.8110999366566771,0.5174036305989527 0.14853823426039237)', '24:00:00', e'\\x1f'), ('BOX(-1.3782714067977908 0.14562970701477906,0.005436316216338499 0.72704321348867)', '22:37:39.641203', '?'), ('BOX(-0.18649144416214813 -0.7199426360801102,0.3148769069426456 0.9375929467813413)', '09:28:30.257182', 'o'), ('BOX(-1.5767412247104424 -1.9508083651131145,0.5371989495315994 -0.011515345431297841)', '04:12:06.996734', '}'), ('BOX(-10 -10,10 10)', '08:42:48.15491', '%'), ('BOX(-1.1649719827144267 -1.2183336963912152,0.8298000496348784 1.27925138215931)', '21:00:52.342986', ''), ('BOX(-1.0865088100556264 -0.5007346514269788,1.5371723891624152 0.07257667284388719)', '12:18:50.520269', '@')), PARTITION table34_part_1 VALUES IN (('BOX(0.15177524258491343 0.6657637170538228,0.9146268168906159 1.0455770315838793)', '07:06:30.272017', '2'), ('BOX(0.19555842439502302 -0.027657817106677474,1.7514012654037163 0.6839724416158071)', '07:46:28.494044', '%'), ('BOX(-0.28219724479428465 1.3192853987023634,0.3530113143100737 1.5506255223481207)', '10:46:57.350359', e'\\x17'), ('BOX(-0.639776053763641 -0.3080320377049511,0.8062925440760199 -0.3060923146761414)', '15:29:34.917556', e'\\x04'), ('BOX(-0.3643046968953012 0.3586766500072135,1.2335026937609985 1.8766872067299638)', '14:25:59.273726', '!'), ('BOX(-0.3792370602042613 -0.9597635465846246,-0.291768925367919 2.5751698577386812)', '20:40:49.956369', 'c'), ('BOX(-0.03229583622262311 0.531129995309495,0.5488788714470543 0.6917724236001495)', '18:15:28.257922', '%'), ('BOX(-1.2930788875460415 0.2091353742497617,0.9868308555917071 0.4864302397153122)', '21:52:22.664248', 'u'), ('BOX(-1.1387758321974832 0.3140456237585604,1.4291770171003186 0.7281822770145192)', '04:29:00.992256', 'J')), PARTITION table34_part_2 VALUES IN (('BOX(-1.7178144605892909 -1.0324388595564864,0.7112763208042603 -0.7701128815204799)', '10:30:06.823366', e'\\''), ('BOX(-10 -10,10 10)', '18:44:26.027361', 'Y'), ('BOX(-0.9085007101708407 0.3448183957717996,1.734725462750076 1.774078221808989)', '02:21:53.973142', e'\\x00'), ('BOX(-1.1780820217350434 -0.7846966199695697,-0.43012811675697216 1.2473947194142285)', '11:40:11.258159', '['), ('BOX(-1.3956632384624577 -0.19895419972009198,1.1935031180200117 0.2078648669865304)', '24:00:00', 'Y'), ('BOX(0.45092646326750196 -1.2761025912491393,1.422195410973808 0.06010218273037235)', '02:30:01.12322', 'M'), ('BOX(-0.9593790758117677 -0.6982214117921737,0.007450723444372809 0.799929335401822)', '22:07:28.871323', e'\\x06'), ('BOX(-0.18379591639608295 0.4583466840463313,0.4572971663751879 2.9049610535165584)', '04:09:02.415273', '-'), ('BOX(-0.042942979949114946 0.13967635654073707,0.16708021121429728 0.15956227986390487)', '00:48:32.451029', '*'))), INVERTED INDEX table34_col34_5_idx (col34_5 gin_trgm_ops), CONSTRAINT table34_col34_0_col34_5_key UNIQUE (col34_0 DESC, col34_5 DESC), INDEX table34_col34_1_idx (col34_1 ASC) STORING (col34_4) NOT VISIBLE, CONSTRAINT table34_col34_1_key UNIQUE (col34_1 DESC) STORING (col34_0, col34_2), INVERTED INDEX table34_col34_0_col34_5_col34_2_idx (col34_0 ASC, col34_5 ASC, col34_2), FAMILY fam_0_col34_2 (col34_2), FAMILY fam_1_col34_5_col34_3_col34_0_col34_1 (col34_5, col34_3, col34_0, col34_1), FAMILY fam_2_col34_4 (col34_4))"
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema844529484
--- FAIL: TestCreateRandomSchema (2.30s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://tanzanite.cluster.engflow.com/invocations/default/74abd2b7-28a1-482d-99fb-c84abeecff78?testReportRun=2&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvc3FsL3Rlc3RzOnRlc3RzX3Rlc3Q=) on master @ [8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8](https://github.com/cockroachdb/cockroach/commits/8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCreateRandomSchema3415244750
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:90: CREATE TABLE public.table93 (col93_0 TIMETZ NOT NULL, col93_1 BPCHAR NOT NULL, col93_2 DECIMAL NOT NULL, col93_3 REGROLE NULL, col93_4 BYTES NOT NULL, col93_5 BOOL NOT NULL, col93_6 TIMESTAMPTZ NULL, col93_7 INT4 NOT NULL, col93_8 DECIMAL NOT NULL AS (abs(col93_2)) STORED, col93_9 STRING NOT NULL AS (CASE WHEN col93_4 IS NULL THEN '':::STRING ELSE e'\x05\x1c\x0f\x1a~<Y':::STRING END) VIRTUAL, col93_10 DECIMAL NOT NULL AS (abs(col93_2)) STORED, col93_11 STRING NOT NULL AS (lower(CAST(col93_0 AS STRING))) VIRTUAL, col93_12 STRING NULL AS (CASE WHEN col93_3 IS NULL THEN e'\x00':::STRING ELSE 'R@3@[uP.F':::STRING END) VIRTUAL, col93_13 STRING NULL AS (CASE WHEN col93_6 IS NULL THEN e'\x04i\x02\x0eS\x1bz':::STRING ELSE '':::STRING END) VIRTUAL, CONSTRAINT table93_pkey PRIMARY KEY (col93_7 ASC), INDEX table93_col93_1_idx (col93_1 DESC) STORING (col93_0, col93_2, col93_5, col93_6, col93_8) PARTITION BY LIST (col93_1) (PARTITION table93_part_0 VALUES IN (('Q'), ('A'), ('!'), (e'\x17'), ('e'), ('X'), ('G'), ('l')), PARTITION table93_part_1 VALUES IN (('_'), (e'\''), ('"'), ('j'), ('7'), ('Y')), PARTITION table93_part_2 VALUES IN (('S'), ('*'), ('H'), ('&'), ('s'), (' ')), PARTITION table93_part_3 VALUES IN ((e'\x03'), (e'\\'), (''), ('Z'), ('?'), ('q'), ('J'), (e'\x18')), PARTITION "DEFAULT" VALUES IN ((DEFAULT))), INDEX table93_col93_12_expr_idx (col93_12 DESC, (CASE WHEN col93_6 IS NULL THEN 'F"{xy':::STRING ELSE e'\x1ep&~':::STRING END) ASC), INDEX table93_col93_11_expr_idx (col93_11 ASC, (CASE WHEN col93_6 IS NULL THEN e'R2V2d\x1f':::STRING ELSE e'\x14w':::STRING END) DESC), INDEX table93_col93_9_idx (col93_9 ASC) STORING (col93_2, col93_3)) pq: relation "table93" (295): ('') cannot be present in more than one partition
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema3415244750
--- FAIL: TestCreateRandomSchema (21.74s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://tanzanite.cluster.engflow.com/invocations/default/fc5812b5-b415-4731-a0bc-5fc36e294a23?testReportRun=1&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvc3FsL3Rlc3RzOnRlc3RzX3Rlc3Q=) on master @ [8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8](https://github.com/cockroachdb/cockroach/commits/8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCreateRandomSchema873617448
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:90: CREATE TABLE public.table79 (col79_0 DECIMAL NULL, col79_1 UUID NOT NULL, col79_2 GEOGRAPHY NULL, col79_3 TSQUERY NOT NULL, col79_4 BOOL NULL, col79_5 BPCHAR NOT NULL, col79_6 JSONB NOT NULL, col79_7 TSVECTOR NOT NULL, col79_8 INET NOT NULL, col79_9 BIT(48) NOT NULL, col79_10 TSQUERY NULL, col79_11 TIMESTAMP NOT NULL, col79_12 STRING NULL AS (lower(CAST(col79_2 AS STRING))) VIRTUAL, CONSTRAINT table79_pkey PRIMARY KEY (col79_5 ASC), CONSTRAINT table79_col79_9_col79_8_key UNIQUE (col79_9 ASC, col79_8 ASC), INDEX table79_col79_5_idx (col79_5 ASC) PARTITION BY LIST (col79_5) (PARTITION table79_part_0 VALUES IN (('X'), ('['), (e'\x14'), (')'), (e'\x07'), ('('), ('r'), ('z')), PARTITION table79_part_1 VALUES IN ((e'\x03'), (e'\x02'), ('n'), ('S'), (''), (e'\x12'), (e'\'')), PARTITION table79_part_2 VALUES IN (('s'), ('%'), ('#'), (e'\x1f'), (e'\x16'), (' '), (e'\x11')), PARTITION table79_part_3 VALUES IN ((e'\x00'), (e'\r'), ('I'), (e'\x0e')), PARTITION table79_part_4 VALUES IN (('J'), ('a'), (';'), ('9'), (e'\x1e'), ('/'))) NOT VISIBLE, INDEX table79_col79_8_col79_11_idx (col79_8 DESC, col79_11 ASC) STORING (col79_1, col79_2, col79_3, col79_4, col79_6, col79_9), CONSTRAINT table79_col79_12_key UNIQUE (col79_12 ASC) STORING (col79_3, col79_4, col79_7, col79_8, col79_9)) pq: relation "table79" (267): ('') cannot be present in more than one partition
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema873617448
--- FAIL: TestCreateRandomSchema (10.74s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://tanzanite.cluster.engflow.com/invocations/default/9e6cb81a-d34f-4716-9828-44bdb4e2ff80?testReportRun=3&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvc3FsL3Rlc3RzOnRlc3RzX3Rlc3Q=) on master @ [8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8](https://github.com/cockroachdb/cockroach/commits/8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCreateRandomSchema2555407770
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:107: for input statement
        CREATE TABLE table19 (col19_0 "char"[] NOT NULL, col19_1 BPCHAR NOT NULL, col19_2 "char" NOT NULL, INDEX (lower(col19_2), col19_1 DESC), INDEX (col19_0 DESC, col19_1 ASC) STORING (col19_2), INDEX (col19_0), INDEX (col19_2 DESC) STORING (col19_0, col19_1), UNIQUE (col19_2 ASC), UNIQUE (col19_1 DESC) PARTITION BY LIST (col19_1) (PARTITION table19_part_0 VALUES IN (('i':::STRING,), ('3':::STRING,)), PARTITION table19_part_1 VALUES IN (('u':::STRING,), ('J':::STRING,)), PARTITION table19_part_2 VALUES IN ((e'\x00':::STRING,)), PARTITION table19_part_3 VALUES IN (('7':::STRING,), ('e':::STRING,)), PARTITION table19_part_4 VALUES IN (('&':::STRING,), ('H':::STRING,)), PARTITION table19_part_5 VALUES IN (('0':::STRING,), ('':::STRING,)), PARTITION table19_part_6 VALUES IN (('Y':::STRING,)), PARTITION table19_part_7 VALUES IN ((e'\t':::STRING,), (e'\'':::STRING,)), PARTITION table19_part_8 VALUES IN (('S':::STRING,)), PARTITION table19_part_9 VALUES IN (('K':::STRING,)), PARTITION "DEFAULT" VALUES IN ((DEFAULT,))), INDEX (col19_0 ASC), INDEX (col19_1, col19_2) PARTITION BY LIST (col19_1, col19_2) (PARTITION table19_part_0 VALUES IN (('/':::STRING, 'K':::STRING), (e'\U00002603':::STRING, e'\b':::STRING), (e'\x02':::STRING, e'\'':::STRING), ('3':::STRING, ',':::STRING)), PARTITION table19_part_1 VALUES IN (('!':::STRING, e'\x00':::STRING), ('d':::STRING, 'Q':::STRING), ('!':::STRING, e'\x15':::STRING), ('|':::STRING, ';':::STRING)), PARTITION table19_part_2 VALUES IN (('':::STRING, 'v':::STRING), ('O':::STRING, '0':::STRING), ('I':::STRING, '7':::STRING), ('A':::STRING, 'G':::STRING)), PARTITION table19_part_3 VALUES IN (('I':::STRING, 'g':::STRING), ('u':::STRING, 'a':::STRING), (e'\x00':::STRING, e'\x00':::STRING), (e'\x1e':::STRING, e'\x06':::STRING)), PARTITION table19_part_4 VALUES IN ((e'\f':::STRING, 'k':::STRING), (e'\x1a':::STRING, 'M':::STRING), ('p':::STRING, 'z':::STRING), (' ':::STRING, e'\x16':::STRING)), PARTITION table19_part_5 VALUES IN (('<':::STRING, '``':::STRING), ('2':::STRING, e'\x03':::STRING), (':':::STRING, e'\x10':::STRING), ('!':::STRING, '':::STRING)), PARTITION table19_part_6 VALUES IN ((e'\f':::STRING, 'r':::STRING), ('o':::STRING, e'\U00002603':::STRING), ('7':::STRING, 'b':::STRING), ('X':::STRING, e'\x02':::STRING)), PARTITION table19_part_7 VALUES IN (('k':::STRING, '9':::STRING), ('s':::STRING, ':':::STRING), (e'\x11':::STRING, 'F':::STRING), ('1':::STRING, 'U':::STRING))), UNIQUE (col19_2 DESC), FAMILY (col19_0), FAMILY (col19_2), FAMILY (col19_1))
        found first output
        "CREATE TABLE public.table19 (col19_0 \"char\"[] NOT NULL, col19_1 BPCHAR NOT NULL, col19_2 \"char\" NOT NULL, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table19_pkey PRIMARY KEY (rowid ASC), INDEX table19_expr_col19_1_idx (lower(col19_2) ASC, col19_1 DESC), INDEX table19_col19_0_col19_1_idx (col19_0 DESC, col19_1 ASC) STORING (col19_2), INDEX table19_col19_0_idx (col19_0 ASC), INDEX table19_col19_2_idx (col19_2 DESC) STORING (col19_0, col19_1), CONSTRAINT table19_col19_2_key UNIQUE (col19_2 ASC), CONSTRAINT table19_col19_1_key UNIQUE (col19_1 DESC) PARTITION BY LIST (col19_1) (PARTITION table19_part_0 VALUES IN (('i'), ('3')), PARTITION table19_part_1 VALUES IN (('u'), ('J')), PARTITION table19_part_2 VALUES IN ((e'\\x00')), PARTITION table19_part_3 VALUES IN (('7'), ('e')), PARTITION table19_part_4 VALUES IN (('&'), ('H')), PARTITION table19_part_5 VALUES IN (('0'), ('')), PARTITION table19_part_6 VALUES IN (('Y')), PARTITION table19_part_7 VALUES IN ((e'\\t'), (e'\\'')), PARTITION table19_part_8 VALUES IN (('S')), PARTITION table19_part_9 VALUES IN (('K')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT))), INDEX table19_col19_0_idx1 (col19_0 ASC), INDEX table19_col19_1_col19_2_idx (col19_1 ASC, col19_2 ASC) PARTITION BY LIST (col19_1, col19_2) (PARTITION table19_part_0 VALUES IN (('/', 'K'), (e'\\U00002603', e'\\b'), (e'\\x02', e'\\''), ('3', ',')), PARTITION table19_part_1 VALUES IN (('!', e'\\x00'), ('d', 'Q'), ('!', e'\\x15'), ('|', ';')), PARTITION table19_part_2 VALUES IN (('', 'v'), ('O', '0'), ('I', '7'), ('A', 'G')), PARTITION table19_part_3 VALUES IN (('I', 'g'), ('u', 'a'), (e'\\x00', e'\\x00'), (e'\\x1e', e'\\x06')), PARTITION table19_part_4 VALUES IN ((e'\\f', 'k'), (e'\\x1a', 'M'), ('p', 'z'), (' ', e'\\x16')), PARTITION table19_part_5 VALUES IN (('<', '``'), ('2', e'\\x03'), (':', e'\\x10'), ('!', '')), PARTITION table19_part_6 VALUES IN ((e'\\f', 'r'), ('o', e'\\U00002603'), ('7', 'b'), ('X', e'\\x02')), PARTITION table19_part_7 VALUES IN (('k', '9'), ('s', ':'), (e'\\x11', 'F'), ('1', 'U'))), CONSTRAINT table19_col19_2_key1 UNIQUE (col19_2 DESC), FAMILY fam_0_col19_0_rowid (col19_0, rowid), FAMILY fam_1_col19_2 (col19_2), FAMILY fam_2_col19_1 (col19_1))"
        but second output
        "CREATE TABLE public.table19 (col19_0 \"char\"[] NOT NULL, col19_1 BPCHAR NOT NULL, col19_2 \"char\" NOT NULL, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table19_pkey PRIMARY KEY (rowid ASC), INDEX table19_expr_col19_1_idx (lower(col19_2) ASC, col19_1 DESC), INDEX table19_col19_0_col19_1_idx (col19_0 DESC, col19_1 ASC) STORING (col19_2), INDEX table19_col19_0_idx (col19_0 ASC), INDEX table19_col19_2_idx (col19_2 DESC) STORING (col19_0, col19_1), CONSTRAINT table19_col19_2_key UNIQUE (col19_2 ASC), CONSTRAINT table19_col19_1_key UNIQUE (col19_1 DESC) PARTITION BY LIST (col19_1) (PARTITION table19_part_0 VALUES IN (('i'), ('3')), PARTITION table19_part_1 VALUES IN (('u'), ('J')), PARTITION table19_part_2 VALUES IN ((e'\\x00')), PARTITION table19_part_3 VALUES IN (('7'), ('e')), PARTITION table19_part_4 VALUES IN (('&'), ('H')), PARTITION table19_part_5 VALUES IN (('0'), ('')), PARTITION table19_part_6 VALUES IN (('Y')), PARTITION table19_part_7 VALUES IN ((e'\\t'), (e'\\'')), PARTITION table19_part_8 VALUES IN (('S')), PARTITION table19_part_9 VALUES IN (('K')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT))), INDEX table19_col19_0_idx1 (col19_0 ASC), INDEX table19_col19_1_col19_2_idx (col19_1 ASC, col19_2 ASC) PARTITION BY LIST (col19_1, col19_2) (PARTITION table19_part_0 VALUES IN (('/', 'K'), (e'\\U00002603', e'\\b'), (e'\\x02', e'\\''), ('3', ',')), PARTITION table19_part_1 VALUES IN (('!', e'\\x00'), ('d', 'Q'), ('!', e'\\x15'), ('|', ';')), PARTITION table19_part_2 VALUES IN (('', 'v'), ('O', '0'), ('I', '7'), ('A', 'G')), PARTITION table19_part_3 VALUES IN (('I', 'g'), ('u', 'a'), (e'\\x00', e'\\x00'), (e'\\x1e', e'\\x06')), PARTITION table19_part_4 VALUES IN ((e'\\f', 'k'), (e'\\x1a', 'M'), ('p', 'z'), ('', e'\\x16')), PARTITION table19_part_5 VALUES IN (('<', '``'), ('2', e'\\x03'), (':', e'\\x10'), ('!', '')), PARTITION table19_part_6 VALUES IN ((e'\\f', 'r'), ('o', e'\\U00002603'), ('7', 'b'), ('X', e'\\x02')), PARTITION table19_part_7 VALUES IN (('k', '9'), ('s', ':'), (e'\\x11', 'F'), ('1', 'U'))), CONSTRAINT table19_col19_2_key1 UNIQUE (col19_2 DESC), FAMILY fam_0_col19_0_rowid (col19_0, rowid), FAMILY fam_1_col19_2 (col19_2), FAMILY fam_2_col19_1 (col19_1))"
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema2555407770
--- FAIL: TestCreateRandomSchema (7.80s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://tanzanite.cluster.engflow.com/invocations/default/e40aa913-9600-4a4a-97bf-1b8c9668ca66?testReportRun=3&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvc3FsL3Rlc3RzOnRlc3RzX3Rlc3Q=) on master @ [8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8](https://github.com/cockroachdb/cockroach/commits/8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCreateRandomSchema3393979339
    test_log_scope.go:76: use -show-logs to present logs inline
    random_schema_test.go:107: for input statement
        CREATE TABLE table15 (col15_0 VECTOR, col15_1 INTERVAL NOT NULL, col15_2 TSQUERY, col15_3 BIT(30) NULL, col15_4 FLOAT4[] NOT NULL, col15_5 BPCHAR NOT NULL, col15_6 DECIMAL NOT NULL, col15_7 VECTOR, col15_8 REGTYPE NOT NULL, col15_9 BOX2D NOT NULL, col15_10 REGTYPE, col15_11 STRING AS (lower(CAST(col15_7 AS STRING))) VIRTUAL, col15_12 STRING AS (lower(CAST(col15_2 AS STRING))) STORED, col15_13 STRING NOT NULL AS (lower(col15_5)) VIRTUAL, col15_14 DECIMAL NOT NULL AS (abs(col15_6)) VIRTUAL, PRIMARY KEY (col15_8 ASC, col15_5, col15_4 ASC), INDEX (col15_8 ASC, col15_1 DESC), INDEX (col15_5) PARTITION BY LIST (col15_5) (PARTITION table15_part_0 VALUES IN (('``':::STRING,), ('"':::STRING,), (',':::STRING,), (e'\x1d':::STRING,), (e'\x02':::STRING,), ('J':::STRING,), ('L':::STRING,), ('m':::STRING,)), PARTITION table15_part_1 VALUES IN (('~':::STRING,), ('f':::STRING,), (' ':::STRING,), ('{':::STRING,), (e'\x12':::STRING,), (e'\x05':::STRING,), ('M':::STRING,), ('a':::STRING,)), PARTITION table15_part_2 VALUES IN (('#':::STRING,), ('e':::STRING,), ('[':::STRING,), ('F':::STRING,), ('V':::STRING,), (e'\x17':::STRING,)), PARTITION table15_part_3 VALUES IN ((e'\U00002603':::STRING,), ('G':::STRING,), ('E':::STRING,), ('R':::STRING,), ('/':::STRING,), (e'\x14':::STRING,), ('-':::STRING,)), PARTITION "DEFAULT" VALUES IN ((DEFAULT,))) NOT VISIBLE, INDEX (col15_5 DESC), INDEX (col15_9 ASC) STORING (col15_1, col15_6, col15_10), UNIQUE (lower(CAST(col15_9 AS STRING)) ASC, col15_1), UNIQUE (col15_9, col15_10 DESC) STORING (col15_1, col15_7))
        found first output
        "CREATE TABLE public.table15 (col15_0 VECTOR NULL, col15_1 INTERVAL NOT NULL, col15_2 TSQUERY NULL, col15_3 BIT(30) NULL, col15_4 FLOAT4[] NOT NULL, col15_5 BPCHAR NOT NULL, col15_6 DECIMAL NOT NULL, col15_7 VECTOR NULL, col15_8 REGTYPE NOT NULL, col15_9 BOX2D NOT NULL, col15_10 REGTYPE NULL, col15_11 STRING NULL AS (lower(CAST(col15_7 AS STRING))) VIRTUAL, col15_12 STRING NULL AS (lower(CAST(col15_2 AS STRING))) STORED, col15_13 STRING NOT NULL AS (lower(col15_5)) VIRTUAL, col15_14 DECIMAL NOT NULL AS (abs(col15_6)) VIRTUAL, CONSTRAINT table15_pkey PRIMARY KEY (col15_8 ASC, col15_5 ASC, col15_4 ASC), INDEX table15_col15_8_col15_1_idx (col15_8 ASC, col15_1 DESC), INDEX table15_col15_5_idx (col15_5 ASC) PARTITION BY LIST (col15_5) (PARTITION table15_part_0 VALUES IN (('``'), ('\"'), (','), (e'\\x1d'), (e'\\x02'), ('J'), ('L'), ('m')), PARTITION table15_part_1 VALUES IN (('~'), ('f'), (' '), ('{'), (e'\\x12'), (e'\\x05'), ('M'), ('a')), PARTITION table15_part_2 VALUES IN (('#'), ('e'), ('['), ('F'), ('V'), (e'\\x17')), PARTITION table15_part_3 VALUES IN ((e'\\U00002603'), ('G'), ('E'), ('R'), ('/'), (e'\\x14'), ('-')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT))) NOT VISIBLE, INDEX table15_col15_5_idx1 (col15_5 DESC), INDEX table15_col15_9_idx (col15_9 ASC) STORING (col15_1, col15_6, col15_10), CONSTRAINT table15_expr_col15_1_key UNIQUE (lower(CAST(col15_9 AS STRING)) ASC, col15_1 ASC), CONSTRAINT table15_col15_9_col15_10_key UNIQUE (col15_9 ASC, col15_10 DESC) STORING (col15_1, col15_7))"
        but second output
        "CREATE TABLE public.table15 (col15_0 VECTOR NULL, col15_1 INTERVAL NOT NULL, col15_2 TSQUERY NULL, col15_3 BIT(30) NULL, col15_4 FLOAT4[] NOT NULL, col15_5 BPCHAR NOT NULL, col15_6 DECIMAL NOT NULL, col15_7 VECTOR NULL, col15_8 REGTYPE NOT NULL, col15_9 BOX2D NOT NULL, col15_10 REGTYPE NULL, col15_11 STRING NULL AS (lower(CAST(col15_7 AS STRING))) VIRTUAL, col15_12 STRING NULL AS (lower(CAST(col15_2 AS STRING))) STORED, col15_13 STRING NOT NULL AS (lower(col15_5)) VIRTUAL, col15_14 DECIMAL NOT NULL AS (abs(col15_6)) VIRTUAL, CONSTRAINT table15_pkey PRIMARY KEY (col15_8 ASC, col15_5 ASC, col15_4 ASC), INDEX table15_col15_8_col15_1_idx (col15_8 ASC, col15_1 DESC), INDEX table15_col15_5_idx (col15_5 ASC) PARTITION BY LIST (col15_5) (PARTITION table15_part_0 VALUES IN (('``'), ('\"'), (','), (e'\\x1d'), (e'\\x02'), ('J'), ('L'), ('m')), PARTITION table15_part_1 VALUES IN (('~'), ('f'), (''), ('{'), (e'\\x12'), (e'\\x05'), ('M'), ('a')), PARTITION table15_part_2 VALUES IN (('#'), ('e'), ('['), ('F'), ('V'), (e'\\x17')), PARTITION table15_part_3 VALUES IN ((e'\\U00002603'), ('G'), ('E'), ('R'), ('/'), (e'\\x14'), ('-')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT))) NOT VISIBLE, INDEX table15_col15_5_idx1 (col15_5 DESC), INDEX table15_col15_9_idx (col15_9 ASC) STORING (col15_1, col15_6, col15_10), CONSTRAINT table15_expr_col15_1_key UNIQUE (lower(CAST(col15_9 AS STRING)) ASC, col15_1 ASC), CONSTRAINT table15_col15_9_col15_10_key UNIQUE (col15_9 ASC, col15_10 DESC) STORING (col15_1, col15_7))"
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema3393979339
--- FAIL: TestCreateRandomSchema (1.32s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=3</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://tanzanite.cluster.engflow.com/invocations/default/0abead4a-c05c-4184-a5d9-cf625e3d4f32?testReportRun=2&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvc3FsL3Rlc3RzOnRlc3RzX3Rlc3Q=) on master @ [8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8](https://github.com/cockroachdb/cockroach/commits/8f5366d09e6cf2144ca43f9cdda7e1128a13fbf8):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCreateRandomSchema2548273610
    test_log_scope.go:76: use -show-logs to present logs inline
    random_schema_test.go:107: for input statement
        CREATE TABLE table5 (col5_0 FLOAT8 NOT NULL, col5_1 STRING NOT NULL, col5_2 BPCHAR, col5_3 TSVECTOR NOT NULL, col5_4 BIT(40) NOT NULL, col5_5 FLOAT8 NOT NULL, col5_6 PG_LSN, col5_7 BOX2D NULL, col5_8 DATE NOT NULL, col5_9 VARCHAR NULL, col5_10 GEOGRAPHY NOT NULL, col5_11 INTERVAL NOT NULL, col5_12 STRING NOT NULL AS (CASE WHEN col5_11 IS NULL THEN '':::STRING ELSE e'\x15\x03+fw\x18&':::STRING END) VIRTUAL, col5_13 STRING AS (lower(col5_2)) STORED, col5_14 FLOAT8 NOT NULL AS (col5_5 + col5_0) STORED, col5_15 FLOAT8 NOT NULL AS (col5_0 + col5_5) STORED, col5_16 FLOAT8 NOT NULL AS (col5_0 + col5_5) STORED, UNIQUE (col5_0 DESC), INDEX (col5_1 ASC, (col5_0 + col5_5) DESC) STORING (col5_3, col5_4, col5_5, col5_6, col5_7, col5_8, col5_9, col5_11, col5_13, col5_15), UNIQUE (lower(CAST(col5_7 AS STRING))), INDEX (col5_2) STORING (col5_1, col5_3, col5_4, col5_7, col5_8, col5_10, col5_11, col5_13, col5_14, col5_15) PARTITION BY LIST (col5_2) (PARTITION table5_part_0 VALUES IN (('Z':::STRING,), (';':::STRING,), (e'\t':::STRING,)), PARTITION table5_part_1 VALUES IN ((e'\x06':::STRING,), (e'\x17':::STRING,), ('~':::STRING,)), PARTITION table5_part_2 VALUES IN ((' ':::STRING,), ('v':::STRING,)), PARTITION table5_part_3 VALUES IN (('M':::STRING,), ('6':::STRING,), ('?':::STRING,)), PARTITION table5_part_4 VALUES IN ((e'\x04':::STRING,), ('c':::STRING,)), PARTITION table5_part_5 VALUES IN (('G':::STRING,), ('u':::STRING,), ('h':::STRING,)), PARTITION table5_part_6 VALUES IN (('m':::STRING,), ('W':::STRING,)), PARTITION table5_part_7 VALUES IN ((e'\x10':::STRING,), ('o':::STRING,))), INDEX (col5_8 ASC), UNIQUE (col5_0, col5_15, col5_6 DESC), INDEX ((col5_0 + col5_5) DESC, col5_8), UNIQUE (col5_7 DESC, col5_4, col5_6), UNIQUE (col5_0 DESC, col5_5 ASC, col5_4 DESC) STORING (col5_1, col5_3, col5_6, col5_8, col5_9, col5_10, col5_11, col5_13), FAMILY (col5_15, col5_11), FAMILY (col5_8), FAMILY (col5_13), FAMILY (col5_14, col5_16, col5_9), FAMILY (col5_4), FAMILY (col5_2), FAMILY (col5_10, col5_5), FAMILY (col5_7), FAMILY (col5_0, col5_6, col5_1), FAMILY (col5_3))
        found first output
        "CREATE TABLE public.table5 (col5_0 FLOAT8 NOT NULL, col5_1 STRING NOT NULL, col5_2 BPCHAR NULL, col5_3 TSVECTOR NOT NULL, col5_4 BIT(40) NOT NULL, col5_5 FLOAT8 NOT NULL, col5_6 PG_LSN NULL, col5_7 BOX2D NULL, col5_8 DATE NOT NULL, col5_9 VARCHAR NULL, col5_10 GEOGRAPHY NOT NULL, col5_11 INTERVAL NOT NULL, col5_12 STRING NOT NULL AS (CASE WHEN col5_11 IS NULL THEN '':::STRING ELSE e'\\x15\\x03+fw\\x18&':::STRING END) VIRTUAL, col5_13 STRING NULL AS (lower(col5_2)) STORED, col5_14 FLOAT8 NOT NULL AS (col5_5 + col5_0) STORED, col5_15 FLOAT8 NOT NULL AS (col5_0 + col5_5) STORED, col5_16 FLOAT8 NOT NULL AS (col5_0 + col5_5) STORED, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table5_pkey PRIMARY KEY (rowid ASC), CONSTRAINT table5_col5_0_key UNIQUE (col5_0 DESC), INDEX table5_col5_1_expr_idx (col5_1 ASC, (col5_0 + col5_5) DESC) STORING (col5_3, col5_4, col5_5, col5_6, col5_7, col5_8, col5_9, col5_11, col5_13, col5_15), CONSTRAINT table5_expr_key UNIQUE (lower(CAST(col5_7 AS STRING)) ASC), INDEX table5_col5_2_idx (col5_2 ASC) STORING (col5_1, col5_3, col5_4, col5_7, col5_8, col5_10, col5_11, col5_13, col5_14, col5_15) PARTITION BY LIST (col5_2) (PARTITION table5_part_0 VALUES IN (('Z'), (';'), (e'\\t')), PARTITION table5_part_1 VALUES IN ((e'\\x06'), (e'\\x17'), ('~')), PARTITION table5_part_2 VALUES IN ((' '), ('v')), PARTITION table5_part_3 VALUES IN (('M'), ('6'), ('?')), PARTITION table5_part_4 VALUES IN ((e'\\x04'), ('c')), PARTITION table5_part_5 VALUES IN (('G'), ('u'), ('h')), PARTITION table5_part_6 VALUES IN (('m'), ('W')), PARTITION table5_part_7 VALUES IN ((e'\\x10'), ('o'))), INDEX table5_col5_8_idx (col5_8 ASC), CONSTRAINT table5_col5_0_col5_15_col5_6_key UNIQUE (col5_0 ASC, col5_15 ASC, col5_6 DESC), INDEX table5_expr_col5_8_idx ((col5_0 + col5_5) DESC, col5_8 ASC), CONSTRAINT table5_col5_7_col5_4_col5_6_key UNIQUE (col5_7 DESC, col5_4 ASC, col5_6 ASC), CONSTRAINT table5_col5_0_col5_5_col5_4_key UNIQUE (col5_0 DESC, col5_5 ASC, col5_4 DESC) STORING (col5_1, col5_3, col5_6, col5_8, col5_9, col5_10, col5_11, col5_13), FAMILY fam_0_col5_15_col5_11_rowid (col5_15, col5_11, rowid), FAMILY fam_1_col5_8 (col5_8), FAMILY fam_2_col5_13 (col5_13), FAMILY fam_3_col5_14_col5_16_col5_9 (col5_14, col5_16, col5_9), FAMILY fam_4_col5_4 (col5_4), FAMILY fam_5_col5_2 (col5_2), FAMILY fam_6_col5_10_col5_5 (col5_10, col5_5), FAMILY fam_7_col5_7 (col5_7), FAMILY fam_8_col5_0_col5_6_col5_1 (col5_0, col5_6, col5_1), FAMILY fam_9_col5_3 (col5_3))"
        but second output
        "CREATE TABLE public.table5 (col5_0 FLOAT8 NOT NULL, col5_1 STRING NOT NULL, col5_2 BPCHAR NULL, col5_3 TSVECTOR NOT NULL, col5_4 BIT(40) NOT NULL, col5_5 FLOAT8 NOT NULL, col5_6 PG_LSN NULL, col5_7 BOX2D NULL, col5_8 DATE NOT NULL, col5_9 VARCHAR NULL, col5_10 GEOGRAPHY NOT NULL, col5_11 INTERVAL NOT NULL, col5_12 STRING NOT NULL AS (CASE WHEN col5_11 IS NULL THEN '':::STRING ELSE e'\\x15\\x03+fw\\x18&':::STRING END) VIRTUAL, col5_13 STRING NULL AS (lower(col5_2)) STORED, col5_14 FLOAT8 NOT NULL AS (col5_5 + col5_0) STORED, col5_15 FLOAT8 NOT NULL AS (col5_0 + col5_5) STORED, col5_16 FLOAT8 NOT NULL AS (col5_0 + col5_5) STORED, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table5_pkey PRIMARY KEY (rowid ASC), CONSTRAINT table5_col5_0_key UNIQUE (col5_0 DESC), INDEX table5_col5_1_expr_idx (col5_1 ASC, (col5_0 + col5_5) DESC) STORING (col5_3, col5_4, col5_5, col5_6, col5_7, col5_8, col5_9, col5_11, col5_13, col5_15), CONSTRAINT table5_expr_key UNIQUE (lower(CAST(col5_7 AS STRING)) ASC), INDEX table5_col5_2_idx (col5_2 ASC) STORING (col5_1, col5_3, col5_4, col5_7, col5_8, col5_10, col5_11, col5_13, col5_14, col5_15) PARTITION BY LIST (col5_2) (PARTITION table5_part_0 VALUES IN (('Z'), (';'), (e'\\t')), PARTITION table5_part_1 VALUES IN ((e'\\x06'), (e'\\x17'), ('~')), PARTITION table5_part_2 VALUES IN ((''), ('v')), PARTITION table5_part_3 VALUES IN (('M'), ('6'), ('?')), PARTITION table5_part_4 VALUES IN ((e'\\x04'), ('c')), PARTITION table5_part_5 VALUES IN (('G'), ('u'), ('h')), PARTITION table5_part_6 VALUES IN (('m'), ('W')), PARTITION table5_part_7 VALUES IN ((e'\\x10'), ('o'))), INDEX table5_col5_8_idx (col5_8 ASC), CONSTRAINT table5_col5_0_col5_15_col5_6_key UNIQUE (col5_0 ASC, col5_15 ASC, col5_6 DESC), INDEX table5_expr_col5_8_idx ((col5_0 + col5_5) DESC, col5_8 ASC), CONSTRAINT table5_col5_7_col5_4_col5_6_key UNIQUE (col5_7 DESC, col5_4 ASC, col5_6 ASC), CONSTRAINT table5_col5_0_col5_5_col5_4_key UNIQUE (col5_0 DESC, col5_5 ASC, col5_4 DESC) STORING (col5_1, col5_3, col5_6, col5_8, col5_9, col5_10, col5_11, col5_13), FAMILY fam_0_col5_15_col5_11_rowid (col5_15, col5_11, rowid), FAMILY fam_1_col5_8 (col5_8), FAMILY fam_2_col5_13 (col5_13), FAMILY fam_3_col5_14_col5_16_col5_9 (col5_14, col5_16, col5_9), FAMILY fam_4_col5_4 (col5_4), FAMILY fam_5_col5_2 (col5_2), FAMILY fam_6_col5_10_col5_5 (col5_10, col5_5), FAMILY fam_7_col5_7 (col5_7), FAMILY fam_8_col5_0_col5_6_col5_1 (col5_0, col5_6, col5_1), FAMILY fam_9_col5_3 (col5_3))"
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema2548273610
--- FAIL: TestCreateRandomSchema (6.12s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17694328?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17694328?buildTab=artifacts#/) on master @ [ce622a3a31ee0999e99ff5f62d7290a20933d302](https://github.com/cockroachdb/cockroach/commits/ce622a3a31ee0999e99ff5f62d7290a20933d302):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema3648577978
    test_log_scope.go:76: use -show-logs to present logs inline
    random_schema_test.go:107: for input statement
        CREATE TABLE table39 (col39_0 BPCHAR NOT NULL, col39_1 VARCHAR NOT NULL, col39_2 INT8 NOT NULL, col39_3 INT8 NOT NULL AS (abs(col39_2)) STORED, col39_4 STRING NOT NULL AS (lower(col39_0)) STORED, PRIMARY KEY (col39_2 DESC), INDEX (col39_3), UNIQUE (col39_3 DESC), INVERTED INDEX (col39_2 DESC, col39_0 DESC, col39_3 DESC, col39_4, col39_1 gin_trgm_ops ASC) PARTITION BY LIST (col39_2, col39_0) (PARTITION table39_part_0 VALUES IN (((-7134316014675995292):::INT8, '"':::STRING), ((-6597444966194311378):::INT8, e'\x12':::STRING), ((-3804639509982973821):::INT8, e'\'':::STRING), ((-6189766613181960086):::INT8, '{':::STRING)), PARTITION table39_part_1 VALUES IN (((-3127797974445236736):::INT8, e'\x18':::STRING), (7725094841668189699:::INT8, e'\'':::STRING), ((-385467336305425343):::INT8, e'\x16':::STRING), ((-8026281670151299239):::INT8, 'O':::STRING)), PARTITION table39_part_2 VALUES IN ((7119761786941540514:::INT8, e'\\':::STRING), (1989133254128124355:::INT8, e'\x1e':::STRING), (7791649268250843890:::INT8, 'f':::STRING), ((-7826442032096910523):::INT8, ' ':::STRING))), INDEX (col39_0 DESC, col39_2 DESC), INDEX (col39_4, col39_0 ASC, col39_3) STORING (col39_1), FAMILY (col39_4), FAMILY (col39_1), FAMILY (col39_0, col39_3, col39_2))
        found first output
        "CREATE TABLE public.table39 (col39_0 BPCHAR NOT NULL, col39_1 VARCHAR NOT NULL, col39_2 INT8 NOT NULL, col39_3 INT8 NOT NULL AS (abs(col39_2)) STORED, col39_4 STRING NOT NULL AS (lower(col39_0)) STORED, CONSTRAINT table39_pkey PRIMARY KEY (col39_2 DESC), INDEX table39_col39_3_idx (col39_3 ASC), CONSTRAINT table39_col39_3_key UNIQUE (col39_3 DESC), INVERTED INDEX table39_col39_2_col39_0_col39_3_col39_4_col39_1_idx (col39_2 DESC, col39_0 DESC, col39_3 DESC, col39_4 ASC, col39_1 gin_trgm_ops) PARTITION BY LIST (col39_2, col39_0) (PARTITION table39_part_0 VALUES IN ((-7134316014675995292, '\"'), (-6597444966194311378, e'\\x12'), (-3804639509982973821, e'\\''), (-6189766613181960086, '{')), PARTITION table39_part_1 VALUES IN ((-3127797974445236736, e'\\x18'), (7725094841668189699, e'\\''), (-385467336305425343, e'\\x16'), (-8026281670151299239, 'O')), PARTITION table39_part_2 VALUES IN ((7119761786941540514, e'\\\\'), (1989133254128124355, e'\\x1e'), (7791649268250843890, 'f'), (-7826442032096910523, ' '))), INDEX table39_col39_0_col39_2_idx (col39_0 DESC, col39_2 DESC), INDEX table39_col39_4_col39_0_col39_3_idx (col39_4 ASC, col39_0 ASC, col39_3 ASC) STORING (col39_1), FAMILY fam_0_col39_4 (col39_4), FAMILY fam_1_col39_1 (col39_1), FAMILY fam_2_col39_0_col39_3_col39_2 (col39_0, col39_3, col39_2))"
        but second output
        "CREATE TABLE public.table39 (col39_0 BPCHAR NOT NULL, col39_1 VARCHAR NOT NULL, col39_2 INT8 NOT NULL, col39_3 INT8 NOT NULL AS (abs(col39_2)) STORED, col39_4 STRING NOT NULL AS (lower(col39_0)) STORED, CONSTRAINT table39_pkey PRIMARY KEY (col39_2 DESC), INDEX table39_col39_3_idx (col39_3 ASC), CONSTRAINT table39_col39_3_key UNIQUE (col39_3 DESC), INVERTED INDEX table39_col39_2_col39_0_col39_3_col39_4_col39_1_idx (col39_2 DESC, col39_0 DESC, col39_3 DESC, col39_4 ASC, col39_1 gin_trgm_ops) PARTITION BY LIST (col39_2, col39_0) (PARTITION table39_part_0 VALUES IN ((-7134316014675995292, '\"'), (-6597444966194311378, e'\\x12'), (-3804639509982973821, e'\\''), (-6189766613181960086, '{')), PARTITION table39_part_1 VALUES IN ((-3127797974445236736, e'\\x18'), (7725094841668189699, e'\\''), (-385467336305425343, e'\\x16'), (-8026281670151299239, 'O')), PARTITION table39_part_2 VALUES IN ((7119761786941540514, e'\\\\'), (1989133254128124355, e'\\x1e'), (7791649268250843890, 'f'), (-7826442032096910523, ''))), INDEX table39_col39_0_col39_2_idx (col39_0 DESC, col39_2 DESC), INDEX table39_col39_4_col39_0_col39_3_idx (col39_4 ASC, col39_0 ASC, col39_3 ASC) STORING (col39_1), FAMILY fam_0_col39_4 (col39_4), FAMILY fam_1_col39_1 (col39_1), FAMILY fam_2_col39_0_col39_3_col39_2 (col39_0, col39_3, col39_2))"
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema3648577978
--- FAIL: TestCreateRandomSchema (4.26s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17696916?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17696916?buildTab=artifacts#/) on master @ [dcc832aa31a00d269537deaccf4417366826baa5](https://github.com/cockroachdb/cockroach/commits/dcc832aa31a00d269537deaccf4417366826baa5):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema271538300
    test_log_scope.go:76: use -show-logs to present logs inline
    random_schema_test.go:107: for input statement
        CREATE TABLE table13 (col13_0 BIT(14) NOT NULL, col13_1 OID, col13_2 INT2 NOT NULL, col13_3 INT4 NOT NULL, col13_4 BPCHAR NULL, col13_5 REGROLE, col13_6 INT8 NOT NULL, col13_7 UUID, col13_8 INT8 NOT NULL AS (col13_2 + col13_6) VIRTUAL, col13_9 STRING AS (lower(CAST(col13_1 AS STRING))) STORED, col13_10 INT8 NOT NULL AS (col13_2 + col13_6) VIRTUAL, col13_11 INT8 NOT NULL AS ((col13_6 + col13_3) + col13_2) STORED, PRIMARY KEY (col13_6), INDEX (col13_7, col13_2, col13_8, col13_3, col13_5, col13_10 ASC, lower(CAST(col13_1 AS STRING)) ASC) STORING (col13_1, col13_4), UNIQUE (col13_5 ASC, ((col13_6 + col13_2) + col13_3), col13_6 DESC, col13_3, col13_1 DESC), UNIQUE (col13_8 ASC, col13_10 DESC) PARTITION BY LIST (col13_8) (PARTITION table13_part_0 VALUES IN (((-1632637376818079773):::INT8,), (882287436063967486:::INT8,), ((-1436520516980498298):::INT8,), ((-6729810836050165052):::INT8,), ((-3551848815488151601):::INT8,), ((-1677951663896347109):::INT8,), ((-1):::INT8,)), PARTITION table13_part_1 VALUES IN (((-4919871186304588435):::INT8,), (2751997400881702956:::INT8,), (9223372036854775807:::INT8,), (2066851092563694428:::INT8,), (8910291701484536552:::INT8,), (554436738413004928:::INT8,), ((-8758253057725269811):::INT8,)), PARTITION table13_part_2 VALUES IN ((3390465744645370266:::INT8,), (275133557116162555:::INT8,), ((-532918918062520371):::INT8,), ((-4021874475553181080):::INT8,), ((-3489573579099160924):::INT8,), (8418459680003932159:::INT8,), ((-2472614309163523851):::INT8,)), PARTITION table13_part_3 VALUES IN ((1521259056379158666:::INT8,), ((-674449726918724643):::INT8,), (2519719013136864786:::INT8,), ((-4190728083900436219):::INT8,), (2147483647:::INT8,), (359804208492043120:::INT8,)), PARTITION table13_part_4 VALUES IN (((-6090897048871569873):::INT8,), (8978027465062797568:::INT8,), (127:::INT8,), (636472475041122186:::INT8,), (8082625940238125308:::INT8,), (4292723476317475078:::INT8,), ((-4088801976334041104):::INT8,)), PARTITION table13_part_5 VALUES IN (((-3851725309242387614):::INT8,), ((-5216507780320083029):::INT8,), ((-8709802786516756120):::INT8,), (1402154819575857092:::INT8,), ((-6678413208676387604):::INT8,), ((-2394676580449677327):::INT8,), (3147833088946196096:::INT8,))), UNIQUE (col13_7 DESC) STORING (col13_1, col13_3, col13_9, col13_11), INDEX (col13_6, col13_4 DESC, col13_3 DESC) PARTITION BY LIST (col13_6, col13_4) (PARTITION table13_part_0 VALUES IN ((6173923143875303592:::INT8, '|':::STRING), ((-7406926593248892823):::INT8, e'\x05':::STRING), (7942912763561528999:::INT8, ' ':::STRING)), PARTITION table13_part_1 VALUES IN ((1872459760158843845:::INT8, 'I':::STRING), ((-2185940779727841283):::INT8, '[':::STRING), (2347911064115735250:::INT8, e'\x11':::STRING)), PARTITION table13_part_2 VALUES IN (((-4799623649001674459):::INT8, e'\x14':::STRING), ((-8627337124600349610):::INT8, '2':::STRING), ((-7592894987965842239):::INT8, e'\x07':::STRING)), PARTITION table13_part_3 VALUES IN ((1416704715862732480:::INT8, e'\'':::STRING), ((-3535443284828648003):::INT8, '[':::STRING), ((-7842859665229645392):::INT8, 'V':::STRING)), PARTITION table13_part_4 VALUES IN ((2893698198563655555:::INT8, 'N':::STRING), ((-2413185517360979306):::INT8, 'U':::STRING), ((-4624301536670250492):::INT8, e'\x16':::STRING)), PARTITION table13_part_5 VALUES IN ((1741776875712814063:::INT8, 'A':::STRING), (6389543513844215097:::INT8, 'Y':::STRING), (9085166856261088810:::INT8, e'\x0b':::STRING)), PARTITION table13_part_6 VALUES IN ((6512687952921006952:::INT8, 'U':::STRING), ((-635297252090650300):::INT8, '+':::STRING), (2147483647:::INT8, 'L':::STRING))), INDEX (col13_0 ASC) STORING (col13_1, col13_3, col13_5) NOT VISIBLE, UNIQUE (col13_1), UNIQUE (col13_7, col13_0 DESC, (col13_3 + col13_2)) STORING (col13_1, col13_2, col13_3, col13_4))
        found first output
        "CREATE TABLE public.table13 (col13_0 BIT(14) NOT NULL, col13_1 OID NULL, col13_2 INT2 NOT NULL, col13_3 INT4 NOT NULL, col13_4 BPCHAR NULL, col13_5 REGROLE NULL, col13_6 INT8 NOT NULL, col13_7 UUID NULL, col13_8 INT8 NOT NULL AS (col13_2 + col13_6) VIRTUAL, col13_9 STRING NULL AS (lower(CAST(col13_1 AS STRING))) STORED, col13_10 INT8 NOT NULL AS (col13_2 + col13_6) VIRTUAL, col13_11 INT8 NOT NULL AS ((col13_6 + col13_3) + col13_2) STORED, CONSTRAINT table13_pkey PRIMARY KEY (col13_6 ASC), INDEX table13_col13_7_col13_2_col13_8_col13_3_col13_5_col13_10_expr_idx (col13_7 ASC, col13_2 ASC, col13_8 ASC, col13_3 ASC, col13_5 ASC, col13_10 ASC, lower(CAST(col13_1 AS STRING)) ASC) STORING (col13_1, col13_4), CONSTRAINT table13_col13_5_expr_col13_6_col13_3_col13_1_key UNIQUE (col13_5 ASC, ((col13_6 + col13_2) + col13_3) ASC, col13_6 DESC, col13_3 ASC, col13_1 DESC), CONSTRAINT table13_col13_8_col13_10_key UNIQUE (col13_8 ASC, col13_10 DESC) PARTITION BY LIST (col13_8) (PARTITION table13_part_0 VALUES IN ((-1632637376818079773), (882287436063967486), (-1436520516980498298), (-6729810836050165052), (-3551848815488151601), (-1677951663896347109), (-1)), PARTITION table13_part_1 VALUES IN ((-4919871186304588435), (2751997400881702956), (9223372036854775807), (2066851092563694428), (8910291701484536552), (554436738413004928), (-8758253057725269811)), PARTITION table13_part_2 VALUES IN ((3390465744645370266), (275133557116162555), (-532918918062520371), (-4021874475553181080), (-3489573579099160924), (8418459680003932159), (-2472614309163523851)), PARTITION table13_part_3 VALUES IN ((1521259056379158666), (-674449726918724643), (2519719013136864786), (-4190728083900436219), (2147483647), (359804208492043120)), PARTITION table13_part_4 VALUES IN ((-6090897048871569873), (8978027465062797568), (127), (636472475041122186), (8082625940238125308), (4292723476317475078), (-4088801976334041104)), PARTITION table13_part_5 VALUES IN ((-3851725309242387614), (-5216507780320083029), (-8709802786516756120), (1402154819575857092), (-6678413208676387604), (-2394676580449677327), (3147833088946196096))), CONSTRAINT table13_col13_7_key UNIQUE (col13_7 DESC) STORING (col13_1, col13_3, col13_9, col13_11), INDEX table13_col13_6_col13_4_col13_3_idx (col13_6 ASC, col13_4 DESC, col13_3 DESC) PARTITION BY LIST (col13_6, col13_4) (PARTITION table13_part_0 VALUES IN ((6173923143875303592, '|'), (-7406926593248892823, e'\\x05'), (7942912763561528999, ' ')), PARTITION table13_part_1 VALUES IN ((1872459760158843845, 'I'), (-2185940779727841283, '['), (2347911064115735250, e'\\x11')), PARTITION table13_part_2 VALUES IN ((-4799623649001674459, e'\\x14'), (-8627337124600349610, '2'), (-7592894987965842239, e'\\x07')), PARTITION table13_part_3 VALUES IN ((1416704715862732480, e'\\''), (-3535443284828648003, '['), (-7842859665229645392, 'V')), PARTITION table13_part_4 VALUES IN ((2893698198563655555, 'N'), (-2413185517360979306, 'U'), (-4624301536670250492, e'\\x16')), PARTITION table13_part_5 VALUES IN ((1741776875712814063, 'A'), (6389543513844215097, 'Y'), (9085166856261088810, e'\\x0b')), PARTITION table13_part_6 VALUES IN ((6512687952921006952, 'U'), (-635297252090650300, '+'), (2147483647, 'L'))), INDEX table13_col13_0_idx (col13_0 ASC) STORING (col13_1, col13_3, col13_5) NOT VISIBLE, CONSTRAINT table13_col13_1_key UNIQUE (col13_1 ASC), CONSTRAINT table13_col13_7_col13_0_expr_key UNIQUE (col13_7 ASC, col13_0 DESC, (col13_3 + col13_2) ASC) STORING (col13_1, col13_2, col13_3, col13_4))"
        but second output
        "CREATE TABLE public.table13 (col13_0 BIT(14) NOT NULL, col13_1 OID NULL, col13_2 INT2 NOT NULL, col13_3 INT4 NOT NULL, col13_4 BPCHAR NULL, col13_5 REGROLE NULL, col13_6 INT8 NOT NULL, col13_7 UUID NULL, col13_8 INT8 NOT NULL AS (col13_2 + col13_6) VIRTUAL, col13_9 STRING NULL AS (lower(CAST(col13_1 AS STRING))) STORED, col13_10 INT8 NOT NULL AS (col13_2 + col13_6) VIRTUAL, col13_11 INT8 NOT NULL AS ((col13_6 + col13_3) + col13_2) STORED, CONSTRAINT table13_pkey PRIMARY KEY (col13_6 ASC), INDEX table13_col13_7_col13_2_col13_8_col13_3_col13_5_col13_10_expr_idx (col13_7 ASC, col13_2 ASC, col13_8 ASC, col13_3 ASC, col13_5 ASC, col13_10 ASC, lower(CAST(col13_1 AS STRING)) ASC) STORING (col13_1, col13_4), CONSTRAINT table13_col13_5_expr_col13_6_col13_3_col13_1_key UNIQUE (col13_5 ASC, ((col13_6 + col13_2) + col13_3) ASC, col13_6 DESC, col13_3 ASC, col13_1 DESC), CONSTRAINT table13_col13_8_col13_10_key UNIQUE (col13_8 ASC, col13_10 DESC) PARTITION BY LIST (col13_8) (PARTITION table13_part_0 VALUES IN ((-1632637376818079773), (882287436063967486), (-1436520516980498298), (-6729810836050165052), (-3551848815488151601), (-1677951663896347109), (-1)), PARTITION table13_part_1 VALUES IN ((-4919871186304588435), (2751997400881702956), (9223372036854775807), (2066851092563694428), (8910291701484536552), (554436738413004928), (-8758253057725269811)), PARTITION table13_part_2 VALUES IN ((3390465744645370266), (275133557116162555), (-532918918062520371), (-4021874475553181080), (-3489573579099160924), (8418459680003932159), (-2472614309163523851)), PARTITION table13_part_3 VALUES IN ((1521259056379158666), (-674449726918724643), (2519719013136864786), (-4190728083900436219), (2147483647), (359804208492043120)), PARTITION table13_part_4 VALUES IN ((-6090897048871569873), (8978027465062797568), (127), (636472475041122186), (8082625940238125308), (4292723476317475078), (-4088801976334041104)), PARTITION table13_part_5 VALUES IN ((-3851725309242387614), (-5216507780320083029), (-8709802786516756120), (1402154819575857092), (-6678413208676387604), (-2394676580449677327), (3147833088946196096))), CONSTRAINT table13_col13_7_key UNIQUE (col13_7 DESC) STORING (col13_1, col13_3, col13_9, col13_11), INDEX table13_col13_6_col13_4_col13_3_idx (col13_6 ASC, col13_4 DESC, col13_3 DESC) PARTITION BY LIST (col13_6, col13_4) (PARTITION table13_part_0 VALUES IN ((6173923143875303592, '|'), (-7406926593248892823, e'\\x05'), (7942912763561528999, '')), PARTITION table13_part_1 VALUES IN ((1872459760158843845, 'I'), (-2185940779727841283, '['), (2347911064115735250, e'\\x11')), PARTITION table13_part_2 VALUES IN ((-4799623649001674459, e'\\x14'), (-8627337124600349610, '2'), (-7592894987965842239, e'\\x07')), PARTITION table13_part_3 VALUES IN ((1416704715862732480, e'\\''), (-3535443284828648003, '['), (-7842859665229645392, 'V')), PARTITION table13_part_4 VALUES IN ((2893698198563655555, 'N'), (-2413185517360979306, 'U'), (-4624301536670250492, e'\\x16')), PARTITION table13_part_5 VALUES IN ((1741776875712814063, 'A'), (6389543513844215097, 'Y'), (9085166856261088810, e'\\x0b')), PARTITION table13_part_6 VALUES IN ((6512687952921006952, 'U'), (-635297252090650300, '+'), (2147483647, 'L'))), INDEX table13_col13_0_idx (col13_0 ASC) STORING (col13_1, col13_3, col13_5) NOT VISIBLE, CONSTRAINT table13_col13_1_key UNIQUE (col13_1 ASC), CONSTRAINT table13_col13_7_col13_0_expr_key UNIQUE (col13_7 ASC, col13_0 DESC, (col13_3 + col13_2) ASC) STORING (col13_1, col13_2, col13_3, col13_4))"
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema271538300
--- FAIL: TestCreateRandomSchema (1.43s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17700733?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17700733?buildTab=artifacts#/) on master @ [2f324cc1bbd12dd54018805a0ddbf2b2b8a98e18](https://github.com/cockroachdb/cockroach/commits/2f324cc1bbd12dd54018805a0ddbf2b2b8a98e18):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema2411824109
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:107: for input statement
        CREATE TABLE table4 (col4_0 INT2 NOT NULL, col4_1 BIT(7) NOT NULL, col4_2 REGTYPE, col4_3 TIMETZ NOT NULL, col4_4 BPCHAR NOT NULL, col4_5 STRING NOT NULL AS (lower(CAST(col4_1 AS STRING))) VIRTUAL, col4_6 INT2 NOT NULL AS (abs(col4_0)) STORED, PRIMARY KEY (col4_6 DESC), UNIQUE (col4_0 DESC, col4_1 DESC), INDEX (col4_5 DESC, col4_4 DESC, col4_2 ASC) STORING (col4_3) PARTITION BY LIST (col4_5, col4_4, col4_2) (PARTITION table4_part_0 VALUES IN ((e'\x19)r[':::STRING, '<':::STRING, 108553386:::REGTYPE), ('N8J':::STRING, 'w':::STRING, 2613225572:::REGTYPE), (e'\x0f':::STRING, e'\x05':::STRING, 2582555660:::REGTYPE), ('':::STRING, '=':::STRING, 0:::OID), ('>f#3#-<7':::STRING, 'c':::STRING, 2058350102:::REGTYPE), (e'\x00':::STRING, 'j':::STRING, 823818057:::REGTYPE), (e'zVC\x18\b\x11aE':::STRING, '8':::STRING, 18992128:::REGTYPE), (e'U\x05E\x11mOfW(':::STRING, 'R':::STRING, 3203610261:::REGTYPE), (e'\bq':::STRING, '/':::STRING, 689947832:::REGTYPE)), PARTITION table4_part_1 VALUES IN (('X':::STRING, e'\'':::STRING, 2674418544:::REGTYPE), (e'7+o5\n':::STRING, '+':::STRING, 828567962:::REGTYPE), (e'o\x1eS':::STRING, e'\b':::STRING, 477264618:::REGTYPE), ('};:Q&':::STRING, '4':::STRING, 4203767263:::REGTYPE), ('':::STRING, 'Y':::STRING, 1968799882:::REGTYPE), (e'HM\x0b\x17z+':::STRING, 'h':::STRING, 4045047171:::REGTYPE), ('~Q~:``zm``':::STRING, 'k':::STRING, 3927205312:::REGTYPE), ('':::STRING, 's':::STRING, 0:::OID), (e'2:8\x1c':::STRING, '2':::STRING, 1679598924:::REGTYPE)), PARTITION table4_part_2 VALUES IN ((e'; Y\x19M':::STRING, e'\f':::STRING, 4035714097:::REGTYPE), (e'\x15':::STRING, e'\'':::STRING, 2510560863:::REGTYPE), (e'\x1b\x02ymR\x13S\x04':::STRING, '<':::STRING, 2417488235:::REGTYPE), (e'k*\tS':::STRING, 'j':::STRING, 4195589227:::REGTYPE), (e'\x13\x19kWe':::STRING, 'e':::STRING, 903597985:::REGTYPE), ('wN:,SeNG':::STRING, 'X':::STRING, 4079306283:::REGTYPE), ('':::STRING, '&':::STRING, 1488840225:::REGTYPE), ('t0':::STRING, '%':::STRING, 3562745105:::REGTYPE), (e'\U00002603':::STRING, '^':::STRING, 1193010910:::REGTYPE)), PARTITION table4_part_3 VALUES IN ((e'\x7fYe':::STRING, '7':::STRING, 3839609211:::REGTYPE), (e'I5kv\n\x1d':::STRING, ')':::STRING, 3779786321:::REGTYPE), ('X':::STRING, e'\x19':::STRING, 0:::OID), (e'^;T5~\x11/\\5':::STRING, '[':::STRING, 1078283111:::REGTYPE), ('oW':::STRING, '^':::STRING, 152371999:::REGTYPE), ('':::STRING, '/':::STRING, 1571936057:::REGTYPE), (e'\x0f\x11':::STRING, e'\t':::STRING, 2693635799:::REGTYPE), (e'tKxPx\x7fP\rg':::STRING, e'\x1b':::STRING, 214166676:::REGTYPE), (e'\x069#\\9\x11v}O':::STRING, '_':::STRING, 1091130754:::REGTYPE)), PARTITION table4_part_4 VALUES IN ((e'#U\x17\x1b\x12#':::STRING, '7':::STRING, 0:::OID), ('jL:;':::STRING, 'p':::STRING, 2123810219:::REGTYPE), ('qP':::STRING, '/':::STRING, 3104581437:::REGTYPE), (e'Q|i4\x03+':::STRING, e'\x13':::STRING, 3266494055:::REGTYPE), (e'\x7f@\x0fLn':::STRING, e'\x0b':::STRING, 2244375423:::REGTYPE), (e'?\n  \x01T"T\b':::STRING, 'y':::STRING, 3501975016:::REGTYPE), (e'\t\x068xl':::STRING, '"':::STRING, 2271730306:::REGTYPE), ('v}':::STRING, 'a':::STRING, 3260316571:::REGTYPE), (e'sL\t2q\x02':::STRING, e'\x00':::STRING, 2101439645:::REGTYPE)), PARTITION table4_part_5 VALUES IN (('R0':::STRING, e'\x11':::STRING, 673358489:::REGTYPE), (e'w\x17':::STRING, 'A':::STRING, 3726715316:::REGTYPE), (e'\x01\x1c':::STRING, '/':::STRING, 0:::OID), ('PG<':::STRING, 'S':::STRING, 3624431705:::REGTYPE), (e'\x19\x10\x13':::STRING, 'g':::STRING, 0:::OID), (e'\x10\x1f\x1d".m':::STRING, e'\x04':::STRING, 1547080936:::REGTYPE), ('$':::STRING, e'\x00':::STRING, 839412529:::REGTYPE), (e'O\b\x12"':::STRING, 'O':::STRING, 0:::OID), (e'\x0f\x10t\x04':::STRING, 'S':::STRING, 2096027704:::REGTYPE)), PARTITION table4_part_6 VALUES IN (('g':::STRING, 'j':::STRING, 2071028640:::REGTYPE), ('':::STRING, 'c':::STRING, 2405874649:::REGTYPE), (e'\x16\x139$|^q':::STRING, 'a':::STRING, 2089154318:::REGTYPE), (e'\x02\bT':::STRING, '^':::STRING, 333603208:::REGTYPE), ('{':::STRING, 'i':::STRING, 3301501767:::REGTYPE), ('{]!}':::STRING, 'r':::STRING, 4095591002:::REGTYPE), (e'\x1aX\x1cC':::STRING, e'\x1a':::STRING, 2609292279:::REGTYPE), (')dO':::STRING, 'c':::STRING, 0:::OID), (e'\x01\x0b\x0eBC4':::STRING, 'J':::STRING, 0:::OID)), PARTITION table4_part_7 VALUES IN ((e'\U00002603':::STRING, e'\t':::STRING, 2141991755:::REGTYPE), ('s':::STRING, 's':::STRING, 846284978:::REGTYPE), (e'.\fXS':::STRING, e'\x0f':::STRING, 997777549:::REGTYPE), ('':::STRING, 'T':::STRING, 3798631380:::REGTYPE), (e'+q\x10\x12R':::STRING, e'\x19':::STRING, 3928399494:::REGTYPE), (e'CJ$K\x02;':::STRING, 'O':::STRING, 3870085288:::REGTYPE), ('X':::STRING, '"':::STRING, 3573195198:::REGTYPE), ('oF:':::STRING, '$':::STRING, 4168147292:::REGTYPE), (e'(\\':::STRING, 'E':::STRING, 0:::OID)), PARTITION table4_part_8 VALUES IN ((e's&\x02i\x12Q``x\x02':::STRING, 'h':::STRING, 3446575851:::REGTYPE), ('!i-':::STRING, e'\x02':::STRING, 3597092236:::REGTYPE), (e'\x10':::STRING, e'\'':::STRING, 1406087395:::REGTYPE), ('':::STRING, 'c':::STRING, 88705775:::REGTYPE), ('':::STRING, 'r':::STRING, 3708936451:::REGTYPE), (e'``x\x05P':::STRING, '1':::STRING, 0:::OID), (e'\x10Eh\x06g':::STRING, e'\x19':::STRING, 137395561:::REGTYPE), (e'z\x13L\x05~':::STRING, 'd':::STRING, 1035604431:::REGTYPE), (e'\f.':::STRING, 'G':::STRING, 2134835641:::REGTYPE)), PARTITION table4_part_9 VALUES IN ((e'.l\x03':::STRING, '':::STRING, 4284958193:::REGTYPE), (e'$WA\x10\x17&a\x16':::STRING, e'\x11':::STRING, 756209360:::REGTYPE), (e'+nux\x1d\x1b':::STRING, '{':::STRING, 2995197263:::REGTYPE), (e'$c\n':::STRING, e'\x01':::STRING, 2953414521:::REGTYPE), (e'\x03``]C4g5':::STRING, '3':::STRING, 3919240825:::REGTYPE), (e'\x01xU\x06s*':::STRING, e'\t':::STRING, 0:::OID), ('[n':::STRING, 'J':::STRING, 94254714:::REGTYPE), (e'\x1b\x14\x1b':::STRING, ' ':::STRING, 3964974415:::REGTYPE), ('c"vG:> "':::STRING, e'\'':::STRING, 4039603590:::REGTYPE)), PARTITION "DEFAULT" VALUES IN ((DEFAULT, DEFAULT, DEFAULT))) NOT VISIBLE, INDEX (col4_5 DESC) NOT VISIBLE, INDEX (abs(col4_0) ASC) NOT VISIBLE, UNIQUE (col4_3 DESC, col4_0 DESC, col4_1 DESC), UNIQUE (col4_4, col4_3 ASC, col4_0 DESC), INDEX (lower(CAST(col4_3 AS STRING)) DESC, col4_4), INDEX (col4_2 ASC, col4_0 DESC) NOT VISIBLE)
        found first output
        "CREATE TABLE public.table4 (col4_0 INT2 NOT NULL, col4_1 BIT(7) NOT NULL, col4_2 REGTYPE NULL, col4_3 TIMETZ NOT NULL, col4_4 BPCHAR NOT NULL, col4_5 STRING NOT NULL AS (lower(CAST(col4_1 AS STRING))) VIRTUAL, col4_6 INT2 NOT NULL AS (abs(col4_0)) STORED, CONSTRAINT table4_pkey PRIMARY KEY (col4_6 DESC), CONSTRAINT table4_col4_0_col4_1_key UNIQUE (col4_0 DESC, col4_1 DESC), INDEX table4_col4_5_col4_4_col4_2_idx (col4_5 DESC, col4_4 DESC, col4_2 ASC) STORING (col4_3) PARTITION BY LIST (col4_5, col4_4, col4_2) (PARTITION table4_part_0 VALUES IN ((e'\\x19)r[', '<', 108553386), ('N8J', 'w', 2613225572), (e'\\x0f', e'\\x05', 2582555660), ('', '=', 0), ('>f#3#-<7', 'c', 2058350102), (e'\\x00', 'j', 823818057), (e'zVC\\x18\\b\\x11aE', '8', 18992128), (e'U\\x05E\\x11mOfW(', 'R', 3203610261), (e'\\bq', '/', 689947832)), PARTITION table4_part_1 VALUES IN (('X', e'\\'', 2674418544), (e'7+o5\\n', '+', 828567962), (e'o\\x1eS', e'\\b', 477264618), ('};:Q&', '4', 4203767263), ('', 'Y', 1968799882), (e'HM\\x0b\\x17z+', 'h', 4045047171), ('~Q~:``zm``', 'k', 3927205312), ('', 's', 0), (e'2:8\\x1c', '2', 1679598924)), PARTITION table4_part_2 VALUES IN ((e'; Y\\x19M', e'\\f', 4035714097), (e'\\x15', e'\\'', 2510560863), (e'\\x1b\\x02ymR\\x13S\\x04', '<', 2417488235), (e'k*\\tS', 'j', 4195589227), (e'\\x13\\x19kWe', 'e', 903597985), ('wN:,SeNG', 'X', 4079306283), ('', '&', 1488840225), ('t0', '%', 3562745105), (e'\\U00002603', '^', 1193010910)), PARTITION table4_part_3 VALUES IN ((e'\\x7fYe', '7', 3839609211), (e'I5kv\\n\\x1d', ')', 3779786321), ('X', e'\\x19', 0), (e'^;T5~\\x11/\\\\5', '[', 1078283111), ('oW', '^', 152371999), ('', '/', 1571936057), (e'\\x0f\\x11', e'\\t', 2693635799), (e'tKxPx\\x7fP\\rg', e'\\x1b', 214166676), (e'\\x069#\\\\9\\x11v}O', '_', 1091130754)), PARTITION table4_part_4 VALUES IN ((e'#U\\x17\\x1b\\x12#', '7', 0), ('jL:;', 'p', 2123810219), ('qP', '/', 3104581437), (e'Q|i4\\x03+', e'\\x13', 3266494055), (e'\\x7f@\\x0fLn', e'\\x0b', 2244375423), (e'?\\n  \\x01T\"T\\b', 'y', 3501975016), (e'\\t\\x068xl', '\"', 2271730306), ('v}', 'a', 3260316571), (e'sL\\t2q\\x02', e'\\x00', 2101439645)), PARTITION table4_part_5 VALUES IN (('R0', e'\\x11', 673358489), (e'w\\x17', 'A', 3726715316), (e'\\x01\\x1c', '/', 0), ('PG<', 'S', 3624431705), (e'\\x19\\x10\\x13', 'g', 0), (e'\\x10\\x1f\\x1d\".m', e'\\x04', 1547080936), ('$', e'\\x00', 839412529), (e'O\\b\\x12\"', 'O', 0), (e'\\x0f\\x10t\\x04', 'S', 2096027704)), PARTITION table4_part_6 VALUES IN (('g', 'j', 2071028640), ('', 'c', 2405874649), (e'\\x16\\x139$|^q', 'a', 2089154318), (e'\\x02\\bT', '^', 333603208), ('{', 'i', 3301501767), ('{]!}', 'r', 4095591002), (e'\\x1aX\\x1cC', e'\\x1a', 2609292279), (')dO', 'c', 0), (e'\\x01\\x0b\\x0eBC4', 'J', 0)), PARTITION table4_part_7 VALUES IN ((e'\\U00002603', e'\\t', 2141991755), ('s', 's', 846284978), (e'.\\fXS', e'\\x0f', 997777549), ('', 'T', 3798631380), (e'+q\\x10\\x12R', e'\\x19', 3928399494), (e'CJ$K\\x02;', 'O', 3870085288), ('X', '\"', 3573195198), ('oF:', '$', 4168147292), (e'(\\\\', 'E', 0)), PARTITION table4_part_8 VALUES IN ((e's&\\x02i\\x12Q``x\\x02', 'h', 3446575851), ('!i-', e'\\x02', 3597092236), (e'\\x10', e'\\'', 1406087395), ('', 'c', 88705775), ('', 'r', 3708936451), (e'``x\\x05P', '1', 0), (e'\\x10Eh\\x06g', e'\\x19', 137395561), (e'z\\x13L\\x05~', 'd', 1035604431), (e'\\f.', 'G', 2134835641)), PARTITION table4_part_9 VALUES IN ((e'.l\\x03', '', 4284958193), (e'$WA\\x10\\x17&a\\x16', e'\\x11', 756209360), (e'+nux\\x1d\\x1b', '{', 2995197263), (e'$c\\n', e'\\x01', 2953414521), (e'\\x03``]C4g5', '3', 3919240825), (e'\\x01xU\\x06s*', e'\\t', 0), ('[n', 'J', 94254714), (e'\\x1b\\x14\\x1b', ' ', 3964974415), ('c\"vG:> \"', e'\\'', 4039603590)), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT, DEFAULT))) NOT VISIBLE, INDEX table4_col4_5_idx (col4_5 DESC) NOT VISIBLE, INDEX table4_expr_idx (abs(col4_0) ASC) NOT VISIBLE, CONSTRAINT table4_col4_3_col4_0_col4_1_key UNIQUE (col4_3 DESC, col4_0 DESC, col4_1 DESC), CONSTRAINT table4_col4_4_col4_3_col4_0_key UNIQUE (col4_4 ASC, col4_3 ASC, col4_0 DESC), INDEX table4_expr_col4_4_idx (lower(CAST(col4_3 AS STRING)) DESC, col4_4 ASC), INDEX table4_col4_2_col4_0_idx (col4_2 ASC, col4_0 DESC) NOT VISIBLE)"
        but second output
        "CREATE TABLE public.table4 (col4_0 INT2 NOT NULL, col4_1 BIT(7) NOT NULL, col4_2 REGTYPE NULL, col4_3 TIMETZ NOT NULL, col4_4 BPCHAR NOT NULL, col4_5 STRING NOT NULL AS (lower(CAST(col4_1 AS STRING))) VIRTUAL, col4_6 INT2 NOT NULL AS (abs(col4_0)) STORED, CONSTRAINT table4_pkey PRIMARY KEY (col4_6 DESC), CONSTRAINT table4_col4_0_col4_1_key UNIQUE (col4_0 DESC, col4_1 DESC), INDEX table4_col4_5_col4_4_col4_2_idx (col4_5 DESC, col4_4 DESC, col4_2 ASC) STORING (col4_3) PARTITION BY LIST (col4_5, col4_4, col4_2) (PARTITION table4_part_0 VALUES IN ((e'\\x19)r[', '<', 108553386), ('N8J', 'w', 2613225572), (e'\\x0f', e'\\x05', 2582555660), ('', '=', 0), ('>f#3#-<7', 'c', 2058350102), (e'\\x00', 'j', 823818057), (e'zVC\\x18\\b\\x11aE', '8', 18992128), (e'U\\x05E\\x11mOfW(', 'R', 3203610261), (e'\\bq', '/', 689947832)), PARTITION table4_part_1 VALUES IN (('X', e'\\'', 2674418544), (e'7+o5\\n', '+', 828567962), (e'o\\x1eS', e'\\b', 477264618), ('};:Q&', '4', 4203767263), ('', 'Y', 1968799882), (e'HM\\x0b\\x17z+', 'h', 4045047171), ('~Q~:``zm``', 'k', 3927205312), ('', 's', 0), (e'2:8\\x1c', '2', 1679598924)), PARTITION table4_part_2 VALUES IN ((e'; Y\\x19M', e'\\f', 4035714097), (e'\\x15', e'\\'', 2510560863), (e'\\x1b\\x02ymR\\x13S\\x04', '<', 2417488235), (e'k*\\tS', 'j', 4195589227), (e'\\x13\\x19kWe', 'e', 903597985), ('wN:,SeNG', 'X', 4079306283), ('', '&', 1488840225), ('t0', '%', 3562745105), (e'\\U00002603', '^', 1193010910)), PARTITION table4_part_3 VALUES IN ((e'\\x7fYe', '7', 3839609211), (e'I5kv\\n\\x1d', ')', 3779786321), ('X', e'\\x19', 0), (e'^;T5~\\x11/\\\\5', '[', 1078283111), ('oW', '^', 152371999), ('', '/', 1571936057), (e'\\x0f\\x11', e'\\t', 2693635799), (e'tKxPx\\x7fP\\rg', e'\\x1b', 214166676), (e'\\x069#\\\\9\\x11v}O', '_', 1091130754)), PARTITION table4_part_4 VALUES IN ((e'#U\\x17\\x1b\\x12#', '7', 0), ('jL:;', 'p', 2123810219), ('qP', '/', 3104581437), (e'Q|i4\\x03+', e'\\x13', 3266494055), (e'\\x7f@\\x0fLn', e'\\x0b', 2244375423), (e'?\\n  \\x01T\"T\\b', 'y', 3501975016), (e'\\t\\x068xl', '\"', 2271730306), ('v}', 'a', 3260316571), (e'sL\\t2q\\x02', e'\\x00', 2101439645)), PARTITION table4_part_5 VALUES IN (('R0', e'\\x11', 673358489), (e'w\\x17', 'A', 3726715316), (e'\\x01\\x1c', '/', 0), ('PG<', 'S', 3624431705), (e'\\x19\\x10\\x13', 'g', 0), (e'\\x10\\x1f\\x1d\".m', e'\\x04', 1547080936), ('$', e'\\x00', 839412529), (e'O\\b\\x12\"', 'O', 0), (e'\\x0f\\x10t\\x04', 'S', 2096027704)), PARTITION table4_part_6 VALUES IN (('g', 'j', 2071028640), ('', 'c', 2405874649), (e'\\x16\\x139$|^q', 'a', 2089154318), (e'\\x02\\bT', '^', 333603208), ('{', 'i', 3301501767), ('{]!}', 'r', 4095591002), (e'\\x1aX\\x1cC', e'\\x1a', 2609292279), (')dO', 'c', 0), (e'\\x01\\x0b\\x0eBC4', 'J', 0)), PARTITION table4_part_7 VALUES IN ((e'\\U00002603', e'\\t', 2141991755), ('s', 's', 846284978), (e'.\\fXS', e'\\x0f', 997777549), ('', 'T', 3798631380), (e'+q\\x10\\x12R', e'\\x19', 3928399494), (e'CJ$K\\x02;', 'O', 3870085288), ('X', '\"', 3573195198), ('oF:', '$', 4168147292), (e'(\\\\', 'E', 0)), PARTITION table4_part_8 VALUES IN ((e's&\\x02i\\x12Q``x\\x02', 'h', 3446575851), ('!i-', e'\\x02', 3597092236), (e'\\x10', e'\\'', 1406087395), ('', 'c', 88705775), ('', 'r', 3708936451), (e'``x\\x05P', '1', 0), (e'\\x10Eh\\x06g', e'\\x19', 137395561), (e'z\\x13L\\x05~', 'd', 1035604431), (e'\\f.', 'G', 2134835641)), PARTITION table4_part_9 VALUES IN ((e'.l\\x03', '', 4284958193), (e'$WA\\x10\\x17&a\\x16', e'\\x11', 756209360), (e'+nux\\x1d\\x1b', '{', 2995197263), (e'$c\\n', e'\\x01', 2953414521), (e'\\x03``]C4g5', '3', 3919240825), (e'\\x01xU\\x06s*', e'\\t', 0), ('[n', 'J', 94254714), (e'\\x1b\\x14\\x1b', '', 3964974415), ('c\"vG:> \"', e'\\'', 4039603590)), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT, DEFAULT))) NOT VISIBLE, INDEX table4_col4_5_idx (col4_5 DESC) NOT VISIBLE, INDEX table4_expr_idx (abs(col4_0) ASC) NOT VISIBLE, CONSTRAINT table4_col4_3_col4_0_col4_1_key UNIQUE (col4_3 DESC, col4_0 DESC, col4_1 DESC), CONSTRAINT table4_col4_4_col4_3_col4_0_key UNIQUE (col4_4 ASC, col4_3 ASC, col4_0 DESC), INDEX table4_expr_col4_4_idx (lower(CAST(col4_3 AS STRING)) DESC, col4_4 ASC), INDEX table4_col4_2_col4_0_idx (col4_2 ASC, col4_0 DESC) NOT VISIBLE)"
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema2411824109
--- FAIL: TestCreateRandomSchema (0.99s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17702130?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17702130?buildTab=artifacts#/) on master @ [3efc583b82c4be8b1a2db8fcfe6a2785713c59a5](https://github.com/cockroachdb/cockroach/commits/3efc583b82c4be8b1a2db8fcfe6a2785713c59a5):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema3884139333
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:107: for input statement
        CREATE TABLE table61 (col61_0 BOX2D NOT NULL, col61_1 BPCHAR NOT NULL, PRIMARY KEY (col61_1), INDEX (col61_0 ASC), INDEX (col61_0 DESC, lower(CAST(col61_0 AS STRING))), INDEX (col61_0, col61_1 DESC) PARTITION BY LIST (col61_0, col61_1) (PARTITION table61_part_0 VALUES IN (('BOX(-0.4980982061925275 -0.4961098757438823,0.5529075360814133 1.1417264962880112)':::BOX2D, e'\x05':::STRING), ('BOX(-0.7239466621302816 0.05786897967927071,0.7878492133434056 0.41091233082344836)':::BOX2D, 'A':::STRING), ('BOX(-1.5446832860627082 -2.1513507315535843,-1.2086379179084923 -1.561383587587697)':::BOX2D, '7':::STRING), ('BOX(-1.497415960422205 -0.07984296050922013,0.09287228103156753 0.5343657236173552)':::BOX2D, 'X':::STRING)), PARTITION table61_part_1 VALUES IN (('BOX(-2.3053245123505666 -0.5803800787512918,0.40819034608408167 -0.35462597640629223)':::BOX2D, e'\x12':::STRING), ('BOX(-0.8601842188972213 0.018745122902653066,0.6148990925220572 1.0869593527476478)':::BOX2D, e'\x14':::STRING), ('BOX(-10 -10,10 10)':::BOX2D, e'\x18':::STRING), ('BOX(0.10912999507250287 -0.6873411757333276,1.210511214194109 0.5130553544534372)':::BOX2D, ' ':::STRING))), UNIQUE (col61_1 ASC))
        found first output
        "CREATE TABLE public.table61 (col61_0 BOX2D NOT NULL, col61_1 BPCHAR NOT NULL, CONSTRAINT table61_pkey PRIMARY KEY (col61_1 ASC), INDEX table61_col61_0_idx (col61_0 ASC), INDEX table61_col61_0_expr_idx (col61_0 DESC, lower(CAST(col61_0 AS STRING)) ASC), INDEX table61_col61_0_col61_1_idx (col61_0 ASC, col61_1 DESC) PARTITION BY LIST (col61_0, col61_1) (PARTITION table61_part_0 VALUES IN (('BOX(-0.4980982061925275 -0.4961098757438823,0.5529075360814133 1.1417264962880112)', e'\\x05'), ('BOX(-0.7239466621302816 0.05786897967927071,0.7878492133434056 0.41091233082344836)', 'A'), ('BOX(-1.5446832860627082 -2.1513507315535843,-1.2086379179084923 -1.561383587587697)', '7'), ('BOX(-1.497415960422205 -0.07984296050922013,0.09287228103156753 0.5343657236173552)', 'X')), PARTITION table61_part_1 VALUES IN (('BOX(-2.3053245123505666 -0.5803800787512918,0.40819034608408167 -0.35462597640629223)', e'\\x12'), ('BOX(-0.8601842188972213 0.018745122902653066,0.6148990925220572 1.0869593527476478)', e'\\x14'), ('BOX(-10 -10,10 10)', e'\\x18'), ('BOX(0.10912999507250287 -0.6873411757333276,1.210511214194109 0.5130553544534372)', ' '))), CONSTRAINT table61_col61_1_key UNIQUE (col61_1 ASC))"
        but second output
        "CREATE TABLE public.table61 (col61_0 BOX2D NOT NULL, col61_1 BPCHAR NOT NULL, CONSTRAINT table61_pkey PRIMARY KEY (col61_1 ASC), INDEX table61_col61_0_idx (col61_0 ASC), INDEX table61_col61_0_expr_idx (col61_0 DESC, lower(CAST(col61_0 AS STRING)) ASC), INDEX table61_col61_0_col61_1_idx (col61_0 ASC, col61_1 DESC) PARTITION BY LIST (col61_0, col61_1) (PARTITION table61_part_0 VALUES IN (('BOX(-0.4980982061925275 -0.4961098757438823,0.5529075360814133 1.1417264962880112)', e'\\x05'), ('BOX(-0.7239466621302816 0.05786897967927071,0.7878492133434056 0.41091233082344836)', 'A'), ('BOX(-1.5446832860627082 -2.1513507315535843,-1.2086379179084923 -1.561383587587697)', '7'), ('BOX(-1.497415960422205 -0.07984296050922013,0.09287228103156753 0.5343657236173552)', 'X')), PARTITION table61_part_1 VALUES IN (('BOX(-2.3053245123505666 -0.5803800787512918,0.40819034608408167 -0.35462597640629223)', e'\\x12'), ('BOX(-0.8601842188972213 0.018745122902653066,0.6148990925220572 1.0869593527476478)', e'\\x14'), ('BOX(-10 -10,10 10)', e'\\x18'), ('BOX(0.10912999507250287 -0.6873411757333276,1.210511214194109 0.5130553544534372)', ''))), CONSTRAINT table61_col61_1_key UNIQUE (col61_1 ASC))"
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema3884139333
--- FAIL: TestCreateRandomSchema (10.84s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17703298?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17703298?buildTab=artifacts#/) on master @ [89d6970fddb9d21f1353c373680aa14ccee0bc35](https://github.com/cockroachdb/cockroach/commits/89d6970fddb9d21f1353c373680aa14ccee0bc35):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema2854896128
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:90: CREATE TABLE public.table40 (col40_0 GEOGRAPHY NOT NULL, col40_1 BPCHAR NOT NULL, col40_2 BOOL NOT NULL, col40_3 FLOAT8 NOT NULL, col40_4 BIT(37) NOT NULL, col40_5 INT4 NULL, col40_6 INT8 NOT NULL, col40_7 INT8 NOT NULL, col40_8 NAME NOT NULL, col40_9 INT4 NULL AS (abs(col40_5)) STORED, col40_10 INT8 NULL AS (col40_7 + col40_5) VIRTUAL, col40_11 FLOAT8 NOT NULL AS (abs(col40_3)) VIRTUAL, col40_12 STRING NOT NULL AS (lower(col40_1)) STORED, CONSTRAINT table40_pkey PRIMARY KEY (col40_2 DESC), INDEX table40_col40_4_col40_6_expr_idx (col40_4 ASC, col40_6 DESC, abs(col40_5) DESC) STORING (col40_1, col40_9), INDEX table40_expr_idx (((col40_5 + col40_6) + col40_7) DESC) STORING (col40_0, col40_1, col40_3, col40_7), INDEX table40_col40_1_idx (col40_1 ASC) PARTITION BY LIST (col40_1) (PARTITION table40_part_0 VALUES IN ((' '), (e'\x19'), (e'\x16'), ('P'), (''), (e'\x1b'), ('-'), ('V'), ('[')), PARTITION table40_part_1 VALUES IN ((e'\x04'), ('h'), (e'\x10'), ('z'), ('?'), ('t'), ('M'), ('"')), PARTITION table40_part_2 VALUES IN ((e'\''), ('q'), (e'\n'), ('A'), (e'\x14')), PARTITION table40_part_3 VALUES IN (('D'), ('b'), (e'\x18'), (e'\b')), PARTITION "DEFAULT" VALUES IN ((DEFAULT)))) pq: relation "table40" (189): ('') cannot be present in more than one partition
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema2854896128
--- FAIL: TestCreateRandomSchema (4.66s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17704712?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17704712?buildTab=artifacts#/) on master @ [5a6cd226601e10e57a4e86da2df6df4a9f6bedc3](https://github.com/cockroachdb/cockroach/commits/5a6cd226601e10e57a4e86da2df6df4a9f6bedc3):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema2406596543
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:107: for input statement
        CREATE TABLE table76 (col76_0 VARCHAR NOT NULL, col76_1 TIME NOT NULL, col76_2 BPCHAR NOT NULL, col76_3 STRING NOT NULL AS (lower(col76_0)) STORED, PRIMARY KEY (col76_1 DESC), UNIQUE (col76_0 ASC, col76_1, col76_2) PARTITION BY LIST (col76_0, col76_1, col76_2) (PARTITION table76_part_0 VALUES IN ((e'@\f\x063\x03OT':::STRING, '05:05:59.57506':::TIME, '7':::STRING), (e'\f0':::STRING, '02:36:38.050893':::TIME, '-':::STRING), ('':::STRING, '16:30:54.10397':::TIME, '?':::STRING), ('R':::STRING, '06:59:59.709395':::TIME, '4':::STRING), ('jL':::STRING, '06:04:39.032208':::TIME, e'\x16':::STRING), ('NDk':::STRING, '00:12:56.673608':::TIME, e'\b':::STRING), ('"':::STRING, '06:59:17.738694':::TIME, '9':::STRING), ('':::STRING, '18:27:46.367522':::TIME, '^':::STRING), (e'caC^\x1fS':::STRING, '02:27:05.866158':::TIME, 'o':::STRING), (e'2\x7f\x16A\\\nF\x03':::STRING, '10:30:03.909803':::TIME, e'\x03':::STRING)), PARTITION table76_part_1 VALUES IN ((e'k\x15b1 o\f':::STRING, '06:27:27.582345':::TIME, e'\x16':::STRING), (' W+':::STRING, '16:10:33.50632':::TIME, 'V':::STRING), (e'\x04y``\x1f\x16':::STRING, '04:09:45.212733':::TIME, 'e':::STRING), (e'a\x13y@$!:t':::STRING, '08:50:00.450785':::TIME, '|':::STRING), (e'\x17a\x1fI^iD\x06F':::STRING, '18:40:58.634872':::TIME, e'\x03':::STRING), ('':::STRING, '02:12:20.069102':::TIME, 'P':::STRING), ('':::STRING, '22:55:22.104649':::TIME, 'x':::STRING), ('=dEA':::STRING, '00:33:05.721751':::TIME, 'p':::STRING), (e'ucD}:T\x14':::STRING, '14:51:37.581606':::TIME, '}':::STRING), ('6':::STRING, '06:27:26.885232':::TIME, e'\x02':::STRING)), PARTITION table76_part_2 VALUES IN (('a':::STRING, '18:38:24.441447':::TIME, 'k':::STRING), (e'/pn\x01n\x7fL\x17':::STRING, '21:07:13.401557':::TIME, '+':::STRING), ('':::STRING, '05:21:05.392543':::TIME, 'r':::STRING), (e'\'':::STRING, '22:39:29.982894':::TIME, 'V':::STRING), (e'!\x13Coh':::STRING, '13:23:43.847819':::TIME, 'Q':::STRING), (e'\\-\x1dT;':::STRING, '11:35:46.849195':::TIME, e'\x11':::STRING), ('C':::STRING, '16:23:04.15306':::TIME, 's':::STRING), ('?':::STRING, '05:18:10.229462':::TIME, 'H':::STRING), ('':::STRING, '00:39:22.525591':::TIME, 'X':::STRING), (e'!F\t\x01nF\'"\x06':::STRING, '00:00:00':::TIME, 'C':::STRING)), PARTITION table76_part_3 VALUES IN ((e'\flm\b\b#\x05(':::STRING, '00:00:00':::TIME, 'p':::STRING), (e'\x00':::STRING, '19:55:16.821062':::TIME, '@':::STRING), (e'Z\x1dj\n':::STRING, '05:26:38.628449':::TIME, '-':::STRING), ('tPs,':::STRING, '10:19:47.418834':::TIME, e'\x00':::STRING), (e'bh2\x13BZl\x1c':::STRING, '02:18:25.640053':::TIME, 'W':::STRING), (e'B\x02\f':::STRING, '01:57:41.384874':::TIME, '0':::STRING), ('jE':::STRING, '10:08:26.469664':::TIME, 'E':::STRING), (e'#kD\x1d':::STRING, '13:15:52.220263':::TIME, 'k':::STRING), (e'\x01"n%#':::STRING, '21:47:26.427966':::TIME, e'\x00':::STRING), (e'J_U\x114':::STRING, '03:15:45.180441':::TIME, e'\'':::STRING)), PARTITION table76_part_4 VALUES IN ((e'2(#\x18\x1d}]T':::STRING, '08:41:01.522306':::TIME, '=':::STRING), ('':::STRING, '11:59:35.78432':::TIME, e'\x06':::STRING), (e'W"\x05f':::STRING, '08:56:02.687212':::TIME, '5':::STRING), (e'=\b\'':::STRING, '23:40:30.10971':::TIME, e'\x19':::STRING), (e'(|+*A\x05.\rh':::STRING, '17:11:53.85349':::TIME, '=':::STRING), (e'\x13q':::STRING, '16:37:20.511695':::TIME, 'r':::STRING), (e'&\x04L':::STRING, '18:03:31.881345':::TIME, 'm':::STRING), (e'\x01}``\x15\x1a\x1d':::STRING, '03:39:53.886279':::TIME, e'\x04':::STRING), (e'lO{3\x01k=':::STRING, '09:36:47.853949':::TIME, 'w':::STRING), ('An8Q$N}{2':::STRING, '08:27:37.222278':::TIME, '-':::STRING)), PARTITION table76_part_5 VALUES IN (('2Uj8lmbk@':::STRING, '18:13:49.645928':::TIME, '%':::STRING), (e'BBFye\x04O':::STRING, '05:57:02.439404':::TIME, e'\x07':::STRING), ('o=3j':::STRING, '00:24:08.526036':::TIME, 'z':::STRING), (e'G3"\x01[':::STRING, '00:00:00':::TIME, e'\x03':::STRING), ('"':::STRING, '12:04:48.523498':::TIME, ' ':::STRING), ('':::STRING, '16:09:36.759558':::TIME, '!':::STRING), (e'klh\x1a\nDI\x16\x1a':::STRING, '14:08:34.41742':::TIME, e'\'':::STRING), (e'Fp3\x1c':::STRING, '03:22:31.53463':::TIME, '9':::STRING), (e'v^\x03N5':::STRING, '14:58:30.518156':::TIME, '<':::STRING), (e'\x16qP7ww(':::STRING, '04:53:46.149708':::TIME, '':::STRING)), PARTITION table76_part_6 VALUES IN ((e'\x0fu':::STRING, '19:13:01.785741':::TIME, 'S':::STRING), ('zCtI':::STRING, '03:26:23.716677':::TIME, 'K':::STRING), ('Mk/@O':::STRING, '11:35:48.457003':::TIME, 'U':::STRING), (e'C\x01NGu\r\x01o':::STRING, '03:13:45.019333':::TIME, '7':::STRING), (e'\x19tR':::STRING, '13:33:45.703202':::TIME, '$':::STRING), ('q{TQ':::STRING, '18:49:22.095058':::TIME, 'F':::STRING), ('X':::STRING, '21:05:27.337941':::TIME, '[':::STRING), ('6$=':::STRING, '12:04:17.997436':::TIME, e'\x18':::STRING), (e'))\t\x15{7':::STRING, '19:38:36.118396':::TIME, '``':::STRING), (e'Tl\x0b&K\x11':::STRING, '14:38:12.163301':::TIME, 'k':::STRING)), PARTITION table76_part_7 VALUES IN ((e'/@\x14rY\x06m':::STRING, '15:47:08.718501':::TIME, 'm':::STRING), (e'E\x1aQ':::STRING, '08:09:16.736493':::TIME, 'h':::STRING), (e'\x13o\x189;':::STRING, '01:52:54.789767':::TIME, e'\x12':::STRING), (e'\x1c<]':::STRING, '24:00:00':::TIME, 'B':::STRING), (e'\U00002603':::STRING, '24:00:00':::TIME, '8':::STRING), ('':::STRING, '06:44:29.469459':::TIME, 'Y':::STRING), ('':::STRING, '05:59:38.749232':::TIME, 'V':::STRING), (e'3$g:\'pC!':::STRING, '07:09:45.732606':::TIME, '':::STRING), (e'a<7\x17Vb':::STRING, '24:00:00':::TIME, '+':::STRING), (e'y\\\x05':::STRING, '01:46:43.292897':::TIME, e'\x15':::STRING)), PARTITION table76_part_8 VALUES IN (('{o2':::STRING, '24:00:00':::TIME, '!':::STRING), ('':::STRING, '00:43:06.076271':::TIME, e'\x1a':::STRING), (e'\x0bL!':::STRING, '01:47:27.110827':::TIME, e'\x1f':::STRING), ('#':::STRING, '13:57:51.982835':::TIME, '"':::STRING), ('':::STRING, '10:28:09.489087':::TIME, '*':::STRING), (e'\x02,\x1e.m\x18b<f':::STRING, '24:00:00':::TIME, e'\x13':::STRING), (e'A\x1f&6:"[\x05l':::STRING, '16:33:25.662748':::TIME, '1':::STRING), (e'su\x1e$_M2':::STRING, '10:15:05.429911':::TIME, 'U':::STRING), (e'r\x10wjEPC':::STRING, '04:52:20.594722':::TIME, 'P':::STRING), (e'\bbu\x1f ':::STRING, '22:13:24.931583':::TIME, 'U':::STRING)), PARTITION table76_part_9 VALUES IN (('tpc':::STRING, '19:50:50.281655':::TIME, 'w':::STRING), (e'\x00':::STRING, '16:53:04.946614':::TIME, 'B':::STRING), (e'\x1aD\x1e\x0f_\n':::STRING, '24:00:00':::TIME, '@':::STRING), (e'\x16,\x18\x06&Do]\x02':::STRING, '00:06:37.542126':::TIME, '{':::STRING), (e'Qec@\x06':::STRING, '00:16:39.244066':::TIME, e'\'':::STRING), (e'``\x1f\x1avBhN':::STRING, '05:39:43.919555':::TIME, 'q':::STRING), ('':::STRING, '24:00:00':::TIME, e'\'':::STRING), (e'H3\x12\x02kGa:2':::STRING, '02:12:28.846667':::TIME, 'H':::STRING), (e'd=a~\x06M4^':::STRING, '23:57:07.4735':::TIME, '|':::STRING), (e'\x10\x13':::STRING, '24:00:00':::TIME, 'X':::STRING)), PARTITION "DEFAULT" VALUES IN ((DEFAULT, DEFAULT, DEFAULT))), INDEX (col76_2 ASC, col76_0 DESC, col76_1 ASC) STORING (col76_3), INDEX (col76_3 ASC, col76_0 ASC, col76_1) STORING (col76_2), INVERTED INDEX (col76_1 ASC, col76_2 DESC, col76_0 gin_trgm_ops))
        found first output
        "CREATE TABLE public.table76 (col76_0 VARCHAR NOT NULL, col76_1 TIME NOT NULL, col76_2 BPCHAR NOT NULL, col76_3 STRING NOT NULL AS (lower(col76_0)) STORED, CONSTRAINT table76_pkey PRIMARY KEY (col76_1 DESC), CONSTRAINT table76_col76_0_col76_1_col76_2_key UNIQUE (col76_0 ASC, col76_1 ASC, col76_2 ASC) PARTITION BY LIST (col76_0, col76_1, col76_2) (PARTITION table76_part_0 VALUES IN ((e'@\\f\\x063\\x03OT', '05:05:59.57506', '7'), (e'\\f0', '02:36:38.050893', '-'), ('', '16:30:54.10397', '?'), ('R', '06:59:59.709395', '4'), ('jL', '06:04:39.032208', e'\\x16'), ('NDk', '00:12:56.673608', e'\\b'), ('\"', '06:59:17.738694', '9'), ('', '18:27:46.367522', '^'), (e'caC^\\x1fS', '02:27:05.866158', 'o'), (e'2\\x7f\\x16A\\\\\\nF\\x03', '10:30:03.909803', e'\\x03')), PARTITION table76_part_1 VALUES IN ((e'k\\x15b1 o\\f', '06:27:27.582345', e'\\x16'), (' W+', '16:10:33.50632', 'V'), (e'\\x04y``\\x1f\\x16', '04:09:45.212733', 'e'), (e'a\\x13y@$!:t', '08:50:00.450785', '|'), (e'\\x17a\\x1fI^iD\\x06F', '18:40:58.634872', e'\\x03'), ('', '02:12:20.069102', 'P'), ('', '22:55:22.104649', 'x'), ('=dEA', '00:33:05.721751', 'p'), (e'ucD}:T\\x14', '14:51:37.581606', '}'), ('6', '06:27:26.885232', e'\\x02')), PARTITION table76_part_2 VALUES IN (('a', '18:38:24.441447', 'k'), (e'/pn\\x01n\\x7fL\\x17', '21:07:13.401557', '+'), ('', '05:21:05.392543', 'r'), (e'\\'', '22:39:29.982894', 'V'), (e'!\\x13Coh', '13:23:43.847819', 'Q'), (e'\\\\-\\x1dT;', '11:35:46.849195', e'\\x11'), ('C', '16:23:04.15306', 's'), ('?', '05:18:10.229462', 'H'), ('', '00:39:22.525591', 'X'), (e'!F\\t\\x01nF\\'\"\\x06', '00:00:00', 'C')), PARTITION table76_part_3 VALUES IN ((e'\\flm\\b\\b#\\x05(', '00:00:00', 'p'), (e'\\x00', '19:55:16.821062', '@'), (e'Z\\x1dj\\n', '05:26:38.628449', '-'), ('tPs,', '10:19:47.418834', e'\\x00'), (e'bh2\\x13BZl\\x1c', '02:18:25.640053', 'W'), (e'B\\x02\\f', '01:57:41.384874', '0'), ('jE', '10:08:26.469664', 'E'), (e'#kD\\x1d', '13:15:52.220263', 'k'), (e'\\x01\"n%#', '21:47:26.427966', e'\\x00'), (e'J_U\\x114', '03:15:45.180441', e'\\'')), PARTITION table76_part_4 VALUES IN ((e'2(#\\x18\\x1d}]T', '08:41:01.522306', '='), ('', '11:59:35.78432', e'\\x06'), (e'W\"\\x05f', '08:56:02.687212', '5'), (e'=\\b\\'', '23:40:30.10971', e'\\x19'), (e'(|+*A\\x05.\\rh', '17:11:53.85349', '='), (e'\\x13q', '16:37:20.511695', 'r'), (e'&\\x04L', '18:03:31.881345', 'm'), (e'\\x01}``\\x15\\x1a\\x1d', '03:39:53.886279', e'\\x04'), (e'lO{3\\x01k=', '09:36:47.853949', 'w'), ('An8Q$N}{2', '08:27:37.222278', '-')), PARTITION table76_part_5 VALUES IN (('2Uj8lmbk@', '18:13:49.645928', '%'), (e'BBFye\\x04O', '05:57:02.439404', e'\\x07'), ('o=3j', '00:24:08.526036', 'z'), (e'G3\"\\x01[', '00:00:00', e'\\x03'), ('\"', '12:04:48.523498', ' '), ('', '16:09:36.759558', '!'), (e'klh\\x1a\\nDI\\x16\\x1a', '14:08:34.41742', e'\\''), (e'Fp3\\x1c', '03:22:31.53463', '9'), (e'v^\\x03N5', '14:58:30.518156', '<'), (e'\\x16qP7ww(', '04:53:46.149708', '')), PARTITION table76_part_6 VALUES IN ((e'\\x0fu', '19:13:01.785741', 'S'), ('zCtI', '03:26:23.716677', 'K'), ('Mk/@O', '11:35:48.457003', 'U'), (e'C\\x01NGu\\r\\x01o', '03:13:45.019333', '7'), (e'\\x19tR', '13:33:45.703202', '$'), ('q{TQ', '18:49:22.095058', 'F'), ('X', '21:05:27.337941', '['), ('6$=', '12:04:17.997436', e'\\x18'), (e'))\\t\\x15{7', '19:38:36.118396', '``'), (e'Tl\\x0b&K\\x11', '14:38:12.163301', 'k')), PARTITION table76_part_7 VALUES IN ((e'/@\\x14rY\\x06m', '15:47:08.718501', 'm'), (e'E\\x1aQ', '08:09:16.736493', 'h'), (e'\\x13o\\x189;', '01:52:54.789767', e'\\x12'), (e'\\x1c<]', '24:00:00', 'B'), (e'\\U00002603', '24:00:00', '8'), ('', '06:44:29.469459', 'Y'), ('', '05:59:38.749232', 'V'), (e'3$g:\\'pC!', '07:09:45.732606', ''), (e'a<7\\x17Vb', '24:00:00', '+'), (e'y\\\\\\x05', '01:46:43.292897', e'\\x15')), PARTITION table76_part_8 VALUES IN (('{o2', '24:00:00', '!'), ('', '00:43:06.076271', e'\\x1a'), (e'\\x0bL!', '01:47:27.110827', e'\\x1f'), ('#', '13:57:51.982835', '\"'), ('', '10:28:09.489087', '*'), (e'\\x02,\\x1e.m\\x18b<f', '24:00:00', e'\\x13'), (e'A\\x1f&6:\"[\\x05l', '16:33:25.662748', '1'), (e'su\\x1e$_M2', '10:15:05.429911', 'U'), (e'r\\x10wjEPC', '04:52:20.594722', 'P'), (e'\\bbu\\x1f ', '22:13:24.931583', 'U')), PARTITION table76_part_9 VALUES IN (('tpc', '19:50:50.281655', 'w'), (e'\\x00', '16:53:04.946614', 'B'), (e'\\x1aD\\x1e\\x0f_\\n', '24:00:00', '@'), (e'\\x16,\\x18\\x06&Do]\\x02', '00:06:37.542126', '{'), (e'Qec@\\x06', '00:16:39.244066', e'\\''), (e'``\\x1f\\x1avBhN', '05:39:43.919555', 'q'), ('', '24:00:00', e'\\''), (e'H3\\x12\\x02kGa:2', '02:12:28.846667', 'H'), (e'd=a~\\x06M4^', '23:57:07.4735', '|'), (e'\\x10\\x13', '24:00:00', 'X')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT, DEFAULT))), INDEX table76_col76_2_col76_0_col76_1_idx (col76_2 ASC, col76_0 DESC, col76_1 ASC) STORING (col76_3), INDEX table76_col76_3_col76_0_col76_1_idx (col76_3 ASC, col76_0 ASC, col76_1 ASC) STORING (col76_2), INVERTED INDEX table76_col76_1_col76_2_col76_0_idx (col76_1 ASC, col76_2 DESC, col76_0 gin_trgm_ops))"
        but second output
        "CREATE TABLE public.table76 (col76_0 VARCHAR NOT NULL, col76_1 TIME NOT NULL, col76_2 BPCHAR NOT NULL, col76_3 STRING NOT NULL AS (lower(col76_0)) STORED, CONSTRAINT table76_pkey PRIMARY KEY (col76_1 DESC), CONSTRAINT table76_col76_0_col76_1_col76_2_key UNIQUE (col76_0 ASC, col76_1 ASC, col76_2 ASC) PARTITION BY LIST (col76_0, col76_1, col76_2) (PARTITION table76_part_0 VALUES IN ((e'@\\f\\x063\\x03OT', '05:05:59.57506', '7'), (e'\\f0', '02:36:38.050893', '-'), ('', '16:30:54.10397', '?'), ('R', '06:59:59.709395', '4'), ('jL', '06:04:39.032208', e'\\x16'), ('NDk', '00:12:56.673608', e'\\b'), ('\"', '06:59:17.738694', '9'), ('', '18:27:46.367522', '^'), (e'caC^\\x1fS', '02:27:05.866158', 'o'), (e'2\\x7f\\x16A\\\\\\nF\\x03', '10:30:03.909803', e'\\x03')), PARTITION table76_part_1 VALUES IN ((e'k\\x15b1 o\\f', '06:27:27.582345', e'\\x16'), (' W+', '16:10:33.50632', 'V'), (e'\\x04y``\\x1f\\x16', '04:09:45.212733', 'e'), (e'a\\x13y@$!:t', '08:50:00.450785', '|'), (e'\\x17a\\x1fI^iD\\x06F', '18:40:58.634872', e'\\x03'), ('', '02:12:20.069102', 'P'), ('', '22:55:22.104649', 'x'), ('=dEA', '00:33:05.721751', 'p'), (e'ucD}:T\\x14', '14:51:37.581606', '}'), ('6', '06:27:26.885232', e'\\x02')), PARTITION table76_part_2 VALUES IN (('a', '18:38:24.441447', 'k'), (e'/pn\\x01n\\x7fL\\x17', '21:07:13.401557', '+'), ('', '05:21:05.392543', 'r'), (e'\\'', '22:39:29.982894', 'V'), (e'!\\x13Coh', '13:23:43.847819', 'Q'), (e'\\\\-\\x1dT;', '11:35:46.849195', e'\\x11'), ('C', '16:23:04.15306', 's'), ('?', '05:18:10.229462', 'H'), ('', '00:39:22.525591', 'X'), (e'!F\\t\\x01nF\\'\"\\x06', '00:00:00', 'C')), PARTITION table76_part_3 VALUES IN ((e'\\flm\\b\\b#\\x05(', '00:00:00', 'p'), (e'\\x00', '19:55:16.821062', '@'), (e'Z\\x1dj\\n', '05:26:38.628449', '-'), ('tPs,', '10:19:47.418834', e'\\x00'), (e'bh2\\x13BZl\\x1c', '02:18:25.640053', 'W'), (e'B\\x02\\f', '01:57:41.384874', '0'), ('jE', '10:08:26.469664', 'E'), (e'#kD\\x1d', '13:15:52.220263', 'k'), (e'\\x01\"n%#', '21:47:26.427966', e'\\x00'), (e'J_U\\x114', '03:15:45.180441', e'\\'')), PARTITION table76_part_4 VALUES IN ((e'2(#\\x18\\x1d}]T', '08:41:01.522306', '='), ('', '11:59:35.78432', e'\\x06'), (e'W\"\\x05f', '08:56:02.687212', '5'), (e'=\\b\\'', '23:40:30.10971', e'\\x19'), (e'(|+*A\\x05.\\rh', '17:11:53.85349', '='), (e'\\x13q', '16:37:20.511695', 'r'), (e'&\\x04L', '18:03:31.881345', 'm'), (e'\\x01}``\\x15\\x1a\\x1d', '03:39:53.886279', e'\\x04'), (e'lO{3\\x01k=', '09:36:47.853949', 'w'), ('An8Q$N}{2', '08:27:37.222278', '-')), PARTITION table76_part_5 VALUES IN (('2Uj8lmbk@', '18:13:49.645928', '%'), (e'BBFye\\x04O', '05:57:02.439404', e'\\x07'), ('o=3j', '00:24:08.526036', 'z'), (e'G3\"\\x01[', '00:00:00', e'\\x03'), ('\"', '12:04:48.523498', ''), ('', '16:09:36.759558', '!'), (e'klh\\x1a\\nDI\\x16\\x1a', '14:08:34.41742', e'\\''), (e'Fp3\\x1c', '03:22:31.53463', '9'), (e'v^\\x03N5', '14:58:30.518156', '<'), (e'\\x16qP7ww(', '04:53:46.149708', '')), PARTITION table76_part_6 VALUES IN ((e'\\x0fu', '19:13:01.785741', 'S'), ('zCtI', '03:26:23.716677', 'K'), ('Mk/@O', '11:35:48.457003', 'U'), (e'C\\x01NGu\\r\\x01o', '03:13:45.019333', '7'), (e'\\x19tR', '13:33:45.703202', '$'), ('q{TQ', '18:49:22.095058', 'F'), ('X', '21:05:27.337941', '['), ('6$=', '12:04:17.997436', e'\\x18'), (e'))\\t\\x15{7', '19:38:36.118396', '``'), (e'Tl\\x0b&K\\x11', '14:38:12.163301', 'k')), PARTITION table76_part_7 VALUES IN ((e'/@\\x14rY\\x06m', '15:47:08.718501', 'm'), (e'E\\x1aQ', '08:09:16.736493', 'h'), (e'\\x13o\\x189;', '01:52:54.789767', e'\\x12'), (e'\\x1c<]', '24:00:00', 'B'), (e'\\U00002603', '24:00:00', '8'), ('', '06:44:29.469459', 'Y'), ('', '05:59:38.749232', 'V'), (e'3$g:\\'pC!', '07:09:45.732606', ''), (e'a<7\\x17Vb', '24:00:00', '+'), (e'y\\\\\\x05', '01:46:43.292897', e'\\x15')), PARTITION table76_part_8 VALUES IN (('{o2', '24:00:00', '!'), ('', '00:43:06.076271', e'\\x1a'), (e'\\x0bL!', '01:47:27.110827', e'\\x1f'), ('#', '13:57:51.982835', '\"'), ('', '10:28:09.489087', '*'), (e'\\x02,\\x1e.m\\x18b<f', '24:00:00', e'\\x13'), (e'A\\x1f&6:\"[\\x05l', '16:33:25.662748', '1'), (e'su\\x1e$_M2', '10:15:05.429911', 'U'), (e'r\\x10wjEPC', '04:52:20.594722', 'P'), (e'\\bbu\\x1f ', '22:13:24.931583', 'U')), PARTITION table76_part_9 VALUES IN (('tpc', '19:50:50.281655', 'w'), (e'\\x00', '16:53:04.946614', 'B'), (e'\\x1aD\\x1e\\x0f_\\n', '24:00:00', '@'), (e'\\x16,\\x18\\x06&Do]\\x02', '00:06:37.542126', '{'), (e'Qec@\\x06', '00:16:39.244066', e'\\''), (e'``\\x1f\\x1avBhN', '05:39:43.919555', 'q'), ('', '24:00:00', e'\\''), (e'H3\\x12\\x02kGa:2', '02:12:28.846667', 'H'), (e'd=a~\\x06M4^', '23:57:07.4735', '|'), (e'\\x10\\x13', '24:00:00', 'X')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT, DEFAULT))), INDEX table76_col76_2_col76_0_col76_1_idx (col76_2 ASC, col76_0 DESC, col76_1 ASC) STORING (col76_3), INDEX table76_col76_3_col76_0_col76_1_idx (col76_3 ASC, col76_0 ASC, col76_1 ASC) STORING (col76_2), INVERTED INDEX table76_col76_1_col76_2_col76_0_idx (col76_1 ASC, col76_2 DESC, col76_0 gin_trgm_ops))"
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema2406596543
--- FAIL: TestCreateRandomSchema (5.20s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17708942?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17708942?buildTab=artifacts#/) on master @ [82d8508e6fa04790eec147a48990e77b2c6ed046](https://github.com/cockroachdb/cockroach/commits/82d8508e6fa04790eec147a48990e77b2c6ed046):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema1155140553
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:107: for input statement
        CREATE TABLE table95 (col95_0 "char" NOT NULL, col95_1 "char" NOT NULL, col95_2 BPCHAR NOT NULL, col95_3 TSVECTOR NOT NULL, col95_4 BIT(27) NOT NULL, col95_5 BPCHAR NOT NULL, col95_6 BIT(49), col95_7 STRING NOT NULL AS (lower(col95_1)) VIRTUAL, col95_8 STRING NOT NULL AS (lower(col95_0)) STORED, col95_9 STRING AS (lower(CAST(col95_6 AS STRING))) VIRTUAL, col95_10 STRING AS (lower(CAST(col95_6 AS STRING))) STORED, col95_11 STRING AS (lower(CAST(col95_6 AS STRING))) VIRTUAL, col95_12 STRING NOT NULL AS (lower(col95_2)) STORED, INDEX (col95_1) STORING (col95_0, col95_3, col95_4, col95_5, col95_8, col95_10, col95_12) NOT VISIBLE, UNIQUE (col95_12 DESC), UNIQUE (col95_0 DESC, lower(col95_5) ASC), UNIQUE (col95_0 DESC, col95_8, col95_4 ASC) STORING (col95_1, col95_2, col95_3, col95_5), INDEX (col95_5 ASC, col95_0 DESC, col95_12 DESC) STORING (col95_1, col95_2, col95_6, col95_8, col95_10) PARTITION BY LIST (col95_5) (PARTITION table95_part_0 VALUES IN (('M':::STRING,), ('"':::STRING,), (e'\x1f':::STRING,), (e'\x0b':::STRING,), ('E':::STRING,), ('s':::STRING,)), PARTITION table95_part_1 VALUES IN ((e'\x1a':::STRING,), (e'\'':::STRING,), (' ':::STRING,), ('X':::STRING,), ('D':::STRING,)), PARTITION table95_part_2 VALUES IN ((e'\x07':::STRING,), (e'\x11':::STRING,), ('*':::STRING,), ('a':::STRING,), ('l':::STRING,), ('p':::STRING,)), PARTITION table95_part_3 VALUES IN (('I':::STRING,), ('B':::STRING,), (':':::STRING,), (')':::STRING,), ('w':::STRING,), ('Z':::STRING,)), PARTITION table95_part_4 VALUES IN ((e'\x18':::STRING,), (e'\x00':::STRING,), ('/':::STRING,), (e'\x12':::STRING,), ('K':::STRING,), (e'\x10':::STRING,), ('4':::STRING,)), PARTITION table95_part_5 VALUES IN (('H':::STRING,), (e'\x14':::STRING,), ('z':::STRING,), ('G':::STRING,)), PARTITION table95_part_6 VALUES IN (('h':::STRING,), ('R':::STRING,), (e'\\':::STRING,), (e'\x1c':::STRING,)), PARTITION "DEFAULT" VALUES IN ((DEFAULT,))), UNIQUE (col95_8) STORING (col95_4, col95_12), INVERTED INDEX (col95_5, col95_1 DESC, col95_10 gin_trgm_ops))
        found first output
        "CREATE TABLE public.table95 (col95_0 \"char\" NOT NULL, col95_1 \"char\" NOT NULL, col95_2 BPCHAR NOT NULL, col95_3 TSVECTOR NOT NULL, col95_4 BIT(27) NOT NULL, col95_5 BPCHAR NOT NULL, col95_6 BIT(49) NULL, col95_7 STRING NOT NULL AS (lower(col95_1)) VIRTUAL, col95_8 STRING NOT NULL AS (lower(col95_0)) STORED, col95_9 STRING NULL AS (lower(CAST(col95_6 AS STRING))) VIRTUAL, col95_10 STRING NULL AS (lower(CAST(col95_6 AS STRING))) STORED, col95_11 STRING NULL AS (lower(CAST(col95_6 AS STRING))) VIRTUAL, col95_12 STRING NOT NULL AS (lower(col95_2)) STORED, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table95_pkey PRIMARY KEY (rowid ASC), INDEX table95_col95_1_idx (col95_1 ASC) STORING (col95_0, col95_3, col95_4, col95_5, col95_8, col95_10, col95_12) NOT VISIBLE, CONSTRAINT table95_col95_12_key UNIQUE (col95_12 DESC), CONSTRAINT table95_col95_0_expr_key UNIQUE (col95_0 DESC, lower(col95_5) ASC), CONSTRAINT table95_col95_0_col95_8_col95_4_key UNIQUE (col95_0 DESC, col95_8 ASC, col95_4 ASC) STORING (col95_1, col95_2, col95_3, col95_5), INDEX table95_col95_5_col95_0_col95_12_idx (col95_5 ASC, col95_0 DESC, col95_12 DESC) STORING (col95_1, col95_2, col95_6, col95_8, col95_10) PARTITION BY LIST (col95_5) (PARTITION table95_part_0 VALUES IN (('M'), ('\"'), (e'\\x1f'), (e'\\x0b'), ('E'), ('s')), PARTITION table95_part_1 VALUES IN ((e'\\x1a'), (e'\\''), (' '), ('X'), ('D')), PARTITION table95_part_2 VALUES IN ((e'\\x07'), (e'\\x11'), ('*'), ('a'), ('l'), ('p')), PARTITION table95_part_3 VALUES IN (('I'), ('B'), (':'), (')'), ('w'), ('Z')), PARTITION table95_part_4 VALUES IN ((e'\\x18'), (e'\\x00'), ('/'), (e'\\x12'), ('K'), (e'\\x10'), ('4')), PARTITION table95_part_5 VALUES IN (('H'), (e'\\x14'), ('z'), ('G')), PARTITION table95_part_6 VALUES IN (('h'), ('R'), (e'\\\\'), (e'\\x1c')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT))), CONSTRAINT table95_col95_8_key UNIQUE (col95_8 ASC) STORING (col95_4, col95_12), INVERTED INDEX table95_col95_5_col95_1_col95_10_idx (col95_5 ASC, col95_1 DESC, col95_10 gin_trgm_ops))"
        but second output
        "CREATE TABLE public.table95 (col95_0 \"char\" NOT NULL, col95_1 \"char\" NOT NULL, col95_2 BPCHAR NOT NULL, col95_3 TSVECTOR NOT NULL, col95_4 BIT(27) NOT NULL, col95_5 BPCHAR NOT NULL, col95_6 BIT(49) NULL, col95_7 STRING NOT NULL AS (lower(col95_1)) VIRTUAL, col95_8 STRING NOT NULL AS (lower(col95_0)) STORED, col95_9 STRING NULL AS (lower(CAST(col95_6 AS STRING))) VIRTUAL, col95_10 STRING NULL AS (lower(CAST(col95_6 AS STRING))) STORED, col95_11 STRING NULL AS (lower(CAST(col95_6 AS STRING))) VIRTUAL, col95_12 STRING NOT NULL AS (lower(col95_2)) STORED, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table95_pkey PRIMARY KEY (rowid ASC), INDEX table95_col95_1_idx (col95_1 ASC) STORING (col95_0, col95_3, col95_4, col95_5, col95_8, col95_10, col95_12) NOT VISIBLE, CONSTRAINT table95_col95_12_key UNIQUE (col95_12 DESC), CONSTRAINT table95_col95_0_expr_key UNIQUE (col95_0 DESC, lower(col95_5) ASC), CONSTRAINT table95_col95_0_col95_8_col95_4_key UNIQUE (col95_0 DESC, col95_8 ASC, col95_4 ASC) STORING (col95_1, col95_2, col95_3, col95_5), INDEX table95_col95_5_col95_0_col95_12_idx (col95_5 ASC, col95_0 DESC, col95_12 DESC) STORING (col95_1, col95_2, col95_6, col95_8, col95_10) PARTITION BY LIST (col95_5) (PARTITION table95_part_0 VALUES IN (('M'), ('\"'), (e'\\x1f'), (e'\\x0b'), ('E'), ('s')), PARTITION table95_part_1 VALUES IN ((e'\\x1a'), (e'\\''), (''), ('X'), ('D')), PARTITION table95_part_2 VALUES IN ((e'\\x07'), (e'\\x11'), ('*'), ('a'), ('l'), ('p')), PARTITION table95_part_3 VALUES IN (('I'), ('B'), (':'), (')'), ('w'), ('Z')), PARTITION table95_part_4 VALUES IN ((e'\\x18'), (e'\\x00'), ('/'), (e'\\x12'), ('K'), (e'\\x10'), ('4')), PARTITION table95_part_5 VALUES IN (('H'), (e'\\x14'), ('z'), ('G')), PARTITION table95_part_6 VALUES IN (('h'), ('R'), (e'\\\\'), (e'\\x1c')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT))), CONSTRAINT table95_col95_8_key UNIQUE (col95_8 ASC) STORING (col95_4, col95_12), INVERTED INDEX table95_col95_5_col95_1_col95_10_idx (col95_5 ASC, col95_1 DESC, col95_10 gin_trgm_ops))"
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema1155140553
--- FAIL: TestCreateRandomSchema (4.10s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://tanzanite.cluster.engflow.com/invocations/default/ff50a083-b10c-44f8-8099-0ac62cb817bd?testReportRun=2&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvc3FsL3Rlc3RzOnRlc3RzX3Rlc3Q=) on master @ [df90f7fbff91224786013aed831359a0e08a4ede](https://github.com/cockroachdb/cockroach/commits/df90f7fbff91224786013aed831359a0e08a4ede):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCreateRandomSchema3138192906
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:107: for input statement
        CREATE TABLE table71 (col71_0 DATE NOT NULL, col71_1 TSVECTOR NOT NULL, col71_2 INT2 NOT NULL, col71_3 VECTOR, col71_4 DATE NOT NULL, col71_5 BPCHAR, col71_6 STRING NOT NULL AS (lower(CAST(col71_1 AS STRING))) VIRTUAL, col71_7 STRING AS (lower(CAST(col71_3 AS STRING))) VIRTUAL, col71_8 STRING NOT NULL AS (lower(CAST(col71_1 AS STRING))) VIRTUAL, col71_9 STRING NOT NULL AS (CASE WHEN col71_4 IS NULL THEN '':::STRING ELSE e'f\x06\x10u':::STRING END) VIRTUAL, col71_10 STRING NOT NULL AS (CASE WHEN col71_0 IS NULL THEN e'MW\x100!\x06^\x05':::STRING ELSE e'X\t7c\x15;':::STRING END) STORED, UNIQUE (col71_4 DESC), UNIQUE (col71_7 DESC), INDEX (col71_5 DESC, col71_4 ASC, col71_0) STORING (col71_1, col71_3, col71_10) PARTITION BY LIST (col71_5, col71_4) (PARTITION table71_part_0 VALUES IN (('7':::STRING, '1972-10-23':::DATE), ('g':::STRING, '1995-12-30':::DATE), ('|':::STRING, '1979-08-01':::DATE), (e'\x07':::STRING, '1993-12-30':::DATE), (e'\U00002603':::STRING, '1982-02-08':::DATE), ('}':::STRING, '4714-11-24 BC':::DATE), ('S':::STRING, '1977-06-06':::DATE), ('I':::STRING, '5874897-12-31':::DATE), ('y':::STRING, '1975-06-23':::DATE), ('d':::STRING, '1974-12-02':::DATE)), PARTITION table71_part_1 VALUES IN (('c':::STRING, '1975-08-31':::DATE), ('*':::STRING, '1992-08-15':::DATE), (e'\x19':::STRING, '1993-01-29':::DATE), (e'\x1b':::STRING, '1988-04-27':::DATE), ('Q':::STRING, '1970-02-14':::DATE), ('l':::STRING, '1972-07-14':::DATE), ('-':::STRING, '1984-09-10':::DATE), ('n':::STRING, '1986-07-17':::DATE), (e'\x1d':::STRING, '1987-09-25':::DATE), ('"':::STRING, '1976-07-29':::DATE)), PARTITION table71_part_2 VALUES IN ((e'\x18':::STRING, '1990-08-03':::DATE), ('~':::STRING, '1989-01-12':::DATE), ('X':::STRING, '1994-04-02':::DATE), (e'\x1a':::STRING, '1981-06-06':::DATE), ('-':::STRING, '1990-05-12':::DATE), ('O':::STRING, '1993-08-19':::DATE), ('e':::STRING, '1981-06-03':::DATE), (' ':::STRING, '1981-06-13':::DATE), ('P':::STRING, '1993-02-20':::DATE), (e'\'':::STRING, '1979-08-11':::DATE)), PARTITION table71_part_3 VALUES IN (('9':::STRING, '1987-07-05':::DATE), ('a':::STRING, '1993-06-02':::DATE), (e'\x0f':::STRING, '1980-12-23':::DATE), ('$':::STRING, '5874897-12-31':::DATE), (e'\x0f':::STRING, '1980-10-06':::DATE), ('H':::STRING, 'infinity':::DATE), ('#':::STRING, '1987-02-07':::DATE), (e'\x1e':::STRING, 'infinity':::DATE), (e'\x0e':::STRING, '1990-10-05':::DATE), (e'\'':::STRING, '1970-01-01':::DATE)), PARTITION table71_part_4 VALUES IN (('C':::STRING, '1994-09-22':::DATE), ('+':::STRING, '1995-04-02':::DATE), ('6':::STRING, '1983-07-16':::DATE), (e'\x06':::STRING, '1976-10-30':::DATE), ('#':::STRING, '1976-11-09':::DATE), ('L':::STRING, '1980-04-25':::DATE), ('|':::STRING, '1974-07-07':::DATE), ('i':::STRING, '1990-05-07':::DATE), ('i':::STRING, '1975-04-21':::DATE), ('v':::STRING, '1982-10-27':::DATE)), PARTITION table71_part_5 VALUES IN (('O':::STRING, '1980-06-20':::DATE), ('^':::STRING, '1977-01-25':::DATE), ('"':::STRING, '1979-10-31':::DATE), ('"':::STRING, '1985-10-24':::DATE), ('$':::STRING, '1988-07-03':::DATE), ('g':::STRING, '1992-09-23':::DATE), ('p':::STRING, '1979-03-24':::DATE), (e'\x06':::STRING, '1973-10-14':::DATE), (e'\n':::STRING, '1973-05-06':::DATE)), PARTITION table71_part_6 VALUES IN (('%':::STRING, '1970-01-01':::DATE), ('d':::STRING, '1976-08-03':::DATE), (e'\'':::STRING, '1982-01-11':::DATE), ('h':::STRING, '1970-12-05':::DATE), ('h':::STRING, '1981-03-07':::DATE), (e'\x1e':::STRING, '1975-12-23':::DATE), ('``':::STRING, '1988-06-24':::DATE), ('':::STRING, '1981-11-18':::DATE), (e'\t':::STRING, '1986-08-31':::DATE), ('D':::STRING, '1979-12-08':::DATE)), PARTITION table71_part_7 VALUES IN (('x':::STRING, '4714-11-24 BC':::DATE), ('!':::STRING, '1972-02-16':::DATE), (e'\x7f':::STRING, '1970-08-23':::DATE), (')':::STRING, '1971-10-19':::DATE), ('f':::STRING, '1986-06-13':::DATE), ('``':::STRING, '1983-04-28':::DATE), (e'\x18':::STRING, '1983-09-12':::DATE), ('A':::STRING, '1978-05-13':::DATE), ('B':::STRING, '1970-01-01':::DATE), ('?':::STRING, '1990-07-31':::DATE)), PARTITION table71_part_8 VALUES IN (('m':::STRING, '1971-12-20':::DATE), (e'\x14':::STRING, '1975-03-25':::DATE), ('x':::STRING, '1977-05-19':::DATE), (';':::STRING, '1984-06-05':::DATE), (e'\x00':::STRING, '1975-11-24':::DATE), ('z':::STRING, '-infinity':::DATE), (e'\x02':::STRING, '1995-08-22':::DATE), ('k':::STRING, '1975-04-30':::DATE), ('/':::STRING, '1979-08-26':::DATE), (e'\x1b':::STRING, 'infinity':::DATE)), PARTITION table71_part_9 VALUES IN (('x':::STRING, '1996-04-13':::DATE), ('J':::STRING, '1980-03-14':::DATE), ('j':::STRING, '1994-07-07':::DATE), ('n':::STRING, '1984-04-06':::DATE), (e'\x1e':::STRING, '1984-07-18':::DATE), ('':::STRING, '5874897-12-31':::DATE), ('%':::STRING, '1991-12-09':::DATE), (e'\x03':::STRING, '1981-04-05':::DATE), ('2':::STRING, '1987-11-21':::DATE), ('``':::STRING, '5874897-12-31':::DATE)), PARTITION "DEFAULT" VALUES IN ((DEFAULT, DEFAULT))), INDEX (col71_5 ASC, col71_6) PARTITION BY LIST (col71_5, col71_6) (PARTITION table71_part_0 VALUES IN (('~':::STRING, e'pf\fKQ~{\x04\'':::STRING), (e'\U00002603':::STRING, e's6\x1ai':::STRING), ('3':::STRING, '':::STRING)), PARTITION table71_part_1 VALUES IN ((e'\x0f':::STRING, e'e\x1f$':::STRING), ('':::STRING, 'a^6A;':::STRING), ('B':::STRING, e'\tG4N\x12%':::STRING)), PARTITION table71_part_2 VALUES IN (('a':::STRING, '':::STRING), (e'\x02':::STRING, e'>\f':::STRING), ('M':::STRING, e'$(\x15\x04O\r':::STRING)), PARTITION table71_part_3 VALUES IN (('x':::STRING, e'- \x144\x0b\x17\r/':::STRING), (e'\x16':::STRING, e'\\\'N\x05':::STRING), ('p':::STRING, e'-\x06IdN\x07V_8':::STRING)), PARTITION table71_part_4 VALUES IN (('N':::STRING, e'\x1f2\x7f\x0f\x12eMD=':::STRING), ('':::STRING, e'\'':::STRING), ('{':::STRING, e'\x14E``*':::STRING)), PARTITION table71_part_5 VALUES IN (('Q':::STRING, e'rn3/}h\'':::STRING), (e'\x1c':::STRING, '&AFC':::STRING), (',':::STRING, e'\b\x0fl\bFy\x1c':::STRING)), PARTITION table71_part_6 VALUES IN (('X':::STRING, e'{\x16\x11b;7~4':::STRING), ('*':::STRING, e'sQN}n\x10``\x0bw':::STRING), ('?':::STRING, 'Xjt2':::STRING)), PARTITION "DEFAULT" VALUES IN ((DEFAULT, DEFAULT))), INDEX (col71_6), UNIQUE (col71_4, col71_8 ASC, col71_10), INDEX (col71_6, col71_8, col71_10) STORING (col71_1, col71_3, col71_5))
        found first output
        "CREATE TABLE public.table71 (col71_0 DATE NOT NULL, col71_1 TSVECTOR NOT NULL, col71_2 INT2 NOT NULL, col71_3 VECTOR NULL, col71_4 DATE NOT NULL, col71_5 BPCHAR NULL, col71_6 STRING NOT NULL AS (lower(CAST(col71_1 AS STRING))) VIRTUAL, col71_7 STRING NULL AS (lower(CAST(col71_3 AS STRING))) VIRTUAL, col71_8 STRING NOT NULL AS (lower(CAST(col71_1 AS STRING))) VIRTUAL, col71_9 STRING NOT NULL AS (CASE WHEN col71_4 IS NULL THEN '':::STRING ELSE e'f\\x06\\x10u':::STRING END) VIRTUAL, col71_10 STRING NOT NULL AS (CASE WHEN col71_0 IS NULL THEN e'MW\\x100!\\x06^\\x05':::STRING ELSE e'X\\t7c\\x15;':::STRING END) STORED, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table71_pkey PRIMARY KEY (rowid ASC), CONSTRAINT table71_col71_4_key UNIQUE (col71_4 DESC), CONSTRAINT table71_col71_7_key UNIQUE (col71_7 DESC), INDEX table71_col71_5_col71_4_col71_0_idx (col71_5 DESC, col71_4 ASC, col71_0 ASC) STORING (col71_1, col71_3, col71_10) PARTITION BY LIST (col71_5, col71_4) (PARTITION table71_part_0 VALUES IN (('7', '1972-10-23'), ('g', '1995-12-30'), ('|', '1979-08-01'), (e'\\x07', '1993-12-30'), (e'\\U00002603', '1982-02-08'), ('}', '4714-11-24 BC'), ('S', '1977-06-06'), ('I', '5874897-12-31'), ('y', '1975-06-23'), ('d', '1974-12-02')), PARTITION table71_part_1 VALUES IN (('c', '1975-08-31'), ('*', '1992-08-15'), (e'\\x19', '1993-01-29'), (e'\\x1b', '1988-04-27'), ('Q', '1970-02-14'), ('l', '1972-07-14'), ('-', '1984-09-10'), ('n', '1986-07-17'), (e'\\x1d', '1987-09-25'), ('\"', '1976-07-29')), PARTITION table71_part_2 VALUES IN ((e'\\x18', '1990-08-03'), ('~', '1989-01-12'), ('X', '1994-04-02'), (e'\\x1a', '1981-06-06'), ('-', '1990-05-12'), ('O', '1993-08-19'), ('e', '1981-06-03'), (' ', '1981-06-13'), ('P', '1993-02-20'), (e'\\'', '1979-08-11')), PARTITION table71_part_3 VALUES IN (('9', '1987-07-05'), ('a', '1993-06-02'), (e'\\x0f', '1980-12-23'), ('$', '5874897-12-31'), (e'\\x0f', '1980-10-06'), ('H', 'infinity'), ('#', '1987-02-07'), (e'\\x1e', 'infinity'), (e'\\x0e', '1990-10-05'), (e'\\'', '1970-01-01')), PARTITION table71_part_4 VALUES IN (('C', '1994-09-22'), ('+', '1995-04-02'), ('6', '1983-07-16'), (e'\\x06', '1976-10-30'), ('#', '1976-11-09'), ('L', '1980-04-25'), ('|', '1974-07-07'), ('i', '1990-05-07'), ('i', '1975-04-21'), ('v', '1982-10-27')), PARTITION table71_part_5 VALUES IN (('O', '1980-06-20'), ('^', '1977-01-25'), ('\"', '1979-10-31'), ('\"', '1985-10-24'), ('$', '1988-07-03'), ('g', '1992-09-23'), ('p', '1979-03-24'), (e'\\x06', '1973-10-14'), (e'\\n', '1973-05-06')), PARTITION table71_part_6 VALUES IN (('%', '1970-01-01'), ('d', '1976-08-03'), (e'\\'', '1982-01-11'), ('h', '1970-12-05'), ('h', '1981-03-07'), (e'\\x1e', '1975-12-23'), ('``', '1988-06-24'), ('', '1981-11-18'), (e'\\t', '1986-08-31'), ('D', '1979-12-08')), PARTITION table71_part_7 VALUES IN (('x', '4714-11-24 BC'), ('!', '1972-02-16'), (e'\\x7f', '1970-08-23'), (')', '1971-10-19'), ('f', '1986-06-13'), ('``', '1983-04-28'), (e'\\x18', '1983-09-12'), ('A', '1978-05-13'), ('B', '1970-01-01'), ('?', '1990-07-31')), PARTITION table71_part_8 VALUES IN (('m', '1971-12-20'), (e'\\x14', '1975-03-25'), ('x', '1977-05-19'), (';', '1984-06-05'), (e'\\x00', '1975-11-24'), ('z', '-infinity'), (e'\\x02', '1995-08-22'), ('k', '1975-04-30'), ('/', '1979-08-26'), (e'\\x1b', 'infinity')), PARTITION table71_part_9 VALUES IN (('x', '1996-04-13'), ('J', '1980-03-14'), ('j', '1994-07-07'), ('n', '1984-04-06'), (e'\\x1e', '1984-07-18'), ('', '5874897-12-31'), ('%', '1991-12-09'), (e'\\x03', '1981-04-05'), ('2', '1987-11-21'), ('``', '5874897-12-31')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT))), INDEX table71_col71_5_col71_6_idx (col71_5 ASC, col71_6 ASC) PARTITION BY LIST (col71_5, col71_6) (PARTITION table71_part_0 VALUES IN (('~', e'pf\\fKQ~{\\x04\\''), (e'\\U00002603', e's6\\x1ai'), ('3', '')), PARTITION table71_part_1 VALUES IN ((e'\\x0f', e'e\\x1f$'), ('', 'a^6A;'), ('B', e'\\tG4N\\x12%')), PARTITION table71_part_2 VALUES IN (('a', ''), (e'\\x02', e'>\\f'), ('M', e'$(\\x15\\x04O\\r')), PARTITION table71_part_3 VALUES IN (('x', e'- \\x144\\x0b\\x17\\r/'), (e'\\x16', e'\\\\\\'N\\x05'), ('p', e'-\\x06IdN\\x07V_8')), PARTITION table71_part_4 VALUES IN (('N', e'\\x1f2\\x7f\\x0f\\x12eMD='), ('', e'\\''), ('{', e'\\x14E``*')), PARTITION table71_part_5 VALUES IN (('Q', e'rn3/}h\\''), (e'\\x1c', '&AFC'), (',', e'\\b\\x0fl\\bFy\\x1c')), PARTITION table71_part_6 VALUES IN (('X', e'{\\x16\\x11b;7~4'), ('*', e'sQN}n\\x10``\\x0bw'), ('?', 'Xjt2')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT))), INDEX table71_col71_6_idx (col71_6 ASC), CONSTRAINT table71_col71_4_col71_8_col71_10_key UNIQUE (col71_4 ASC, col71_8 ASC, col71_10 ASC), INDEX table71_col71_6_col71_8_col71_10_idx (col71_6 ASC, col71_8 ASC, col71_10 ASC) STORING (col71_1, col71_3, col71_5))"
        but second output
        "CREATE TABLE public.table71 (col71_0 DATE NOT NULL, col71_1 TSVECTOR NOT NULL, col71_2 INT2 NOT NULL, col71_3 VECTOR NULL, col71_4 DATE NOT NULL, col71_5 BPCHAR NULL, col71_6 STRING NOT NULL AS (lower(CAST(col71_1 AS STRING))) VIRTUAL, col71_7 STRING NULL AS (lower(CAST(col71_3 AS STRING))) VIRTUAL, col71_8 STRING NOT NULL AS (lower(CAST(col71_1 AS STRING))) VIRTUAL, col71_9 STRING NOT NULL AS (CASE WHEN col71_4 IS NULL THEN '':::STRING ELSE e'f\\x06\\x10u':::STRING END) VIRTUAL, col71_10 STRING NOT NULL AS (CASE WHEN col71_0 IS NULL THEN e'MW\\x100!\\x06^\\x05':::STRING ELSE e'X\\t7c\\x15;':::STRING END) STORED, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table71_pkey PRIMARY KEY (rowid ASC), CONSTRAINT table71_col71_4_key UNIQUE (col71_4 DESC), CONSTRAINT table71_col71_7_key UNIQUE (col71_7 DESC), INDEX table71_col71_5_col71_4_col71_0_idx (col71_5 DESC, col71_4 ASC, col71_0 ASC) STORING (col71_1, col71_3, col71_10) PARTITION BY LIST (col71_5, col71_4) (PARTITION table71_part_0 VALUES IN (('7', '1972-10-23'), ('g', '1995-12-30'), ('|', '1979-08-01'), (e'\\x07', '1993-12-30'), (e'\\U00002603', '1982-02-08'), ('}', '4714-11-24 BC'), ('S', '1977-06-06'), ('I', '5874897-12-31'), ('y', '1975-06-23'), ('d', '1974-12-02')), PARTITION table71_part_1 VALUES IN (('c', '1975-08-31'), ('*', '1992-08-15'), (e'\\x19', '1993-01-29'), (e'\\x1b', '1988-04-27'), ('Q', '1970-02-14'), ('l', '1972-07-14'), ('-', '1984-09-10'), ('n', '1986-07-17'), (e'\\x1d', '1987-09-25'), ('\"', '1976-07-29')), PARTITION table71_part_2 VALUES IN ((e'\\x18', '1990-08-03'), ('~', '1989-01-12'), ('X', '1994-04-02'), (e'\\x1a', '1981-06-06'), ('-', '1990-05-12'), ('O', '1993-08-19'), ('e', '1981-06-03'), ('', '1981-06-13'), ('P', '1993-02-20'), (e'\\'', '1979-08-11')), PARTITION table71_part_3 VALUES IN (('9', '1987-07-05'), ('a', '1993-06-02'), (e'\\x0f', '1980-12-23'), ('$', '5874897-12-31'), (e'\\x0f', '1980-10-06'), ('H', 'infinity'), ('#', '1987-02-07'), (e'\\x1e', 'infinity'), (e'\\x0e', '1990-10-05'), (e'\\'', '1970-01-01')), PARTITION table71_part_4 VALUES IN (('C', '1994-09-22'), ('+', '1995-04-02'), ('6', '1983-07-16'), (e'\\x06', '1976-10-30'), ('#', '1976-11-09'), ('L', '1980-04-25'), ('|', '1974-07-07'), ('i', '1990-05-07'), ('i', '1975-04-21'), ('v', '1982-10-27')), PARTITION table71_part_5 VALUES IN (('O', '1980-06-20'), ('^', '1977-01-25'), ('\"', '1979-10-31'), ('\"', '1985-10-24'), ('$', '1988-07-03'), ('g', '1992-09-23'), ('p', '1979-03-24'), (e'\\x06', '1973-10-14'), (e'\\n', '1973-05-06')), PARTITION table71_part_6 VALUES IN (('%', '1970-01-01'), ('d', '1976-08-03'), (e'\\'', '1982-01-11'), ('h', '1970-12-05'), ('h', '1981-03-07'), (e'\\x1e', '1975-12-23'), ('``', '1988-06-24'), ('', '1981-11-18'), (e'\\t', '1986-08-31'), ('D', '1979-12-08')), PARTITION table71_part_7 VALUES IN (('x', '4714-11-24 BC'), ('!', '1972-02-16'), (e'\\x7f', '1970-08-23'), (')', '1971-10-19'), ('f', '1986-06-13'), ('``', '1983-04-28'), (e'\\x18', '1983-09-12'), ('A', '1978-05-13'), ('B', '1970-01-01'), ('?', '1990-07-31')), PARTITION table71_part_8 VALUES IN (('m', '1971-12-20'), (e'\\x14', '1975-03-25'), ('x', '1977-05-19'), (';', '1984-06-05'), (e'\\x00', '1975-11-24'), ('z', '-infinity'), (e'\\x02', '1995-08-22'), ('k', '1975-04-30'), ('/', '1979-08-26'), (e'\\x1b', 'infinity')), PARTITION table71_part_9 VALUES IN (('x', '1996-04-13'), ('J', '1980-03-14'), ('j', '1994-07-07'), ('n', '1984-04-06'), (e'\\x1e', '1984-07-18'), ('', '5874897-12-31'), ('%', '1991-12-09'), (e'\\x03', '1981-04-05'), ('2', '1987-11-21'), ('``', '5874897-12-31')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT))), INDEX table71_col71_5_col71_6_idx (col71_5 ASC, col71_6 ASC) PARTITION BY LIST (col71_5, col71_6) (PARTITION table71_part_0 VALUES IN (('~', e'pf\\fKQ~{\\x04\\''), (e'\\U00002603', e's6\\x1ai'), ('3', '')), PARTITION table71_part_1 VALUES IN ((e'\\x0f', e'e\\x1f$'), ('', 'a^6A;'), ('B', e'\\tG4N\\x12%')), PARTITION table71_part_2 VALUES IN (('a', ''), (e'\\x02', e'>\\f'), ('M', e'$(\\x15\\x04O\\r')), PARTITION table71_part_3 VALUES IN (('x', e'- \\x144\\x0b\\x17\\r/'), (e'\\x16', e'\\\\\\'N\\x05'), ('p', e'-\\x06IdN\\x07V_8')), PARTITION table71_part_4 VALUES IN (('N', e'\\x1f2\\x7f\\x0f\\x12eMD='), ('', e'\\''), ('{', e'\\x14E``*')), PARTITION table71_part_5 VALUES IN (('Q', e'rn3/}h\\''), (e'\\x1c', '&AFC'), (',', e'\\b\\x0fl\\bFy\\x1c')), PARTITION table71_part_6 VALUES IN (('X', e'{\\x16\\x11b;7~4'), ('*', e'sQN}n\\x10``\\x0bw'), ('?', 'Xjt2')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT))), INDEX table71_col71_6_idx (col71_6 ASC), CONSTRAINT table71_col71_4_col71_8_col71_10_key UNIQUE (col71_4 ASC, col71_8 ASC, col71_10 ASC), INDEX table71_col71_6_col71_8_col71_10_idx (col71_6 ASC, col71_8 ASC, col71_10 ASC) STORING (col71_1, col71_3, col71_5))"
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema3138192906
--- FAIL: TestCreateRandomSchema (33.64s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://tanzanite.cluster.engflow.com/invocations/default/98a4e2b6-734f-477d-b763-6cb4a934984d?testReportRun=1&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvc3FsL3Rlc3RzOnRlc3RzX3Rlc3Q=) on master @ [ea4644b040dd4503f2eb7292cfebc31a58fd16fb](https://github.com/cockroachdb/cockroach/commits/ea4644b040dd4503f2eb7292cfebc31a58fd16fb):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCreateRandomSchema1300442983
    test_log_scope.go:76: use -show-logs to present logs inline
    random_schema_test.go:107: for input statement
        CREATE TABLE table92 (col92_0 INT4 NOT NULL, col92_1 TSQUERY NOT NULL, col92_2 TIMESTAMP NOT NULL, col92_3 TIMESTAMP, col92_4 INTERVAL NOT NULL, col92_5 BPCHAR, col92_6 STRING AS (CASE WHEN col92_3 IS NULL THEN 'q(':::STRING ELSE e'..y2\r!-v':::STRING END) STORED, col92_7 STRING AS (CASE WHEN col92_3 IS NULL THEN e'\U00002603':::STRING ELSE e'\x00':::STRING END) VIRTUAL, INDEX (col92_5 DESC) STORING (col92_0, col92_1, col92_3, col92_4, col92_6) PARTITION BY LIST (col92_5) (PARTITION table92_part_0 VALUES IN (('"':::STRING,)), PARTITION table92_part_1 VALUES IN ((e'\x0b':::STRING,)), PARTITION table92_part_2 VALUES IN ((e'\x1a':::STRING,)), PARTITION table92_part_3 VALUES IN ((e'\x03':::STRING,)), PARTITION table92_part_5 VALUES IN (('R':::STRING,)), PARTITION table92_part_6 VALUES IN (('6':::STRING,)), PARTITION table92_part_8 VALUES IN (('~':::STRING,)), PARTITION table92_part_9 VALUES IN ((' ':::STRING,)), PARTITION "DEFAULT" VALUES IN ((DEFAULT,))), UNIQUE (col92_4 ASC) STORING (col92_0, col92_6), INVERTED INDEX (col92_6 gin_trgm_ops), UNIQUE (col92_5) STORING (col92_0, col92_1), UNIQUE (col92_7 DESC))
        found first output
        "CREATE TABLE public.table92 (col92_0 INT4 NOT NULL, col92_1 TSQUERY NOT NULL, col92_2 TIMESTAMP NOT NULL, col92_3 TIMESTAMP NULL, col92_4 INTERVAL NOT NULL, col92_5 BPCHAR NULL, col92_6 STRING NULL AS (CASE WHEN col92_3 IS NULL THEN 'q(':::STRING ELSE e'..y2\\r!-v':::STRING END) STORED, col92_7 STRING NULL AS (CASE WHEN col92_3 IS NULL THEN e'\\U00002603':::STRING ELSE e'\\x00':::STRING END) VIRTUAL, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table92_pkey PRIMARY KEY (rowid ASC), INDEX table92_col92_5_idx (col92_5 DESC) STORING (col92_0, col92_1, col92_3, col92_4, col92_6) PARTITION BY LIST (col92_5) (PARTITION table92_part_0 VALUES IN (('\"')), PARTITION table92_part_1 VALUES IN ((e'\\x0b')), PARTITION table92_part_2 VALUES IN ((e'\\x1a')), PARTITION table92_part_3 VALUES IN ((e'\\x03')), PARTITION table92_part_5 VALUES IN (('R')), PARTITION table92_part_6 VALUES IN (('6')), PARTITION table92_part_8 VALUES IN (('~')), PARTITION table92_part_9 VALUES IN ((' ')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT))), CONSTRAINT table92_col92_4_key UNIQUE (col92_4 ASC) STORING (col92_0, col92_6), INVERTED INDEX table92_col92_6_idx (col92_6 gin_trgm_ops), CONSTRAINT table92_col92_5_key UNIQUE (col92_5 ASC) STORING (col92_0, col92_1), CONSTRAINT table92_col92_7_key UNIQUE (col92_7 DESC))"
        but second output
        "CREATE TABLE public.table92 (col92_0 INT4 NOT NULL, col92_1 TSQUERY NOT NULL, col92_2 TIMESTAMP NOT NULL, col92_3 TIMESTAMP NULL, col92_4 INTERVAL NOT NULL, col92_5 BPCHAR NULL, col92_6 STRING NULL AS (CASE WHEN col92_3 IS NULL THEN 'q(':::STRING ELSE e'..y2\\r!-v':::STRING END) STORED, col92_7 STRING NULL AS (CASE WHEN col92_3 IS NULL THEN e'\\U00002603':::STRING ELSE e'\\x00':::STRING END) VIRTUAL, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table92_pkey PRIMARY KEY (rowid ASC), INDEX table92_col92_5_idx (col92_5 DESC) STORING (col92_0, col92_1, col92_3, col92_4, col92_6) PARTITION BY LIST (col92_5) (PARTITION table92_part_0 VALUES IN (('\"')), PARTITION table92_part_1 VALUES IN ((e'\\x0b')), PARTITION table92_part_2 VALUES IN ((e'\\x1a')), PARTITION table92_part_3 VALUES IN ((e'\\x03')), PARTITION table92_part_5 VALUES IN (('R')), PARTITION table92_part_6 VALUES IN (('6')), PARTITION table92_part_8 VALUES IN (('~')), PARTITION table92_part_9 VALUES IN (('')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT))), CONSTRAINT table92_col92_4_key UNIQUE (col92_4 ASC) STORING (col92_0, col92_6), INVERTED INDEX table92_col92_6_idx (col92_6 gin_trgm_ops), CONSTRAINT table92_col92_5_key UNIQUE (col92_5 ASC) STORING (col92_0, col92_1), CONSTRAINT table92_col92_7_key UNIQUE (col92_7 DESC))"
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestCreateRandomSchema1300442983
--- FAIL: TestCreateRandomSchema (6.00s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17732152?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17732152?buildTab=artifacts#/) on master @ [2520c51663e57de54706a58e8e68370467dbe871](https://github.com/cockroachdb/cockroach/commits/2520c51663e57de54706a58e8e68370467dbe871):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema2842434747
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    random_schema_test.go:90: CREATE TABLE public.table12 (col12_0 BPCHAR NOT NULL, col12_1 NAME NOT NULL, col12_2 TIMESTAMP NOT NULL, col12_3 STRING NOT NULL AS (CASE WHEN col12_2 IS NULL THEN e' M\x0bNZF\'':::STRING ELSE e'\fqv\x18a\x16':::STRING END) STORED, CONSTRAINT table12_pkey PRIMARY KEY (col12_2 ASC, col12_0 ASC, col12_1 ASC), INDEX table12_col12_3_col12_2_idx (col12_3 ASC, col12_2 DESC), INDEX table12_col12_0_idx (col12_0 ASC) PARTITION BY LIST (col12_0) (PARTITION table12_part_0 VALUES IN ((e'\x14'), (''), ('Z'), (' '), (e'\x04'), ('(')), PARTITION table12_part_1 VALUES IN (('&'), (e'\t'), ('2'), ('"'), ('p'), ('$')), PARTITION table12_part_2 VALUES IN (('G'), ('X'), ('i'), ('@'), ('F')), PARTITION table12_part_3 VALUES IN (('['), ('%'), (e'\x07'), (';'), ('z'), ('u')), PARTITION table12_part_4 VALUES IN (('y'), (e'\U00002603'), ('9'), ('~'), (e'\r'), ('7'))), CONSTRAINT table12_col12_1_expr_col12_2_key UNIQUE (col12_1 ASC, lower(col12_0) DESC, col12_2 ASC), CONSTRAINT table12_col12_0_key UNIQUE (col12_0 ASC) STORING (col12_3), CONSTRAINT table12_col12_0_expr_col12_3_key UNIQUE (col12_0 DESC, lower(col12_1) DESC, col12_3 ASC), INDEX table12_col12_2_idx (col12_2 DESC), CONSTRAINT table12_col12_1_key UNIQUE (col12_1 DESC), FAMILY fam_0_col12_2 (col12_2), FAMILY fam_1_col12_1_col12_3 (col12_1, col12_3), FAMILY fam_2_col12_0 (col12_0)) pq: relation "table12" (133): ('') cannot be present in more than one partition
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema2842434747
--- FAIL: TestCreateRandomSchema (1.49s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17732580?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17732580?buildTab=artifacts#/) on master @ [2520c51663e57de54706a58e8e68370467dbe871](https://github.com/cockroachdb/cockroach/commits/2520c51663e57de54706a58e8e68370467dbe871):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema1206636766
    test_log_scope.go:76: use -show-logs to present logs inline
    random_schema_test.go:107: for input statement
        CREATE TABLE table54 (col54_0 JSONB NOT NULL, col54_1 REGCLASS NULL, col54_2 VECTOR NOT NULL, col54_3 BIT(35), col54_4 BPCHAR, col54_5 PG_LSN NOT NULL, col54_6 TSQUERY NOT NULL, col54_7 DECIMAL, col54_8 DATE NOT NULL, col54_9 BYTES NOT NULL, col54_10 BOOL NOT NULL, col54_11 BIT(41) NOT NULL, col54_12 REGCLASS, col54_13 TIMESTAMP NOT NULL, col54_14 INET NOT NULL, col54_15 STRING NOT NULL AS (lower(CAST(col54_5 AS STRING))) STORED, INDEX (col54_14 DESC) STORING (col54_1, col54_2, col54_3, col54_5, col54_8, col54_9, col54_10, col54_11, col54_12, col54_15) NOT VISIBLE, INDEX (col54_4 DESC, col54_11 DESC) PARTITION BY LIST (col54_4, col54_11) (PARTITION table54_part_0 VALUES IN (('U':::STRING, B'11111110011001110011011011100101010011111'), ('N':::STRING, B'00010011010001100011101110111010010111000'), (e'\U00002603':::STRING, B'10100000101011111001110001000111110100111'), ('(':::STRING, B'11000000010001111000000101010100111110000'), ('d':::STRING, B'10011010010011111110111000100110101110011'), ('J':::STRING, B'11101101011001110111010111000010010011101'), (e'\x00':::STRING, B'01010010100010001101100101000111000100010'), ('|':::STRING, B'00111111010100111000111100111110001000100'), (e'\x00':::STRING, B'00111000111011010010101001110100000100011')), PARTITION table54_part_1 VALUES IN ((e'\x0b':::STRING, B'01111111111111111111111111111111111111111'), ('k':::STRING, B'10011110001001111011010010110100010101010'), (e'\x00':::STRING, B'11101010000011000001010011000001010101110'), ('y':::STRING, B'01001010011101101101100010010001110110111'), ('}':::STRING, B'11001000001100011010111000000110000010010'), ('7':::STRING, B'00011001101001110111011001001010101110000'), (')':::STRING, B'00111001101000010111000101100110010100010'), (e'\x14':::STRING, B'11101100001010101011001001011001001000001'), (e'\x10':::STRING, B'10000001000000101010000110100010011000111')), PARTITION table54_part_2 VALUES IN (('':::STRING, B'11001110111110101000111010001001100101101'), ('a':::STRING, B'10011001100111010110111101101101110101000'), ('*':::STRING, B'01101001100111101100001110011001000110110'), (')':::STRING, B'01100010010001010011110001100000011110101'), ('M':::STRING, B'00000000000000000000000000000000000000000'), ('Y':::STRING, B'01001010001001110100111010111000000000100'), ('}':::STRING, B'11101001101010001101101011001010101000100'), (e'\x0f':::STRING, B'01000101101010101011001111100111100111100'), ('"':::STRING, B'10001010011110010101000100110000100110011')), PARTITION table54_part_3 VALUES IN ((e'\x00':::STRING, B'01001011100111110001101100000100101010101'), ('A':::STRING, B'11001111000010110010000010001001100110100'), ('#':::STRING, B'11001011110110100000001110011100010101010'), ('M':::STRING, B'01110110011100010111110100111100101111000'), ('w':::STRING, B'11111100010101000000111010111000110111011'), ('5':::STRING, B'00010110011110010011011010000111111101011'), ('4':::STRING, B'00000000000000000000000000000000000000000'), (e'\x00':::STRING, B'10000111000101010110011000011001100101001'), ('W':::STRING, B'00101100001101111000010110100100000001101')), PARTITION table54_part_4 VALUES IN (('"':::STRING, B'01011000110010011000110100111001110101111'), ('r':::STRING, B'00000100100010100010000011010011010111110'), ('(':::STRING, B'11101011100111011011101001101010001111100'), (e'\'':::STRING, B'00000000000000000000000000000000000000000'), ('+':::STRING, B'01111110011110111010000010100011010110001'), ('x':::STRING, B'00000000000000000000000000000000000000000'), (' ':::STRING, B'10111001110101100000111010010101110100111'), ('y':::STRING, B'10001000100101000100001100001100101000100'), (e'\n':::STRING, B'00001100000100110001000101111101110001101')), PARTITION table54_part_5 VALUES IN (('_':::STRING, B'11001010010101001000010011110101110111101'), (e'\x17':::STRING, B'00000000000000000000000000000000000000000'), ('Y':::STRING, B'10100010101010010111100001000010001110001'), ('&':::STRING, B'01111000000100111011101010111010011010101'), ('^':::STRING, B'01111111111111111111111111111111111111111'), ('@':::STRING, B'00111010001110011011011000110000011000010'), (')':::STRING, B'11001011000011001011011101101011110111001'), ('>':::STRING, B'10001001101101101100000011101100110110001'), ('X':::STRING, B'11010110011100111011111001111010001110011')), PARTITION "DEFAULT" VALUES IN ((DEFAULT, DEFAULT))), INDEX (col54_11))
        found first output
        "CREATE TABLE public.table54 (col54_0 JSONB NOT NULL, col54_1 REGCLASS NULL, col54_2 VECTOR NOT NULL, col54_3 BIT(35) NULL, col54_4 BPCHAR NULL, col54_5 PG_LSN NOT NULL, col54_6 TSQUERY NOT NULL, col54_7 DECIMAL NULL, col54_8 DATE NOT NULL, col54_9 BYTES NOT NULL, col54_10 BOOL NOT NULL, col54_11 BIT(41) NOT NULL, col54_12 REGCLASS NULL, col54_13 TIMESTAMP NOT NULL, col54_14 INET NOT NULL, col54_15 STRING NOT NULL AS (lower(CAST(col54_5 AS STRING))) STORED, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table54_pkey PRIMARY KEY (rowid ASC), INDEX table54_col54_14_idx (col54_14 DESC) STORING (col54_1, col54_2, col54_3, col54_5, col54_8, col54_9, col54_10, col54_11, col54_12, col54_15) NOT VISIBLE, INDEX table54_col54_4_col54_11_idx (col54_4 DESC, col54_11 DESC) PARTITION BY LIST (col54_4, col54_11) (PARTITION table54_part_0 VALUES IN (('U', B'11111110011001110011011011100101010011111'), ('N', B'00010011010001100011101110111010010111000'), (e'\\U00002603', B'10100000101011111001110001000111110100111'), ('(', B'11000000010001111000000101010100111110000'), ('d', B'10011010010011111110111000100110101110011'), ('J', B'11101101011001110111010111000010010011101'), (e'\\x00', B'01010010100010001101100101000111000100010'), ('|', B'00111111010100111000111100111110001000100'), (e'\\x00', B'00111000111011010010101001110100000100011')), PARTITION table54_part_1 VALUES IN ((e'\\x0b', B'01111111111111111111111111111111111111111'), ('k', B'10011110001001111011010010110100010101010'), (e'\\x00', B'11101010000011000001010011000001010101110'), ('y', B'01001010011101101101100010010001110110111'), ('}', B'11001000001100011010111000000110000010010'), ('7', B'00011001101001110111011001001010101110000'), (')', B'00111001101000010111000101100110010100010'), (e'\\x14', B'11101100001010101011001001011001001000001'), (e'\\x10', B'10000001000000101010000110100010011000111')), PARTITION table54_part_2 VALUES IN (('', B'11001110111110101000111010001001100101101'), ('a', B'10011001100111010110111101101101110101000'), ('*', B'01101001100111101100001110011001000110110'), (')', B'01100010010001010011110001100000011110101'), ('M', B'00000000000000000000000000000000000000000'), ('Y', B'01001010001001110100111010111000000000100'), ('}', B'11101001101010001101101011001010101000100'), (e'\\x0f', B'01000101101010101011001111100111100111100'), ('\"', B'10001010011110010101000100110000100110011')), PARTITION table54_part_3 VALUES IN ((e'\\x00', B'01001011100111110001101100000100101010101'), ('A', B'11001111000010110010000010001001100110100'), ('#', B'11001011110110100000001110011100010101010'), ('M', B'01110110011100010111110100111100101111000'), ('w', B'11111100010101000000111010111000110111011'), ('5', B'00010110011110010011011010000111111101011'), ('4', B'00000000000000000000000000000000000000000'), (e'\\x00', B'10000111000101010110011000011001100101001'), ('W', B'00101100001101111000010110100100000001101')), PARTITION table54_part_4 VALUES IN (('\"', B'01011000110010011000110100111001110101111'), ('r', B'00000100100010100010000011010011010111110'), ('(', B'11101011100111011011101001101010001111100'), (e'\\'', B'00000000000000000000000000000000000000000'), ('+', B'01111110011110111010000010100011010110001'), ('x', B'00000000000000000000000000000000000000000'), (' ', B'10111001110101100000111010010101110100111'), ('y', B'10001000100101000100001100001100101000100'), (e'\\n', B'00001100000100110001000101111101110001101')), PARTITION table54_part_5 VALUES IN (('_', B'11001010010101001000010011110101110111101'), (e'\\x17', B'00000000000000000000000000000000000000000'), ('Y', B'10100010101010010111100001000010001110001'), ('&', B'01111000000100111011101010111010011010101'), ('^', B'01111111111111111111111111111111111111111'), ('@', B'00111010001110011011011000110000011000010'), (')', B'11001011000011001011011101101011110111001'), ('>', B'10001001101101101100000011101100110110001'), ('X', B'11010110011100111011111001111010001110011')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT))), INDEX table54_col54_11_idx (col54_11 ASC))"
        but second output
        "CREATE TABLE public.table54 (col54_0 JSONB NOT NULL, col54_1 REGCLASS NULL, col54_2 VECTOR NOT NULL, col54_3 BIT(35) NULL, col54_4 BPCHAR NULL, col54_5 PG_LSN NOT NULL, col54_6 TSQUERY NOT NULL, col54_7 DECIMAL NULL, col54_8 DATE NOT NULL, col54_9 BYTES NOT NULL, col54_10 BOOL NOT NULL, col54_11 BIT(41) NOT NULL, col54_12 REGCLASS NULL, col54_13 TIMESTAMP NOT NULL, col54_14 INET NOT NULL, col54_15 STRING NOT NULL AS (lower(CAST(col54_5 AS STRING))) STORED, rowid INT8 NOT NULL NOT VISIBLE DEFAULT unique_rowid(), CONSTRAINT table54_pkey PRIMARY KEY (rowid ASC), INDEX table54_col54_14_idx (col54_14 DESC) STORING (col54_1, col54_2, col54_3, col54_5, col54_8, col54_9, col54_10, col54_11, col54_12, col54_15) NOT VISIBLE, INDEX table54_col54_4_col54_11_idx (col54_4 DESC, col54_11 DESC) PARTITION BY LIST (col54_4, col54_11) (PARTITION table54_part_0 VALUES IN (('U', B'11111110011001110011011011100101010011111'), ('N', B'00010011010001100011101110111010010111000'), (e'\\U00002603', B'10100000101011111001110001000111110100111'), ('(', B'11000000010001111000000101010100111110000'), ('d', B'10011010010011111110111000100110101110011'), ('J', B'11101101011001110111010111000010010011101'), (e'\\x00', B'01010010100010001101100101000111000100010'), ('|', B'00111111010100111000111100111110001000100'), (e'\\x00', B'00111000111011010010101001110100000100011')), PARTITION table54_part_1 VALUES IN ((e'\\x0b', B'01111111111111111111111111111111111111111'), ('k', B'10011110001001111011010010110100010101010'), (e'\\x00', B'11101010000011000001010011000001010101110'), ('y', B'01001010011101101101100010010001110110111'), ('}', B'11001000001100011010111000000110000010010'), ('7', B'00011001101001110111011001001010101110000'), (')', B'00111001101000010111000101100110010100010'), (e'\\x14', B'11101100001010101011001001011001001000001'), (e'\\x10', B'10000001000000101010000110100010011000111')), PARTITION table54_part_2 VALUES IN (('', B'11001110111110101000111010001001100101101'), ('a', B'10011001100111010110111101101101110101000'), ('*', B'01101001100111101100001110011001000110110'), (')', B'01100010010001010011110001100000011110101'), ('M', B'00000000000000000000000000000000000000000'), ('Y', B'01001010001001110100111010111000000000100'), ('}', B'11101001101010001101101011001010101000100'), (e'\\x0f', B'01000101101010101011001111100111100111100'), ('\"', B'10001010011110010101000100110000100110011')), PARTITION table54_part_3 VALUES IN ((e'\\x00', B'01001011100111110001101100000100101010101'), ('A', B'11001111000010110010000010001001100110100'), ('#', B'11001011110110100000001110011100010101010'), ('M', B'01110110011100010111110100111100101111000'), ('w', B'11111100010101000000111010111000110111011'), ('5', B'00010110011110010011011010000111111101011'), ('4', B'00000000000000000000000000000000000000000'), (e'\\x00', B'10000111000101010110011000011001100101001'), ('W', B'00101100001101111000010110100100000001101')), PARTITION table54_part_4 VALUES IN (('\"', B'01011000110010011000110100111001110101111'), ('r', B'00000100100010100010000011010011010111110'), ('(', B'11101011100111011011101001101010001111100'), (e'\\'', B'00000000000000000000000000000000000000000'), ('+', B'01111110011110111010000010100011010110001'), ('x', B'00000000000000000000000000000000000000000'), ('', B'10111001110101100000111010010101110100111'), ('y', B'10001000100101000100001100001100101000100'), (e'\\n', B'00001100000100110001000101111101110001101')), PARTITION table54_part_5 VALUES IN (('_', B'11001010010101001000010011110101110111101'), (e'\\x17', B'00000000000000000000000000000000000000000'), ('Y', B'10100010101010010111100001000010001110001'), ('&', B'01111000000100111011101010111010011010101'), ('^', B'01111111111111111111111111111111111111111'), ('@', B'00111010001110011011011000110000011000010'), (')', B'11001011000011001011011101101011110111001'), ('>', B'10001001101101101100000011101100110110001'), ('X', B'11010110011100111011111001111010001110011')), PARTITION \"DEFAULT\" VALUES IN ((DEFAULT, DEFAULT))), INDEX table54_col54_11_idx (col54_11 ASC))"
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema1206636766
--- FAIL: TestCreateRandomSchema (3.57s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/tests.TestCreateRandomSchema [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17732729?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17732729?buildTab=artifacts#/) on master @ [2520c51663e57de54706a58e8e68370467dbe871](https://github.com/cockroachdb/cockroach/commits/2520c51663e57de54706a58e8e68370467dbe871):


```
=== RUN   TestCreateRandomSchema
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema84755627
    test_log_scope.go:76: use -show-logs to present logs inline
    random_schema_test.go:107: for input statement
        CREATE TABLE table68 (col68_0 BPCHAR NOT NULL, col68_1 INTERVAL NOT NULL, col68_2 FLOAT8 NOT NULL, col68_3 TIMESTAMP NOT NULL, col68_4 TIMESTAMP NOT NULL, col68_5 FLOAT8 NOT NULL AS (abs(col68_2)) STORED, PRIMARY KEY (col68_0 ASC, col68_2 ASC, col68_4), UNIQUE (col68_5) STORING (col68_1, col68_3), UNIQUE (col68_1 DESC, abs(col68_2) DESC, col68_0 ASC) STORING (col68_3, col68_5), UNIQUE (col68_5, col68_4 DESC, col68_3), UNIQUE (col68_0 DESC, col68_3 DESC), UNIQUE (col68_0 ASC) STORING (col68_1, col68_5) PARTITION BY LIST (col68_0) (PARTITION table68_part_0 VALUES IN (('}':::STRING,), (e'\r':::STRING,), ('d':::STRING,), ('E':::STRING,), ('?':::STRING,), ('i':::STRING,)), PARTITION table68_part_1 VALUES IN (('S':::STRING,), (',':::STRING,), (e'\U00002603':::STRING,), ('e':::STRING,), (e'\x10':::STRING,), ('a':::STRING,)), PARTITION table68_part_2 VALUES IN ((e'\x0e':::STRING,), ('x':::STRING,), ('H':::STRING,), ('A':::STRING,), ('/':::STRING,)), PARTITION table68_part_3 VALUES IN (('&':::STRING,), ('"':::STRING,), ('D':::STRING,), (e'\'':::STRING,)), PARTITION table68_part_4 VALUES IN (('b':::STRING,), ('7':::STRING,), (e'\n':::STRING,), ('W':::STRING,), ('X':::STRING,)), PARTITION table68_part_5 VALUES IN (('m':::STRING,), ('4':::STRING,), ('>':::STRING,), ('n':::STRING,), ('J':::STRING,)), PARTITION table68_part_6 VALUES IN (('%':::STRING,), ('f':::STRING,), ('P':::STRING,)), PARTITION table68_part_7 VALUES IN (('t':::STRING,), ('Q':::STRING,), ('g':::STRING,), ('2':::STRING,), (' ':::STRING,)), PARTITION table68_part_8 VALUES IN ((e'\x0f':::STRING,), (e'\x13':::STRING,), (e'\x06':::STRING,))), INDEX (abs(col68_2) ASC) STORING (col68_3, col68_5))
        found first output
        "CREATE TABLE public.table68 (col68_0 BPCHAR NOT NULL, col68_1 INTERVAL NOT NULL, col68_2 FLOAT8 NOT NULL, col68_3 TIMESTAMP NOT NULL, col68_4 TIMESTAMP NOT NULL, col68_5 FLOAT8 NOT NULL AS (abs(col68_2)) STORED, CONSTRAINT table68_pkey PRIMARY KEY (col68_0 ASC, col68_2 ASC, col68_4 ASC), CONSTRAINT table68_col68_5_key UNIQUE (col68_5 ASC) STORING (col68_1, col68_3), CONSTRAINT table68_col68_1_expr_col68_0_key UNIQUE (col68_1 DESC, abs(col68_2) DESC, col68_0 ASC) STORING (col68_3, col68_5), CONSTRAINT table68_col68_5_col68_4_col68_3_key UNIQUE (col68_5 ASC, col68_4 DESC, col68_3 ASC), CONSTRAINT table68_col68_0_col68_3_key UNIQUE (col68_0 DESC, col68_3 DESC), CONSTRAINT table68_col68_0_key UNIQUE (col68_0 ASC) STORING (col68_1, col68_5) PARTITION BY LIST (col68_0) (PARTITION table68_part_0 VALUES IN (('}'), (e'\\r'), ('d'), ('E'), ('?'), ('i')), PARTITION table68_part_1 VALUES IN (('S'), (','), (e'\\U00002603'), ('e'), (e'\\x10'), ('a')), PARTITION table68_part_2 VALUES IN ((e'\\x0e'), ('x'), ('H'), ('A'), ('/')), PARTITION table68_part_3 VALUES IN (('&'), ('\"'), ('D'), (e'\\'')), PARTITION table68_part_4 VALUES IN (('b'), ('7'), (e'\\n'), ('W'), ('X')), PARTITION table68_part_5 VALUES IN (('m'), ('4'), ('>'), ('n'), ('J')), PARTITION table68_part_6 VALUES IN (('%'), ('f'), ('P')), PARTITION table68_part_7 VALUES IN (('t'), ('Q'), ('g'), ('2'), (' ')), PARTITION table68_part_8 VALUES IN ((e'\\x0f'), (e'\\x13'), (e'\\x06'))), INDEX table68_expr_idx (abs(col68_2) ASC) STORING (col68_3, col68_5))"
        but second output
        "CREATE TABLE public.table68 (col68_0 BPCHAR NOT NULL, col68_1 INTERVAL NOT NULL, col68_2 FLOAT8 NOT NULL, col68_3 TIMESTAMP NOT NULL, col68_4 TIMESTAMP NOT NULL, col68_5 FLOAT8 NOT NULL AS (abs(col68_2)) STORED, CONSTRAINT table68_pkey PRIMARY KEY (col68_0 ASC, col68_2 ASC, col68_4 ASC), CONSTRAINT table68_col68_5_key UNIQUE (col68_5 ASC) STORING (col68_1, col68_3), CONSTRAINT table68_col68_1_expr_col68_0_key UNIQUE (col68_1 DESC, abs(col68_2) DESC, col68_0 ASC) STORING (col68_3, col68_5), CONSTRAINT table68_col68_5_col68_4_col68_3_key UNIQUE (col68_5 ASC, col68_4 DESC, col68_3 ASC), CONSTRAINT table68_col68_0_col68_3_key UNIQUE (col68_0 DESC, col68_3 DESC), CONSTRAINT table68_col68_0_key UNIQUE (col68_0 ASC) STORING (col68_1, col68_5) PARTITION BY LIST (col68_0) (PARTITION table68_part_0 VALUES IN (('}'), (e'\\r'), ('d'), ('E'), ('?'), ('i')), PARTITION table68_part_1 VALUES IN (('S'), (','), (e'\\U00002603'), ('e'), (e'\\x10'), ('a')), PARTITION table68_part_2 VALUES IN ((e'\\x0e'), ('x'), ('H'), ('A'), ('/')), PARTITION table68_part_3 VALUES IN (('&'), ('\"'), ('D'), (e'\\'')), PARTITION table68_part_4 VALUES IN (('b'), ('7'), (e'\\n'), ('W'), ('X')), PARTITION table68_part_5 VALUES IN (('m'), ('4'), ('>'), ('n'), ('J')), PARTITION table68_part_6 VALUES IN (('%'), ('f'), ('P')), PARTITION table68_part_7 VALUES IN (('t'), ('Q'), ('g'), ('2'), ('')), PARTITION table68_part_8 VALUES IN ((e'\\x0f'), (e'\\x13'), (e'\\x06'))), INDEX table68_expr_idx (abs(col68_2) ASC) STORING (col68_3, col68_5))"
    panic.go:626: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5f3a151259b697f0a5a8cbe156e7f1b7/logTestCreateRandomSchema84755627
--- FAIL: TestCreateRandomSchema (4.42s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCreateRandomSchema.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/randgen/datum.go
+++ b/pkg/sql/randgen/datum.go
@@ -11,6 +11,7 @@ import (
 	"math"
 	"math/bits"
 	"math/rand"
+	"strings"
 	"time"
 	"unicode"
 
@@ -261,6 +262,9 @@ func RandDatumWithNullChance(
 		}
 		if typ.Oid() == oid.T_name {
 			return tree.NewDName(string(p))
+		}
+		if typ.Oid() == oid.T_bpchar {
+			return tree.NewDString(strings.TrimRight(string(p), " "))
 		}
 		return tree.NewDString(string(p))
 	case types.BytesFamily:

